\documentclass[11pt]{article}
%\documentclass[draft]{article} for better debugging!

\usepackage{cite}

\linespread{1.3}

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}

\begin{document}

The function of our society depends on safe and efficient municipal wastewater remediation while sustainably recycling water and reducing environmental impact. Waste influent created by domestic households is extensive and fluctuates in composition. Hence the wastewater treatment system for influent remediation needs to be adaptable to deal with the volume and variability. The biological component employed by the treatment facilities is the least well understood aspect. This study sought to develop an experimental system to explore the interaction between lipids and activated sludge. Specifically, five aims were addressed relating to spatial aspects of lipid partitioning and colonization, impacts on bacterial community composition, isolation of lipid degrading bacteria and testing for AHL production.

\subsection{Physical behaviour of lipids in activated sludge}
The physical partitioning behaviour of lipids upon introduction to activated sludge is poorly described. As a starting point it was unclear whether the addition of glycerol trioleate (GT) to activated sludge would result in the formation of a single free phase lipid droplet or whether sludge flocs and the incubation vessel would be coated in lipid with an absence of free phase or if multiple small lipid droplets or an emulsion would form. This lack of information on the physical partitioning behaviour in activated sludge hampers development of an experimental model for lipid dependent floc formation. To this end, 1 and 2 \% (v/v) GT was incubated in the presence of activated sludge and direct observations were made (Aim 1). \\

During the enrichment on 1 \% GT (v/v), multiple lipid droplets formed which were suspended throughout the sludge. Objects that resemble droplets in lipid challenged activated sludge have been reported previously (ref), however the lipid concentration to elicit this response was not described. It stands to reason that such lipid droplets represent an attractive surface substratum for colonization by activated sludge bacteria enabling evasion of predation and ready access to a carbon and energy source. Biofilm formation on lipid droplets would ultimately generate aggregated biomass resembling a floc. The result was encouraging in that it suggested lipid colonization may represent a realistic model system for floc formation and development. \\

When the GT concentration was increased to 2 \% (v/v), lipid droplets did not form and the sludge increased in viscosity and lightened in colour, resembling an emulsion. This is likely due to the excessive release of fatty acids released from the high concentration of triacylglyceride applied. The 2 \% (v/v) lipid concentration applied to the activated sludge was adapted from a study conducted by Haba et al \cite{haba2000isolation} on isolation of lipase producing bacteria. However the typical lipid concentration found in municipal wastewater is approximately 0.01 \% (v/v) \cite{Forster_92}. \\

Continued enrichment of the emulsion through sub-culturing resulted in increased viscosity to an almost solid state and white appearance in the tertiary sub-culture 45 days. No incidence of an emulsion like phenotype in the wastewater treatment plant setting has been found. Hence, these phenomena are considered to be artifacts of the unrealistically high GT concentration applied.

\subsection{Bacteria colonise lipid droplets in activated sludge}
Addition of 1 \% (v/v) GT to activated sludge resulted in lipid droplet formation. Epifluorescence microscopy was then used to observe physical interactions (spatial relationships) between the activated sludge biomass and the lipid droplets (Aim 2). \\

The microscopy images obtained clearly showcase the close association of biomass with the lipids. Whilst the aggregation around the droplets resembles biofilm formation, it is not clear if the physicochemical properties of cells and lipid droplets drive the association or if active energy dependent processes result in microbial colonization of the lipid droplets.

Whether the dense mass of cells observed is surrounding and growing on the lipid or whether it was embedded within the lipid phase is unclear from the microscopy images generated. Cells embedded in the lipid, with no access to water, are unlikely to proliferate. If lipids seed flocs, the buoyancy of the lipid core could be responsible for lack of settlement rather than the presence of lipids preventing flocculation as suggested by Forster et al \cite{Forster_92}. Determining which microbial communities dominate during lipid induced floc settlement failure could provide another aspect for monitoring and anticipation of system failure.

\subsection{Activated sludge bacteria produce lipases and degrade lipid droplets}

Lipid droplet size was observed to decrease over time until they were no longer visible. This is consistent with microbes colonising the droplets and degrading the surface substratum as a carbon and energy source. Lipase activity was also observed to increase in activated sludge samples amended with GT. An increase in lipase activity was observed in the EPS over the first 3 days of incubation but the greatest fold increase in lipase activity was observed in the culture supernatant between 3 and 6 days of incubation as the EPS based lipase activity decreased. It has previously been shown that lipases are weakly bound to the EPS of flocs through hydrogen bonding \cite{mayer1999role,wicker1987}. The sequential appearance of lipase first in the EPS and then in the supernatant fraction is likely due to release of lipases from the EPS.

While lipase activity was observed in the biomass fraction, there was little difference in activity when comparing GT amended treatments and unamended controls. The unamended controls displayed a high background level of lipase activity presumably owing to the presence of background levels of lipid in activated sludge. The decline in lipase activity observed in the EPS fraction in unamended controls is concordant with this, with the background lipid component being consumed. It should also be noted that as the viscosity of samples increased it was difficult to separate the three distinct fractions.

\subsection{Sphingomonads dominate in GT amended activated sludge.}
It is not known which bacteria are responsible for the degradation of lipids in activated sludge. This information is crucial in the development of an activated sludge floc formation model based on lipid colonization. As a starting point, the response of bacteria in activated sludge to the addition of GT was assessed using DGGE (Aim 3). It is evident from the DGGE community analysis that lipids have a profound impact on the microbial community structure in activated sludge. Whilst bacterial lineages enriched in the presence of GT can be implicated in lipid degradation in activated sludge this does not represent unambiguous evidence of lipid biodegradation by these bacteria.

Based on band intensity the most abundant bacterial lineages in the untreated sludge samples belonged to the \emph{Bacillus}, \emph{Dechloromonas} and \emph{Azospira} genera. \emph{Bacillus} species have been observed in sludge previously and the sequence retrieved here was associated with lipolytic activity in olive mill wastewater \cite{ertuugrul2007isolation}. Bacillus species have also been associated with bioflocculation in starch wastewater treatment \cite{deng2003characteristics}.

Dechloromonas species were observed in membrane fouling biofilms of municipal wastewater treatment plants in Japan, with over 30 \% of clones belonging to this genus \cite{miura2007membrane}. Azospira are also common activated sludge occupants \cite{tan2003dechlorosoma,reinhold2000reassessment,hunter2007azospira,wilhelmus2013microbiological}.
Overall, the DGGE profiles of the unamended sludge community was typical of activated sludge and did not differ greatly over the incubation period.

In sludge samples exposed to GT the DGGE profiles over time shifted dramatically. The most abundant bacteria observed in the unamended sludge were replaced within 9 days of incubation with \emph{Sphingomonas}, \emph{Novosphingobium} and \emph{Roseomonas} species.
\emph{Sphingomonas} species are present at about 5 - 10 \% relative abundance in sludge as shown by FISH \cite{neef1999detection}, and play an important role in wastewater remediation. Members of this genera degrade testosterone and sterol hormones as well as the pollutant nonylphenol \cite{fujii2001sphingomonas,roh201017beta}. \emph{Novosphingobium} species have been shown to play an important role in wastewater remediation by degrading toxic dyes as well as estrogen - both which impact the ecosystem if released \cite{addison2007novosphingobium,hashimoto2009contribution}.
Roseomonas species have been found at about 5 \% relative abundance in activated sludge and are known to degrade organophosphate pesticides \cite{jiang2008bacterial,jiang2006isolation}. None of these lineages have previously been associated with lipid consumption in wastewater treatment plants. From the data generated here bacteria belonging to the \emph{Sphingomonas}, \emph{Novosphingobium} and \emph{Roseomonas} genera can be considered candidates for inclusion in an experimental system for investigating lipid based floc formation. 
%mike worked till here.
The further enrichment DGGE shows the communities dominant in several settings. It was noted that aggregates formed within the emulsions, they appeared solid suspended within the viscous cultures but disintegrated when probed and may have been entirely colonised lipids. Hence the microbial community within these was of interest and one such aggregate was targeted for further secondary and tertiary enrichment. The aggregate expanded and developed tendrils upon further enrichment and cells dispersed into the supernatant, which was tertiarily enriched, seperate to the aggregate. The tertiary enrichment time point (t3) is taken after 45 days to elucidate the microbiome of long term enrichment.


\emph{Nevskia} sp., band 1, is present in the aggregate at day 12 and then cease to exist at detectable levels at later time points for the aggregate. It is present in t3 aggregate supernatant, so it could have dissociated from the conglomerate to enter the supernatant phase. \emph{Nevskia} sp. are slow growing, where colony formation takes about a week, and they produce lipase \cite{kim2011nevskia}. A study by Chooklin et al endeavoured to isolate the most efficient surfactant producer from palm oil mill effluent for lipid remediation and found \emph{Nevskia} sp. \cite{chooklinutilization}.


Similarly, \emph{Sphingomonas} sp. from band 2 are present throughout day 12 to 18 but are absent in the t3 for the aggregate. They do appear in the aggregate supernatant as band 10, which band was also assigned the identity of \emph{Sphingomonas} sp. by NCBI, albeit with a different accession number. The sequences used to identify band 2 and band 10 were aligned with BLAST, with lengths of 445 bp and 410 bp respectively. The sequence identity was 100 \% with an E-value of 0.0 hence the organisms that represent the two bands are regarded as synonymous regardless of differing accession numbers.
\emph{Sphingomonas} sp. also represent bands 4 and 8. The former is exclusive to primary aggregate sampling and disappears by the tertiary enrichment. The latter arises in t3 aggregate and emulsion.
Band 12, \emph{Sphingomonas} sp., appears in t3 emulsion. Whether the band in the 1 and 2 \% GT comparison is the same organisms is not possible to determine without sequencing as these bands are fractionally above the t3 bands.
From the DGGE gel, bands 2 and 13 appear to be the same organism. However during identity assignment, the latter returned as \emph{Stakelama pacifica} from NCBI with and E-value of 3e$^{-77}$ from a 163 bp sequence, while band 2 was concluded to be \emph{Sphingomonas} sp. with a more reliable E-value of 0.0 based on a 445 bp sequence. The two sequences were aligned with NCBI BLAST to investigate sequence similarity. The sequence alignment of band 13, which covers the 269 to 431 bp region of the amplified 16S rRNA fragment, aligns from the 269$^{th}$ bp of band 2 with a sequence identity was 100 \% with an E-value of 7e$^{-87}$. It is concluded that band 13 represents the \emph{Sphingomonas} from band 2 rather than \emph{Stakelama pacifica}.
The designation \emph{Sphingomonas} sp. arose for 5 different bands which shows high variability in GC content in this genus. Similar variance for DGGE results has been recoded \cite{qiao2012effect}.

\emph{Sphingobium} sp., the designation for band 3, is prevalent day 15 and 18 in the aggregate as well as day 7 in the secondary emulsion enrichment. A members of this genus, isolated from river sediment, can degrade nonylphenol \cite{ushiba2003sphingobium}.


Band 5, \emph{Rhodovarius lipocyclicus}, is abundant from day 18 in the aggregate as well as in the tertiary enrichment and day 7 for t2 emulsion. However the highest abundance is recorded in the t3 aggregate supernatant, but it was not recorded in the tertiary emulsion enrichment. Limited information is available about \emph{Rhodovarius lipocyclicus}, beyond the basic information required for classification \cite{kampfer2004rhodovarius}.


\emph{Xanthobacter} sp., bands 6 and 7, are prevalent on day 18 in the aggregate. Band 7 also appears in the tertiary enrichment of the aggregate supernatant. As DGGE separates sequences by GC content, these may be two closely related species of the genus \emph{Xanthobacter} with slightly divergent GC content of the region amplified. This genus is reported to remediate aliphatic halogenated compounds which are commonly found in municipal wastewater \cite{janssen1985degradation} and have been shown to co-ordinate with \emph{Novosphingobium} sp. to degrade polyvinyl alcohol \cite{rong2009symbiotic}. \emph{Novosphingobium} sp., bands 7 and 8 in the GT enriched  4, also degrade polycyclic aromatic hydrocarbons, which can by synthesised from lipid precursors \cite{addison2007novosphingobium}.


Members of the \emph{Bradyrhizobium} genera were designated as band 9 in the t3 aggregate supernatant and found in the emulsions t2 and t3. This genus is slow growing \cite{rebah2002wastewater} and usually associated with plant nodules. In plant symbiosis, they contribute by fixing nitrogen - an attribute required for successful A/A/O process. When exposed to activated sludge, these microbes have been shown to become highly antibiotic and heavy metal resistant \cite{ahmad17samiullah}.


Candidatus \emph{Competibacter} came up as band 11, only in the secondary emulsion enrichment. Members of this genus are glycogen accumulating organisms, which produce polycyclic aromatic hydrocarbons, and represent 22 - 26 \% of enriched sludge microbiota \cite{bengtsson2008production,lemaire2008microbial}. 


Band 14 shows low abundance \emph{Oleomonas} sp., in t3 emulsion. However this genus was identified with FISH and DGGE, to constitute about 16 \% of the biomass in an upstream anaerobic bioreactor fed with brewery wastewater \cite{fernandez2008analysis}. Specifically \emph{Oleomonas sagaranensis} is involved in breaking down urea \cite{kanamori2005allophanate,kanamori2004enzymatic} which is essential for nitrification of ammonia in activated sludge.


While none of the bands from the 1 and 2 \% enrichment comparison were amplifiable for sequencing, the organisms active in these cultures were very similar. While the bands are present for both treatments, three of the bands show a difference in intensity and hence abundance.
mike - should I include letters for these bands on the DGGE picture in results so I can refer to them?

\subsubsection{\emph{Common players}}
\emph{Sphingomonas} sp. are common in both replicate 4 enrichment as well as the various further enrichments.
Approximately 92 \% of identified key organisms from DGGEs belonged to the proteobacteria. Of these 75 \% were \emph{$\alpha$}-proteobacteria  and each \emph{$\gamma$}- and \emph{$\beta$}- proteobacteria were 12.5 \%.
Hesham et al compared the microbial communities of two differently operated municipal wastewater treatment plants over six months via DGGE. The 11 OTUs common to both plants, and the most abundant, were 18 \% to alpha-proteobacteria and 18 \% to beta-proteobacteria \cite{Hesham_11}.

If the bands found in t3 from the aggregate and the aggregate supernatant are considered together, the banding pattern closely resembles that of the t3 emulsion. Despite the removal of the aggregate two transfers prior, the microbial community in the t3 aggregate and supernatant is highly similar with the difference of the absence of \emph{Rhodovarius lipocyclicus} in t3 emulsion.


While the majority of the microbes identified in the further enrichments, but not found in the enrichment or control cultures for the 25 day duration of the enrichment experiment, these microorganisms are present in low abundance and become prevalent when their niche ability to degrade certain recalcitrant compounds becomes relevant. Also the detection of organisms was limited as several bands were not sequenceable.

Wagner et al suggest fluorescence in situ hybridisation (FISH) with 16S rRNA  group-specific probes is necessary to accompany 16S rRNA sequencing sampling to ensure the accuracy of the OTU representation. However as this is a time consuming process, the majority of the community analysis is on 16S rRNA only \cite{Wagner_02} . A skewered representation of the OTUs in the community can arise from 16S primer bias which should be checked and accounted for.

\subsection{Lipid colonising isolates behave differently in the presence of lipids}

LC1, \emph{Pandoraea} sp., produces a pink phenotype when cultured in the presence of lipids which could be due to pigment production. This suggests that this expression pattern is activated in the presence of lipids. \emph{Pandoraea} sp. did not evoke a response from the LuxR bioassay and a search of NCBI Protein BLAST returns a LuxR homolog but not a LuxI homolog. Nonetheless a \emph{Pandoraea} sp. soil isolate has been shown to produce the AHL \emph{N}-octanoylhomoserine lactone, which has 8 Cs attached to the lactone ring \cite{han2013pandoraea}. As the activation was negligible in this study, this AHL may be out of the detection range of the LuxR bioassay and should be tested with the \emph{Chromobacterium violacein} CV026 bioassay.

\emph{Achromobacter} sp., LC2, produced a minor LuxR response when cultured without oil and cultured in the presence of oil, a response equivalent between 0.5 and 1 nm OHHL when compared to the standard curve (Figure x). The \emph{Achromobacter} sp. genome was searched a \emph{luxI} homolog by using a \emph{Burkholderia} sp. homolog as  this is a \emph{$\beta$}- proteobacterium known to contain an AHL synthase. The databases UniProt and SwissProt were used through EXPASY in order to search annotated genomes only and exclude draft genomes. This returned  22 \% similarity to an \emph{Agrobacterium} sp. LuxI homolog.

LC3, \emph{Enterobacter} sp., evoked a LuxR response equivalent to approximately 1 nM OHHL when cultured without oil and a response equivalent to 2.5 nM when cultured with lipid. The LuxI homolog for members of this genus can be found under the accession number  WP\_008503217.

LC4, a \emph{Pseudomonas} sp., evoked a response akin to 0.5 nm OHHL from the LuxR bioassay when cultured in the presence of lipid. Members of this genus tend to have two AHL dependent gene expression systems and an autoinducer synthesis protein can be found under the accession number   WP\_018604918.

Isolate LC5 was of interest due to it's high bioassay response, exceeding the 2.5 nM OHHL standard. LuxR response is similar when cultured with and without lipid. The organism could not be identified as the sequencing data was inconclusive for both attempts.

\end{document}