\documentclass{article}
\title{\textbf{Quorum-sensing influence on lipase production in activated sludge}}
\author{Anna Liza Kretzschmar\\
        z3218219\\
        Supervisor: Mike Manefield}
\date{}
%\usepackage{cite}
\usepackage[nottoc,numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\linespread{1.3}
\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}
\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\begin{document}

\maketitle

\tableofcontents

\newpage

% remove preamble incl \begin\end{doc} from \input files.

\section{Acknowledgements}

\newpage
\section{List of Figures}

\listoffigures

\newpage

\section{List of Tables}

\listoftables

\newpage
\section{List of Abbreviations}

\newpage
\section{Introduction}
Wastewater treatment plants are common in the global urban setting and a necessity for the continued function of our society. Fluid waste enters the plant, is purified and the effluent can be re-introduced into the environment. It is necessary to understand the processes in order to optimise the functionality of the plant, as the global population expands and strains the limited water resources available. In 1914 Arden and Lockett pioneered the development of activated sludge, which is commonly employed today \cite{ardern1914experiments}.


The microbes in activated sludge, which are incubated with filtered waste influent, oxidise the sewage particulates \cite{Price_95}. Microbial consortia in activated sludge form a detached biofilm called a floc, however what seeds floc formation is still unknown. Inter bacterial communications induce transcriptomic change via a system called quorum-sensing, which plays a major role in a consortium's interaction\cite{parsek2005sociomicrobiology}. Hence our understanding of the phenotypes impacted by this communication is essential for optimising the process of water remediation \cite{singh2006biofilms}

\subsection{Wastewater treatment is essential to civilisation}
The development of the current municipal wastewater treatment system has evolved over millennia to combat the waste arising from congregated human habitation. The Mesopotamian empire, from between 3500 to 2500 BC, built connections from some homes to storm water drains to remove wastes, while the Babylonians developed clay pipes to fulfil the same purpose \cite{lofrano2010}. It is a logical necessity to develop such a system to prevent disease and minimise environmental impact globally and it has been continuously improved. In 1914 Arden and Lockett pioneered the development of activated sludge, which is one of the most commonly employed today \cite{jenkins2004manual,muchie2010bioremediation}


Current plants combine physical, chemical and biological treatment strategies to the polluted influent in order to reduce environmental and health impacts. Hence these treatments are in place to minimise the presence of N, P and C; reduce the biological and chemical oxygen demands; as well as to filter out harmful substances and pathogens \cite{mayhew1997low}.
Water quality is in part assessed by the dissolved oxygen concentration, which reducing agents can unbalance. The parameters for this assessment are measured by the concentration of substances present that can be oxidised chemically or biologically to inorganic end products \cite{pisarevsky2005chemical}.

\subsubsection{\emph{Microbial floc formation underlies successful wastewater treatment}}
Initially, large solids in the influent are either filtered out or broken down into smaller particulates to prevent blockages or excessive mechanical strain during the process. During the primary treatment phase, the influent is deposited in a large settlement tank where the majority of the particles settle to the bottom due to gravitational force. The weir controlled outlet flow is directed to the aeration tank for secondary treatment. The activated sludge in the aeration tank comprises the biological aspect of the treatment. It is an oxygenated system where the influent is incubated with microbial consortia, predominantly affiliated with flocs, which live by oxidising the sewage particles in the mixed liquor \cite{mayhew1997low}. The suspension is drawn off into a final settlement tank where the sludge flocs settle at the bottom, and are pumped out and recycled back into the mixed liquor; or discarded. Depending on the required effluent quality, tertiary treatment may also be required to remove residual solids, compounds or planktonic bacteria that could pollute or enrich the effluent destination \cite{Price_95}.
% knowledgegap
% jesus fucking christ, more references

Bioflocculation is essential for wastewater treatment, as the settlement of the flocs is key to drawing off remediated water. Biofilms, of which flocs are a type of, constitute an aggregated community of microbes at the liquid-solid interface surrounded by secreted extracellular polymeric substances (EPS), consisting of polysaccharides, nucleic acids, lipids and proteins \cite{wingender1999}, which represents 85-90 \% of biofilm dry weight \cite{Frolund_96}. 


The composition of the biofilm is dependent on the participating species coupled with the environment, and are sites of rapid adaptation through continuous natural selection for survival \cite{boles2008,matz2005,palmer2001}.  
The EPS offers the advantages of defence against predation by other bacteria \cite{rao2005} and protozoic grazing \cite{matz2005}; increased resistance against some chemicals such as antibiotics and hydrogen peroxide \cite{burmolle_06}; a stable matrix for the cells to reside in \cite{Flemming_10}; and to immobilise extracellular enzymes offering proximity while protecting them from proteolysis and conferring resistance to higher temperatures \cite{wingender2002extracellular,Flemming_10,skillman1998}.

\subsubsection{\emph{Microbial composition determines wastewater treatment efficiency}}
To optimise the efficiency of wastewater treatment, understanding the composition of the microbial communities at work in activated sludge and how their interactions influence their capabilities of remediation is essential \cite{daims2006}.
Single celled organisms from all three domains of life reside in this system - protozoa, bacteria, nematodes, fungi and viruses. All of these play a role in degradation of solids and nutrient cycling and hence in wastewater remediation \cite{muchie2010bioremediation}. The domain most active in remediation are the bacteria \cite{spellman2008handbook}, which will be focused upon.


The advance of community analysis techniques, which exclude the necessity to culture microbes in the laboratory, has made it possible to investigate the composition of microbial communities, including in activated sludge. The most frequent target for operational taxonomic unit analysis centres around the 16S ribonucleic acid (rRNA) gene, which is then compared to expansive 16S databases such as NCBI BLAST. Several techniques can be utilised around 16S rRNA, such as constructing a 16S rRNA gene library \cite{McGarvey_04}, analysing ribosomal intergenic spacer sequences \cite{Yu_01}, 16S-restriction fragment length polymorphism \cite{Gilbride_06} and comparing community structures via denaturing gradient gel electrophoresis (DGGE) \cite{Hesham_11} or pyrosequencing \cite{wang2012pyrosequencing}.


In wastewater treatment plants, the microbial community composition is strongly dependent on what type of waste influent the microbes are exposed to, as well as the operational settings, such as chemical oxygen demand \cite{Gilbride_06,wang2012pyrosequencing,hu2012microbial}. Municipal wastewater treatment plant influent varies in composition \citep{henze2002wastewater}.


\begin{sidewaystable}[!htbp]
\begin{tabular}{ | l | l | p{4.5cm} | p{7cm} | l | }
\hline
Study & Method & Sample size & Core microbiome composition & Refernce\\
\hline
Hesham et al. & DGGE & One plant with two different operational modes over 6 months, China & 2x \emph{$\alpha$-} \& 2x \emph{$\beta$- proteobacteria}; 3x \emph{bacteroidetes}; 2x \emph{actinobacteria}; 2x \emph{firmicutes} & \cite{Hesham_11} \\
\hline
Wagner et al. & DGGE & 750 16S rRNA sequences from wastewater treatment plants and reactors & \emph{$\alpha$-, $\beta$-} \& \emph{$\gamma$- proteobacteria}; \emph{bacteroidetes};\emph{actinobacteria} & \cite{Wagner_02} \\
\hline
Wang et al. & Pyrosequencing & 14 plants \& pilot/benchtop operations in China & 21 - 53 \% Proteobacteria, where \emph{$\beta$-, $\alpha$-} \& \emph{$\gamma$-proteobacteria} were present 21 - 52 \%, 7 - 48 \% and 8 - 34 \% respectively & \cite{wang2012pyrosequencing} \\
\hline
Hu et al. & Pyrosequencing & 12 plants where 4 were A/A/O, China & In A/A/O proteobacteria dominated, with up to 2:1 ratio to bacteroidetes. In other types of plants the dominant phylum proteobacteria was occasionally replaced by bacteroidetes & \cite{hu2012microbial} \\
\hline
Ranasinghe et al. & Pyrosequencing & 12 plants over two years, Japan & Proteobacteria represented 38 \% of total assigned reads where 15 \% belonged to $\beta$-proteobacteria & \cite{ranasinghe2012revealing} \\
\hline
Xia et al. & Microarray & Three plants in China and two in USA & Proteobacteria wast the dominant phylum at 50 \% to 62 \%, where $\gamma$-proteobacteria represented  31 \% to 38 \% and $beta$-proteobacteria 30 \% to 35 \% & \cite{xia2010bacterial} \\
\hline

\end{tabular}
\caption{Studies characterising core microbiomes found in activated sludge}
\end{sidewaystable}


Table 1 summarises methods used to analyse activated sludge microbial communities across geographical and operational settings, showing a core microbiome. Pyrosequencing and microarrays studies give a snapshot of the microbial community, including low level abundance organisms \cite{ranasinghe2012revealing}. DGGE gives an insight into the change in abundance of dominant organisms active under certain conditions and over time. 

\subsubsection{\emph{Samples were aquired from activated sludge in St. Mary's wastewater treatment plant}}
St Mary's municipal waste water treatment plant is located in St Mary's in western Sydney serving a population of app. 160000 in a catchment area of 84 km$^{2}$ and discharges into the Hawkesbury-Nepean River. On a daily basis the plant processes about 35 million L of wastewater. Wastewater can enter two influent streams - stage 1 and 2 or stage 3. The streams are subjected to differing primary treatments, however secondary and tertiary treatments are the same. The biological treatment is part of the secondary treatment stage, as can be seen in Figure 1. The biological process is conducted in three stages (A/A/O): 1. Anaerobic zone, where microbes take up carbon and release phosphates; 2. Anoxic zone, where carbon is consumed and nitrates are released as nitrogen gas; 3. Aerobic zone, where nitrification of ammonia occurs and organic matter increasing the oxygen demand is reduced \cite{stmarys}.
\begin{figure}
\includegraphics[scale=0.9]{SMary_process.png}
\caption{Wastewater processing as employed by St Mary's treatment plant \cite{stmarys}}
\end{figure}

\subsection{\emph{N}-acyl-L-homoserine lactone mediated gene expression facilitates bacterial communication and co-operation}
The principle of bacterial communication and behavioural change in response to extracellular molecules, for example in biofilms \cite{webb2003}. These molecules diffuse in and out of the cells and trigger a change in transcriptomic regulation depending on their concentration. This allows the consortia, whether multi-species or cross-kingdom \citep{williams2007quorum}, to engage in a concerted effort that resembles multi-cellularity \cite{kjelleberg2002}. While there are several bacterial communication systems, this report focuses on \emph{N}-acyl-L-homoserine lactone (AHL) dependent communication only. The bacteria that actively participate in this style of communication are predominately from the phylum \emph{proteobacteria}, which are diverse and well represented in the environment and especially so in municipal activated sludge \cite{Hesham_11,Wagner_02}.  


The machinery facilitating AHL-mediated communication consists of two major constituents - LuxR and LuxI, or their respective homologs. The latter is an AHL synthase, while the former is a receptor that binds AHLs at a threshold. The complex functions as a transcription factor for \textit{luxR} and \textit{luxI}, or their homologs, and species specific functional genes. These genes typically play a role in bacterial cell adhesion to surfaces, biofilm formation and the expression of extracellular enzymes \cite{Flemming_10}.
These traits are usually under further inhibiotion and activational control outside of AHL-mediated transcription factors as well as potential cross regulation between several AHL systems within the organism \cite{juhas2005}.


\subsubsection{\emph{AHL responsive gene circuits are found in Proteobacteria}}
In 2008 Case et al. compiled a list of isolates containing \emph{luxR} and \emph{luxI} homologs from 512 completed genomes on the NCBI platform, where 13 \% of those contained both homologs. These were exclusively from the phylum \emph{Proteobacteria} and were found in 26 \% of all finished proteobacterial genomes \cite{case_08}.
The type II secretion system is only found in this phylum, usually associated with delivering quorum communication virulence factors and enzymes, such as lipase, into extracellular space \cite{sandkvist2001}. 
% explain lux box
When assessing the frequency in which Case et al. found \emph{luxR} and \emph{luxI} in the \emph{Proteobacteria} \cite{case_08}, coupled with the prevalence of this phylum in activated sludge \cite{Wagner_02,Hesham_11}, it follows that this system is relevant in activated sludge flocs. 
%This is reflected in the change of enzyme expression in sludge isolates in response to AHL stimulation as detected by Chong et al \cite{Chong_12}. 

\subsection{Lipids in wastewater need to be degraded by lipase}
Municipal wastewater contains lipids, at a concentration between 40 and 100 mg/m$^{3}$ \cite{Forster_92} representing 31 \% of the chemical oxygen demand of domestic wastewater \cite{Raunkjaer_94}. Oleic acid is the most common fatty acid found in native as well as used olive and sunflower oils, which are commonly used in domestic settings \cite{haba2000isolation}.  An excess in lipid content has been shown to inhibit flocculation and promote the growth of filamentous bacteria, which are linked to sludge bulking - an event that induces foaming and reduces effluent quality \cite{Forster_92}.
%statement of lipase in lipid degr in sludge

Due to the hydrophobic nature of lipids, they may be associated with other particulates in the activated sludge or form a separate phase. In the former case, membrane or EPS bound lipases are likely expressed for degradation, whereas lipids in a separate phase could be targeted with extracellular lipases. It is advantageous for consortia to express lipase to utilise this carbon source and and lipase is commonly expressed in activated sludge \cite{gessesse2003lipase}. 
%Investigations as to the most effective strategy to remove lipids in wastewater have been conducted extensively. Strategies including pretreatment with large concentrations of lipases, introducing pure or mixed lipid degrading cultures and exposure activated sludge \cite{Wakelin_97}. %Activated sludge showed the highest grease removing activity - non funct \cite{Wakelin_97}. 
%need more refs - extensive doesnt equal singular ref.


\subsubsection{\emph{Lipase are extracellular degradation units}}
Triacylglycerol acylhydrolase (Enzyme Class 3.1.1.3) catalyses the reversible hydrolysis of triacylglycerols by targeting the ester bonds that attach the fatty acid side chain to the glycerol backbone. The model lipid Glyceryl trioleate (GT) used in this study consists of three oleic acid side chains. Genes encoding for lipase will be denoted as \emph{lip}.
The activity of lipase is highly chemo-, enantio- and regioselective. Common to this class of enzyme is an extruding loop which extends over the active site, called the lid. The aggregated, hydrophobic nature of the substrate causes interfacial activation, which causes the lid to move to reveal the underlying active site to the substrate \cite{derewenda1992,van_Tilbeurgh1993}. These features contribute to lipases being highly efficient biocatalysts in organic chemistry and their extensive representation in the industrial setting. 
They are found within the textile, detergent, food processing, leather, pharmaceutical, pulp and paper industries \cite{hasan_06}; are essential for the production of fine chemicals such as flavours, cosmetics, agrochemicals and therapeutics \cite{jaeger2002}; and are under investigation for their potentially to produce biodiesel as a replace fossil fuels \cite{hasan_06,iso2001}. 


Microbial lipases are secreted into the extracellular space, where they can catalyse the liberation of fatty acids at the lipid-water interface, which are then absorbed and utilised as a C source. In Gram negative bacteria, including \emph{proteobacteria}, the lipase zymogen passes two membranes separated by the periplasm, before secretion and folding into the active conformation \cite{bos2007,michel2009}. There are two pathways by which this can occur: the type I or type II secretory pathways. 


\subsubsection{\emph{AHL mediated gene expression of lipase in proteobacteria is convoluted}}
The influence of AHL mediated gene expression on lipase production has been documented in several genera, discussed in this section with their respective classification and \emph{luxRI} homologs as can be seen in Table 2.

\begin{table}
\begin{tabular}{ | p{2.5cm} | p{3cm} | p{1.5cm} | p{1.5cm} | p{2.5cm} | }
\hline
Species & Taxonomy (class, order) & \emph{luxR} homologue & \emph{luxI} homologue & Reference/ GenBank accession \# \\
\hline
\emph{Burkholderia cepacia} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{cepR} & \emph{cepI} & \cite{lewenza1999} \\
\hline
\emph{Burkholderia kururiensis} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{braR} & \emph{braI} & \cite{suarez2008} \\
\hline
\emph{Burkholderia unamae} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{unaR} & \emph{unaI} & \cite{suarez2010} \\
\hline
\emph{Burkholderia xenovorans} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{“luxR homologue”} & - & NC007951.1 \\
\hline
\emph{Burkholderia glumae} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{tofR} & \emph{tofI} & AB757840.1 \\
\hline
\emph{Burkholderia vietnamiensis} & \emph{$\beta$-proteobacteria, Burkholderiales} & \emph{} & \emph{} & \cite{conway_02}, \cite{ulrich2004}
 \\
\hline
\emph{Pseudomonas aeruginosa} & \emph{$\gamma$-proteobacteria, Pseudomonadales} & \emph{} & \emph{} & \cite{juhas2005} \\
\hline
\emph{Pseudomonas fluorescens} & \emph{$\gamma$-proteobacteria, Pseudomonadales} & \emph{phzR} & \emph{phzI} & L48616 \\
\hline
\emph{Xenorhabdus nematophilus} & \emph{$\gamma$-proteobacteria, Enterobacteriales} & \emph{“luxR homologue”} & - & FN667742.1 \\
\hline
\emph{Serratia marcescens} & \emph{$\gamma$-proteobacteria, Enterobacteriales} & \emph{smaR} & \emph{smaI} & AJ5980 \\
\hline
\emph{Serratia protemaculans} & \emph{$\gamma$-proteobacteria, Enterobacteriales} & \emph{sprR} & \emph{sprI} & AY040209.1 \\
\hline
\end{tabular}
\caption{The taxonomic classification and \emph{luxRI} homologs for species shown to have AHL mediated influence on lipase expression.}
\end{table}

\emph{\underline{Burkholderia spp.}} 
\\In \emph{Burkholderia cepacia}, Lewenza et al. have demonstrated that the expression of \emph{lipA} is linked to CepR, the LuxR homolog \cite{lewenza1999}, which is in direct opposition to the results obtained by Huber et al. \cite{huber2001}. In Lewenza et al.'s study, lipase production in \emph{cepR} mutants was reduced by up to 45 \%. However the supplementation of CepR via a plasmid did not restore wild type lipase expression, which suggests that the link between \emph{cepR} and \emph{lipA} is other than transcriptomal activation. The lack of a \emph{lux} box within \emph{lipA}, to which the CepR:AHL transcription factor would bind, supports the postulation that \emph{lipA} is situated downstream of \emph{cepR} and under same operon control \cite{lewenza1999}. 



\emph{B. glumae}, an emerging rice pathogen, regulates it's lipase production with AHL mediated gen expression and lipase expression is innately linked with the pathogenic phenotype. TofR, the LuxR homolog, activates \emph{lipA} transcription \cite{devescovi_07}. 


A study by Suarez-Moreno et al. which compared the plant associated species \emph{B. kururiensis}, \emph{B. unamae} and \emph{B. xenovorans} found no discernible impact of disabling the respective \emph{luxR} homologs on lipase production, in agreement with Huber et al. \cite{huber2001,suarez2010}. For these species tested, which live in symbiosis with the plant host, it follows that lipase expression, which is generally considered a virulent attribute, should not be population density dependent as increasing cell density would imply successful symbiosis. 



In \emph{B. vietnamiensis}, the literature on how AHLs influence lipase secretion are contradictory. Conway et al. recorded no detectable influence of AHLs on lipase production whether in the wild type or an AHL synthase (\emph{bviI}) deficient mutant \cite{conway_02}. On the other hand Ulrich et al. conducted a they found three separate AHL synthases referred to as \emph{btaI123} and five separate \emph{luxR}-like transcriptional regulators named \emph{btaR12345}, which is a nomenclature system distinct to other nomenclature reported here. They found that \emph{btaR1}, \emph{btaR3}, \emph{btaR4} and \emph{btaR5} acted as repressors on lipase production \cite{ulrich2004}. \emph{btaI1} and \emph{btaI3} inhibit lipase production while \emph{btaI2} enhances it. Even though the data in regard to lipase activation and repression implies that \emph{luxR123} and \emph{luxI123} represent complete circuit systems, the evidence was not compelling to Ulrich et al. to draw that conclusion without further investigation \cite{ulrich2004}.
It is unclear whether the studies by Conway et al. and Ulrich et al. refer to the same AHL-mediated systems, if \emph{bviRI} and \emph{btaRI} are synonymous. If this is the case, a possible explanation for Conway et al. not detecting any impact of the \emph{bviR} mutant on lipase production could be given by them investigating what Ulrich et al. termed \emph{btaR2}, in which they also reported no discernible impact on lipase production. 
The complicated and cross-regulatory system surrounding lipase expression within the \emph{Burkholderia} genus may indicate the complexity of lipase production in other \emph{proteobacteria}, including the role that AHLs play on the expression.
\\
\\ \emph{\underline{Pseudomonas spp.}}
\\ \emph{P. aeruginosa} has two AHL dependent systems: \emph{las} and \emph{rhl} which each represent \emph{lux} homologs and respond to separate AHLs. The LasR:AHL complex controls \emph{rhlIR} expression, while RhlR activates \emph{lipA} transcription. Hence RhlR directly influences lipase production while LasR's influence is indirect. However a third non-AHL QS system, called the P. aeruginosa quinilone signal, regulates the other two systems \cite{juhas2005}. 


The AHLs are also influenced by other factors such as GacA, which promotes the AHL regulatory cascade \cite{reimmann1997}, as well as RsmA which degrades AHLs \cite{pessi2001}. However even though RsmA degrades the AHL signalling molecule, it positively affects lipase production as it binds to the \textit{lip} mRNA and stabilises the transcript to reduce the transcript's rate of degradation. RsmA in turn is regulated by RsmZ, whose suppression of RsmA therefore negatively affects lipase production \cite{heurlier2004}. 
\\
\\ \emph{\underline{Xenorhabdus nematophilus}} 
\\The insect pathogen \emph{Xenorhabdus nematophilus} resides in the intestinal tract of the symbiotic host \emph{Steinernema carpocapsae}. It aids the host in reproducing while the nematode represents a reservoir for the bacterium to infect insects \cite{herbert2007}. Lipase production is upregulated by  by AHL mediated transcriptional activation \cite{dunphy_97} if \emph{flhDC}, which is the flagella master operon, is also expressed \cite{rosenau2000}. Another regulator is Lrp, which controls LrhA, and positively impacts lipase production and motility \cite{richards2008} 
The link between lipase expression, motility and virulence is established \cite{givaudan_00}, which suggests that the lipase contributes to the feeding mechanism while infecting rather than being a virulent agent itself \cite{richards2010}. Considering the organism's life style, which involves the necessity to leave the host in order to access the target, the controls associated with lipase expression are consistent with the requirement to move to attack the food source before releasing extracellular enzymes.
\\
\\ \emph{\underline{Serratia spp.}} 
\\Lipase production in both \emph{S. marcescens} and emph{S. proteamaculanss} is influenced by AHL-mediated gene expression \cite{horng2002,shibatani2000,christensen_03}. Furthermore, in \emph{Serratia marcescens} surface attachment and biofilm formation in  is influenced by the same mechanism \cite{labbate2007} and all contribute to the virulent phenotype \cite{hejazi_97}.

\newpage
\section{Aims}
This project explores the hypotheses that 
\begin{itemize}
\item Glycerol trioleate serves as a surface substratum for activated sludge floc formation;
\item AHL-mediated gene expression regulates lipase production in activated sludge flocs.
\end{itemize}

\noindent
Specifically we aim to

\begin{enumerate}
\item Investigate lipid morphology in activated sludge;
\item Monitor change in floc community in the presence of lipids;
%\item quantify to which extent AHLs partition into lipids;
\item Isolate and identify lipid degrading bacteria in activated sludge that produce AHLs.
\end{enumerate}
To this end the model lipid GT was incubated in activated sludge.
\newpage
\section{Methods}

\subsection{Enrichment cultures}
Enrichment cultures consisted of 50 ml AS collected from St. Mary's wastewater treatment plant in each of eight identically proportioned 250 ml Boecko flasks. The experiment was conducted in quadruplicates of 2 \% GT enrichment (Glyceryl trioleate, Sigma-Aldrich T7140-10G) and no treatment. The cultures were incubated at $37\,^{\circ}\mathrm{C}$ on a shaker set at 140 rpm. Samples were taken over a 25 day period and consisted of 0.5 ml for Deoxyribonucleic Acids (DNA) extraction and 0.5 ml for lipase assays. The 1 ml removed was replaced with sterile, syringe filtered (0.22 $\mu$m Millipore) activated sludge supernatant. The trial enrichment cultures were enriched with 1 \% GT rather than 2 \%.

\subsubsection{\emph{Microscopy}}
Lipid droplets were removed with tweezers and washed in PBS in two subsequent crystallizing dishes. After transferral to a glass slide, the droplet was stained with 5 $\mu$l SybrGreen, a DNA binding dye, and covered with a cover slide. The droplets were imaged with an Olympus Fluorescence Microscope which elicits a visual response from DNA bound dye.

\subsubsection{\emph{DNA extraction}}
DNA was extracted by the amended xanthogenate-SDS method \cite{tillett2000xanthogenate}. The samples for DNA extraction were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube, supernatant removed and the pellets frozen . In the case of DNA extraction from GT droplets, 0.5 ml from GT enrichment cultures were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube and the pellet was decanted into sterile MilliQ water in a crystallizing dish. Semi-solid lipid droplets were removed with tweezers and washed in a second crystallizing dish, transferred to a 1.5 ml eppendorf tube and frozen. Thawed pellets were re-suspended in 0.2 ml phosphate buffered saline (PBS, as per Appendix 12.4).


For each sample, 0.9 ml phenol solution and 0.9 ml XS-buffer (as per Appendix 12.8) was  heated to $70\,^{\circ}\mathrm{C}$ in a 2 ml microcentrifuge tube in a water bath for 5 min. The 0.2 ml cell suspension was added to the 2 ml tubes, mixed by inversion and left at $70\,^{\circ}\mathrm{C}$ for 15 min. Then the samples were vortexed for 10 sec and frozen for 2 min, followed by centrifugation at 16.1 rcf for 5 min. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 0.9 ml phenol:chloroform:isoamyl alcohol mix and centrifuged at 16.1 rcf for 5 min. This step was repeated. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 1 ml ice cold isopropanol, 50 $\mu$l 3 M sodium acetate pH 7.5 and 1.3 $\mu$l glyco blue, mixed by inversion and frozen over night. Subsequent centrifugation at 16.1 rcf for 60 min at $0\,^{\circ}\mathrm{C}$ was followed by removal of isopropanol and addition of 1 ml 80 \% ethanol. Centrifugation at 16.1 rcf for 20 min at $0\,^{\circ}\mathrm{C}$ followed, then ethanol was removed and samples were left to air dry for 30 min before re-suspension in 0.1 ml molecular water.
\\ 
Extracted DNA concentration was analysed via NanoDrop (NanoDrop ND-1000 Spectrophotometer) as well as Qubit (Invitrogen, Qubit 2.0 Flourometer), as per standard protocol for broad range DNA detection.

\subsubsection{\emph{Lipase assay}}
The lipase assay was adapted from Christensen et al \cite{christensen_03}. From each replicate 0.5 ml was taken and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and syringe sterilised (0.22 $\mu$m Millipore filter) to test for suspended extracellular lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water with the addition of 0.1 ml Zirconium beads. The samples were bead beaten for three 45 sec cycles and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and and syringe sterilised (0.22 $\mu$m Millipore filter) to test for EPS associated lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water to test for membrane bound lipase.


\begin{figure}
\begin{center}
\includegraphics[width=.65\textwidth]{lipase_assay.png}\\
\includegraphics[width=.65\textwidth]{lipase_assay2.png}
\caption{Fractionation of each sample into three fractions, each subjected to lipase activity assay.}
\end{center}
\end{figure}

For each sample 0.1 ml were incubated with 0.9 ml of  p-nitrophenyl palmitate (p-npp) containing substrate solution (as per Appendix 12.2) for one hour. Lipase cleaves the p-nitrophenyl group off the plamitate which can be quantified at an absorbance of 410. The p-nitrophenyl concentration is directly correlative with lipase activity. The reaction is terminated by alkaline pH inactivation of lipase with 1 ml 1 M Sodium Carbonate. Samples were centrifuged at 16.1 rcf for 5 min. From each terminated reaction, 0.3 ml supernatant was transferred to 96 well micro titre plate and the absorbance read at a wavelength of 410 with a Microtitre reader (Molecular Devices, Spectra Max 340).

\subsection{community analysis of enriched cultures}
DNA was extracted from enrichment culture isolates, verified with DGGE polymerase chain reaction (PCR). The DGGE PCR assay was set up with the primers 357FGC and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 15.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 2 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.
Bands of interest were selected from the DGGE and excised and re-run on DGGE PCR to test their integrity, and if satisfactory underwent sequencing PCR. 

\subsubsection{\emph{Further enrichment}}
To investigate the change of microbial composition with long term enrichment, secondary (t2) and tertiary (t3) enrichments were conducted. Further enrichments with aggregate supernatant or emulsion were transferred at 5 \% to 25 ml activated sludge syringe filtered (0.22 $\mu$m Millipore filter) supernatant with 2 \% GT. The aggregate was transferred to 25 ml activated sludge syringe filtered (0.22 $\mu$m Millipore filter) supernatant with 2 \% GT. 


Also to investigate the difference in behaviour of activated sludge in the presence of 1 vs. 2 \% GT, a comparative enrichment was conducted with DNA extracted after 7 days and processed for DGGE and sequencing.
 
\subsubsection{\emph{Denaturing Gradient Gel Electrophoresis}}
DGGE was conducted using the BioRad DCode system and protocol for 50 \% to 70 \% denaturing gradient on a 6.7 \% arcylamide gel. Electrophoresis was conducted at 75V and $60\,^{\circ}\mathrm{C}$ for 16.5 hrs in 1x TAE. The gel was stained with SybrGold () for 20 min and imaged on a BioRad Gel Doc XR+ Imaging system and BioRad Image Lab software. Bands of interest were excised and suspended in 40 $\mu$l molecular water over night, which used as template for nested PCR set up with 357F and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 12.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 5 $\mu$l template. The PCR protocol was the following:  min at $30\,^{\circ}\mathrm{C}$, then  cycles of denaturing phase:    2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$. Products for sequencing were cleaned up with Zymo Research DNA Clean \& Concentrator-5 kit, following the PCR product protocol.

\subsubsection{\emph{Sequencing}}
The sequencing PCR assay, protocol from the Ramaciotti centre, was set up for the 357F primer at 20 $\mu$l as follows: 1 $\mu$l BigDye terminator V3.1, 20 - 50 ng PCR product, 0.32 $\mu$l primer, 3.5 $\mu$l 5x sequencing buffer and made up to 20 $\mu$l with molecular water. The PCR protocol was the following: 26 cycles of denaturing phase: 10 sec at $96\,^{\circ}\mathrm{C}$, annealing phase:  at 5 sec $50\,^{\circ}\mathrm{C}$, extension phase: 4 min at $60\,^{\circ}\mathrm{C}$, followed by and storage at $4\,^{\circ}\mathrm{C}$.

The samples were cleaned up by addition of 5 $\mu$l of 125 mM EDTA and 60 $\mu$l 100 \% ethanol followed by vortexing and 15 min precipitation period. The samples were then spun at 14000 rcf for 20 min and supernatant was removed. After addition of 160 $\mu$l fresh 70 \% ethanol, samples were spun at 14000 rcf for 10 min and supernatant discarded. This step was repeated with addition of 80 $\mu$l 70 \% ethanol. The samples were dried in the dark and submitted to the Ramaciotti centre at UNSW for Sanger Sequencing.

\subsection{Lipid coloniser isolation}
Sludge isolates  were isolated from lipid droplets, washed twice in PBS, by three subsequent transfers on Lipid coloniser medium (LCM as per Appendix 12.3) plates. Isolates were prepared for 16S rRNA PCR for the primers 27F and 1492R at 20 $\mu$l as follows: 10 $\mu$l EconoTaq Master mix, 7.4 $\mu$l molecular water, 0.8 $\mu$l of each primer and 1 $\mu$l template. The PCR protocol was the following: 2 min at $94\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 1 min at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $59\,^{\circ}\mathrm{C}$, extension phase: 1.40 min at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$ submitted for Sanger sequencing by the Ramaciotti centre as previously outlined.
% incl diff primers and hene length, PCR protocol.

\subsubsection{\emph{Lipid coloniser behaviour in the presence and absence of olive oil}}
Isolates were pre-cultured in 25 ml LCM broth, either with or without olive oil (Mike - should i just exclude here that I ran out of GT?). 

\subsubsection{\emph{Screening lipid colonising isolates for AHL production}}
Pre-cultures were centrifuged at 16.1 rcf for 10 min and 20 ml supernatant was transferred to a 50 ml falcon tube. After addition of 20 ml Ethyl acetate with 0.01 \% glacial acetic acid, the mixture was vortexed and left to separate. The top  layer was transferred to an 100 ml beaker. This was repeated twice and left over night for evaporation in a chemical fume hood. The AHLs were resuspended in 100  $\mu$l methanol and filter sterilised with PVDF Filter Vials (CP-ANALYTICA GmbH). 
 The reporter strain \emph{Escherichia coli} carrying the plasmid pJBA357 was pre-cultured in Lysogeny Broth (LB10, as per Appendix 12.1) \cite{bertani1951studies}. The plasmid contains \emph{gfp}, which  is preceded by the \emph{luxR} promoter, activated by different types of AHLs and the response is directly proportionate to the AHLs present.
The assay was conducted in triplicate with \emph{Aeromonas hydrophila} GC1 as the positive control, LB10 as a blank and 2.5 nm, 1 nm and 0.5 nm OHHL as a comparative reference. Per microtitre plate well 20 $\mu$l of extract was added and the methanol left to evaporate. The reporter strain was incubated overnight in LB10 with ampicillin (50ul/ml) shaking with 150 rpm at $30\,^{\circ}\mathrm{C}$ and was diluted 20 times in fresh LB10 prior to addition into microtitre plate.  After 4 hrs of incubation OD600 was read to verify \emph{E. coli} growth as well as fluorescence by excitation at 485 nm and then emission at 535 nm.

%The PCR assay was set up for $\mu$l as follows: $\mu$l EconoTaq Master mix, $\mu$l molecular water, $\mu$l of each primer and $\mu$l template. The PCR protocol was the following:  min at $30\,^{\circ}\mathrm{C}$, then  cycles of denaturing phase:   at $30\,^{\circ}\mathrm{C}$, annealing phase:  at  $30\,^{\circ}\mathrm{C}$, extension phase:  at $30\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.
\newpage
\section{Results}

\subsection{Bacteria colonise lipids in activated sludge}
It was essential to establish an experimental system and adapt protocols to observe the impact of lipids on activated sludge before a controlled enrichment experiment could be conducted. 

\subsubsection{\emph{Change in activated sludge morphology in response to lipid exposure}}
A trial run preceded the actual enrichment run. The trial was conducted with 1 \% GT  addition to sludge replicates and sampling frequency was every 7 days as the expectation was that the enrichment would run for an extended period of time. GT formed a transparent and hydrophobic layer on the surface of the enrichment cultures, which was replaced with white droplets suspended below the culture surface within 7 days (Figure 3B). While varying in size, the droplets consistently decreased in diameter with increased incubation period until the they were indistinguishable from the sludge after 21 days. Droplet formation decreased floc settlement for the duration of their presence. 
Due to the lipid being assimilated into the sludge within the first 3 sampling points, the sampling frequency was increased from 7 to every 3 days. To compensate, the GT \% was increased from 1 to 2 with the effect of droplet formation no longer taking place. Instead the cultures became increasingly viscous and lighter in colour (Figure 3A), which prevented sludge settleability (Figure 3C). Aggregates developed within the viscous enrichment cultures, which were transferred fro t2 enrichment (Figure 3D).  
\begin{figure}
\includegraphics[scale=.77]{cultures.jpg}
\caption{Impact of lipid addition to activated sludge cultures; A) the comparison between 2 \% GT enrichment and no treatment; B) lipid droplet formation in the presence of 1 \% GT in trial run; C) the lack of settleability of sludge after incubation with 2 \% GT; D) secondary enrichment by transferral of aggregate into sludge supernatant and GT.}
\end{figure}


\subsubsection{\emph{Microscopy}}
 Droplets were isolated and stained with SybrGreen before imaging with epifluorescence microscopy (Figure 4). The fluorophore binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black.
Biomass association with lipid droplets was evident (Figure 4A -F). As lipid droplets were washed twice before staining and imaging, biomass was firmly attached to the droplets.

\begin{figure}
\includegraphics[scale=.4]{august_microscopy.png}
\caption{Imaging of lipid droplets post SybrGreen staining with fluorescence microscopy. A); B); C); D); E) and F) demonstrate the close association of biomass with lipid compared to the intermediate liquid phase.}
\end{figure}




\subsubsection{\emph{Monitoring lipase production}}
Lipase activity was detected upon colorimetric change from p-npp cleavage in the substrate solution. The assay separated lipase suspended in the supernatant, EPS bound and cell membrane bound lipases from each enrichment and Figure 5 shows the total average lipase activity, with the enriched cultures displayed a higher activity than the control replicates.
                                                                                                                                                                                                                                         

\begin{figure}
\includegraphics[scale=1]{lip_av.PNG}
\caption{Total lipase activity of all fractions and replicates}
\end{figure}

The lipase activity in the supernatant fraction of the enriched cultures spikes between 3 and 9 days as well as 15 to 18 days in comparison to the untreated activate sludge where the lipase activity remains at a consistent level (Figure 6).

\begin{figure}
\includegraphics[scale=1.1]{lip_sn.PNG}
\caption{Average lipase activity in the supernatant fraction}
\end{figure}

An immediate increase of lypolytic activity in the EPS  fraction within the first 3 days is evident from Figure 7. Within 9 days the lipase activity in this fraction drops below the initial activity. At this time point the activty within the control cultures exceeds the enrichment cultures, which decreases rapidly and remains below the enrichment cultures lipase activity for all other time points.

\begin{figure}
\includegraphics[scale=1.1]{lip_eps.PNG}
\caption{Average lipase activity in the EPS fraction}
\end{figure}

It is evident from  Figure 8 that membrane bound lipase is the most abundant across both control and enrichment cultures, where the average activity in the control replicates increased in comparison to the enriched cultures between days 9 to 22.

\begin{figure}
\includegraphics[scale=1.1]{lip_biom.PNG}
\caption{Average lipase activity in the biomass fraction}
\end{figure}

\subsection{Lipid colonising isolates behave differently in the presence of lipids}
% change in morphology in LC1 and LuxR data

LC1 - LC5 were isolated from lipid droplets in 1 \% GT enrichment trial run. These were cultured on a minimal medium, with and without lipids. LC1 cultured in the presence of lipid displays a pink phenotype (Figure 9A) and appears white when cultured without lipids (Figure 9B).
This further led to the question whether AHL profiles are affected by these culture conditions as well as morphology. This was addressed by using LuxR bioassay to determine AHL mediated activation (Figure 10). LC1 exhibits minimal AHL activity detectable by LuxR, while LC4 displays no activity. LC5 elicits the highest response form the biosensor, marginally more so when cultured in the presence of lipids. LC2 and LC3 both evoke a higher LuxR responses when cultured in the presence of lipid than without.

\begin{figure}
\includegraphics[scale=.65]{LC1_comp.png}
\caption{Lipid colonising isolate LC1 cultured in the presence (A) and absence (B) of lipid in LCM medium}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{LCM_LuxR.png}
\caption{Response to LuxR bioassay by lipid colonising isolates, comparatively cultured in the presence and absence of lipid }
\end{figure}

Sequencing of lipid colonising isolates resulted in identification of LC1 - LC4 with NCBI BLAST (Table 3), while LC5 was not identifiable as only 6.7 \% of the bases in the chromatogram of high enough quality to compare to the NCBI BLAST and taxonomic databases.
	
\begin{table}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
Isolate & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 &  \emph{Pandoraea} sp. (KF378759.1) & \emph{$\beta$-proteobacteria} & 2e$^{-79}$ \\
\hline
2 & Uncultured \emph{Achromobacter} sp. (KF448091.1) & \emph{$\beta$-proteobacteria} & 1e$^{-139}$ \\
\hline
3 & \emph{Enterobacter} sp. (KF411353.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Pseudomonas} sp. (KC822768.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
\end{tabular}
\caption{Sequencing results for 16S rRNA fragments of lipid colonising isolates 1 - 4, the sequences used for identification can be found in Appendix 12.5}
\end{table}

\subsection{Microbial community changes in the presence of lipids}
The PCR products of 16S DGGE PCR  of replicate 4 were run on DGGE (Figure 11) where the numbered bands excised and successfully amplified, sequenced and identified with NCBI BLAST and taxonomic databases (Table 4). 

\begin{figure}
\includegraphics[scale=2.7]{DGGE_R4_450bp_thesis.jpg}
\caption{Community structure of control and enriched cultures of replicate 4, from day 0 to day 25 on DGGE.}
\end{figure}

\begin{table}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1   &  \emph{Bacteroidetes} (JX473581.1) & - & 8e$^{-45}$ \\
\hline
2  & \emph{Bacillus} sp. (GU271888.1) & \emph{Bacilli} & 5e$^{-70}$ \\
\hline
3 & Uncultured \emph{Dechloromonas} sp. (JQ012310.1) & \emph{$\beta$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Azospira oryzae} (KF260987.1) & \emph{$\beta$-proteobacteria} & 1E$^{-50}$ \\
\hline
5 & Uncultured \emph{Xanthmonadales} (KC588330.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
6 & Uncultured \emph{Dechloromonas} sp. (KF003189.1) & \emph{$\beta$-proteobacteria} & 4e$^{-103}$ \\
\hline
7 & \emph{Novosphingobium} sp. (KF544940.1) & \emph{$\alpha$-proteobacteria} & 1e$^{-173}$ \\
\hline
8 & \emph{Novosphingobium} (KF544932.1) & \emph{$\alpha$-proteobacteria} & 4e$^{-61}$ \\
\hline
9 & \emph{Sphingomoas} sp. (AY521009.2) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
10 & \emph{Sphingomonas suberifaciens} (AY521009.2) & \emph{$\alpha$-proteobacteria} & 3e$^{-119}$ \\
\hline
11 & \emph{Sphingomonas} sp. (JQ928361.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$ \\
\hline
12 & \emph{Roseomonas} sp.  (KF254767.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-65}$ \\
\hline
\end{tabular}
\caption{Sequencing results for 16S rRNA DGGE fragments of replicate 4, the sequences used for identification with NCBI BLAST can be found in Appendix 12.6}
\end{table}

The PCR products of 16S DGGE PCR  of various further enrichments were run on DGGE (Figure 12) where the numbered bands excised and successfully amplified and sequenced (Table 5).

\begin{figure}
\includegraphics[scale=1.2]{DGGE_misc_450bp_thesis.jpg}
\caption{Community structure of aggregates, emulsions, secondary and tertiary enrichments on DGGE.}
\end{figure}

\begin{table}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 & \emph{Nevskia} sp. (GQ845011.1) & \emph{$\gamma$-proteobacteria} & 0.0  \\
\hline
2 & \emph{Sphingomonas} sp. (KC172307.1) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
3 & \emph{Sphingobium} sp. (KF437579.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-74}$ \\
\hline
4 & \emph{Sphingomonas} sp. (KF544924.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
5 & \emph{Rhodovarius lipocyclicus} (NR\_025629.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-114}$ \\
\hline
6 & \emph{Xanthobacter} sp. (AB847934.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$  \\
\hline
7 & \emph{Xanthobacter} sp. (AB245351.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-45}$  \\
\hline
8 & \emph{Sphingomonas} sp.(KF551133.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-100}$  \\
\hline
9 & \emph{Bradyrhizobium} sp. (JX505076.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-165}$  \\
\hline
10 & \emph{Sphingomonas} sp. (EF636068.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
11 & Candidatus \emph{Competibacter} sp. (JQ480426.1) & \emph{$\gamma$-proteobacteria} & 1e$^{-123}$  \\
\hline
12 & \emph{Sphingomonas} sp. (HE974351.1) & \emph{$\alpha$-proteobacteria} &  1e$^{-148}$ \\
\hline
13 & \emph{Stakelama pacifica} (HE662817.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-77}$  \\
\hline
14 & \emph{Oleomonas sagaranensis} (AJ784808.1) & \emph{$\alpha$-proteobacteria} & 6e$^{-101}$  \\
\hline
\end{tabular}
\caption{Sequencing results for 16S rRNA DGGE fragments of various further enrichments, the sequences used for identification with NCBI BLAST can be found in Appendix 12.7}
\end{table}

\newpage
\section{Discussion}

%\input{discussion}
\newpage
\section{Conclusions and Future directions}

%\input{conclusion}

\newpage 
%fix inclusion of refereces in table of contents
\bibliographystyle{plain}
\bibliography{thesis_ref.bib}

\newpage 
\section{Appendix}

\subsection{LB10 low yeast}
\begin{itemize}
\item 10 g bacto-tryptone
\item 1.25 g yeast extract
\item 10 g NaCl
\item 12 g agar optional
\end{itemize}

\subsection{Lipase assay substrate solution}
mix:
\begin{itemize}
\item 1 times volume of 0.3 \% (w/v) p-nitrophenyl palmitate in isopropanol;
\item 9 times volume of 0.2 \% (w/v) sodium deoxycholate and 0.1 \% (w/v) gum arabicum in 50 mM Phosphate buffer pH 8;
\end{itemize}

\subsection{Lipid coloniser medium}
\begin{itemize}
\item 0.5 g yeast extract
\item 0.5 g proteose peptone
\item 0.5 g starch
\item 0.3 g di-potassium phosphate
\item 0.024 g magnesium sulphate
\item 10 ml Glyceryl Trioleate
\item 15 g agar optional
\end{itemize}

\subsection{Phosphate buffered saline}
per litre of MilliQ water, pH adjusted to 7.4 and autoclaved:
\begin{itemize}
\item 8 g Sodium chloride
\item 0.2 g Potassium chloride
\item 1.44 g Disodium phosphate
\item 0.24 g Monopotassium phosphate
\end{itemize}

\subsection{Sequencing data of 16S rRNA DGGE fragments used to identify lipid colonising isolates}
\begin{enumerate}
\item GGGGGATGACGGTACCGGAAGAATAAGCACCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGTGCGCAGGCGGTTTTGTAAGACGGATGTGAAATCCCCGGGCTTAACCTGGGAACTGCATT
\item TGGGTTAATACCCCGTGAAACTGACGGTACCTGCAGAATAAGCACCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGTGCGCAGGCGGTTCGGAAAGAAAGATGTGAAATCCCAGAGCTTAACTTTGGAACTGCATTTTTAACTACCGGGCTAGAGTGTGTCAGAGGGAGGTGGAATTCCGCGTGTAGCAGTGAAATGCGTAGATATGCGGAGGAACACCGATGGCGAAGG
\item TAATAACCTTGTCGATTGACGTTACCCGCAGAAGAAGCACCGGCTAACTCCGTGCCAGCAGCCGCGGTAATACGGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCACGCAGGCGGTCTGTCAAGTCGGATGTGAAATCCCCGGGCTCAACCTGGGAACTGCATTCGAAACTGGCAGGCTAGAGTCTTGTAGAGGGGGGTAGAATTCCAGGTGTAGCGGTGAAATGCGTAGAGATCTGGAGGAATACCGGTGGCGAAGGCGGCCCCCTGGACAAAGACTGACGCTCAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGTCGACTTGGAGGTTGTGCCCTTGAGGCGTGGCTTCCGGAGCTAACGCGTTAAGTCGACCGCCTGGGGAGTACGGCCGCAAGGTTAAAACTCAAATGAATTGAC
\item TTGTAGATTAATACTCTGCAATTTTGACGTTACCGACAGAATAAGCACCGGCTAACTCTGTGCCAGCAGCCGCGGTAATACAGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCGCGTAGGTGGTTTGTTAAGTTGGATGTGAAATCCCCGGGCTCAACCTGGGAACTGCATCCAAAACTGGCAAGCTAGAGTATGGTAGAGGGTGGTGGAATTTCCTGTGTAGCGGTGAAATGCGTAGATATAGGAAGGAACACCAGTGGCGAAGGCGACCACCTGGACTGATACTGACACTGAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGTCAACTAGCCGTTGGGAGCCTTGAGCTCTTAGTGGCGCAGCTAACGCATTAAGTTGACCGCCTGGGGAGTACGGCCGCAAGGTTAAAACTCAAATGAATTGAC
\end{enumerate}

\subsection{Sequencing data of 16S rRNA DGGE fragments used to identify replicate 4 community members}
\begin{enumerate}
\item AACGCCGACCGCGAAGGCAGGGGGCGGGCCACCTACCGACGCTCATGCACGAAAGCGCGGGTATCGAACAGGATTAGATACCCTGGTAGTCCGCGCCGTAAACGATGATAGCTGGCCGTGCGGGAGCGATCCTGCGGGGCTGAGGGAAACCATTAAGCTATCCGCCTGGGGAGTACGCCCGCAAGGGTGAAACTC
\item AAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCTGTAAACGATGGGTACTCGGTGTCGCGGGTATCGACCCCTGCGGTGCCTTAGCTAACGCGTTAAGTACCCCGCCTGGGGAGTACGGTCGCAAGGCTGAAACTCAAAGG
\item TCGCATGGGTGAATACCCTGTGTGGATGACGGTACCGGAACAAGAAGCACCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGTGCGCAGGCGGTTTGGTAAGACAGGCGTGAAATCCCCGGGCTTAACCTGGGAACTGCGCTTGTGACTGCCAGGCTAGAGTACGGCAGAGGGGGGTGGAATTCCACGTGTAGCAGTGAAATGCGTAGAGATGTGGAGGAACACCAATGGCGAAGGCAGCCCCCTGGGTCGATACTGACGCTCATGCACGAAAGCGTGGGTAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCCTAAACGATGTCAACTAGGTGTTGGGTGGGTAAAACCATTTA
\item ATGACGGTACCCGCATAAGAAGCACCGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGTGCGCAGGCGGTT
\item AATAGCGCGCGGCTCTGACGTTACCCGCAGAATAAGCACCGGCTAACTCCGTGCCAGCAGCCGCGGTAATACGGAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGTGCGTAGGCGGTTCGGTCAGTCAGCCGTGAAAGCCCCGGGCTCAACCTGGGAACGGCGGTTGAGACGGCCGGACTAGAGTGGGCTAGAGGATCGTGGAATTCCCGGTGTAGCGGTGAAATGCGTAGAGATCGGGAGGAACACCGATGGCGAAGGCAGCGGTCTGGGGCCACACTGACGCTGAGGCACGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCGGTAAACGATGAGCACTAGACGTCGGGTGGGTGACCGTCCGGTGTCGCAGCTAACGCGCTAAGTGCTCCGCCTGGGGAGTAC
\item TAGAGTACGGCAGAGGGGGGTGGAATTCCACGTGTAGCAGTGAAATGCGTAGAGATGTGGAGGAACACCGATGGCGAAGGCAGCCCCCTGGGCCGATACTGACGCTCATGCACGAAAGCGTGGGTAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCCTAAACTATGTCAACTAGGTGTTGGGTGGGTAAAACCATTTAGTACCGTA
\item TGACAGTACCTGGAGAATAAGCTCCGGCTAACTCCGTGCCAGCAGCCGCGGTAATACGGAGGGAGCTAGCGTTGTTCGGAATTACTGGGCGTAAAGCGCGCGTAGGCGGTTACTCAAGTCAGAGGTGAAAGCCCGGGGCTCAACCCCGGAACTGCCTTTGAAACTAGGTGACTAGAATCTTGGAGAGGTCAGTGGAATTCCGAGTGTAGAGGTGAAATTCGTAGATATTCGGAAGAACACCAGTGGCGAAGGCGACTGACTGGACAAGTATTGACGCTGAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGT
\item GAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGATAACTAGCTGTCCGGGTACTTGGAGCTCGGGTGGCGCAGCTAACGCATTAAGTTATCCGCCTGGGGAGTACGGTCGCAAGATTAAAACTCAAA
\item GCTCTTTTACCCGGGATGATAATGACAGTACCGGGAGAATAAGCCCCGGCTAACTCCGTGCCAGCAGCCGCGGTAATACGGAGGGGGCTAGCGTTGTTCGGAATTACTGGGCGTAAAGCGCACGTAGGCGGCTTTGTAAGTTAGAGGTGAAAGCCCGGGGCTCAACCCCGGAATAGCCTTTAAGACTGCATCGCTTGAATCCAGGAGAGGTGAGTGGAATTCCGAGTGTAGAGGTGAAATTCGTAGATATTCGGAAGAACACCAGTGGCGAAGGCGGCTCACTGGACTGGTATTGACGCTGAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGT
\item GCTCTTTTACCCGGGATGATAATGACAGTACCGGGAGAATAAGCCCCGGCTAACTCCGTGCCAGCAGCCGCGGTAATACGGAGGGGGCTAGCGTTGTTCGGAATTACTGGGCGTAAAGCGCACGTAGGCGGCTTTGTAAGTTAGAGGTGAAAGCCCGGGGCTCAACCCCGGAATAGCCTTTAAGACTGCATCGCTTGAATCCAGGAGAGGTGAGTGGAATTCCGAGTGTAGAGGTGAAATTC
\item GGGAGAGGTGAGTGGAATTCCGAGTGTAGAGGTGAAATTCGTAGATATTCGGAAGAACACCAGTGGCGAAGGCGGCTCACTGGACCAGTATTGACGCTGAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGATAACTAGCTGTCCGGG 
\item AGTGTAGAGGTGAAATTCGTAGATATTGGGAAGAACACCGGTGGCGAAGGCGGCCACCTGGCTCGGTACTGACGCTGAGGCGCGACAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGT
\end{enumerate}

\subsection{Sequencing data of 16S rRNA DGGE fragments used to identify secondary and tertiary enrichment community members}
\begin{enumerate}
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\item 
\end{enumerate}

\subsection{XS buffer}
per litre of MilliQ water and autoclaved:
\begin{itemize}
\item 10 g Potassium ethyl Xanthogenate
\item 200 ml 4 M Ammonium acetate
\item 100 ml 1 M Tris-Hydrochloride pH 4
\item 40 ml 0.45 M EDTA
\item 50 ml 20 \% SDS
 
\end{itemize}


\end{document}