\subsection{Partitioning behaviour of glyceryl trioleate in activated sludge}

To establish an experimental model of floc formation based on interactions between lipids and bacteria in activated sludge it was first essential to examine the physical partitioning of lipids in sludge (Aim 1). Glyceryl trioleate (GT) was added to activated sludge samples in quadruplicate at 1 \% v/v. On addition GT formed a transparent and hydrophobic layer on the surface of the enrichment cultures, which was replaced with white droplets suspended below the culture surface within 7 days (Figure ,). While varying in size, the droplets consistently decreased in diameter with increased incubation period until they were indistinguishable from the sludge after 21 days. Droplet formation decreased floc settlement for the duration of their presence.
A second experiment was conducted with 2 \% GT added to activated sludge samples. The increased GT concentration did not result in lipid droplet formation. Instead the cultures became increasingly viscous and lighter in colour (Figure xA and xC) with lipid apparently adsorbed to the sludge flocs. No settlement of sludge flocs was observed, however large dense aggregates appeared in the viscous cultures. One of these was targeted for secondary enrichment by moving the aggregate into sludge supernatant, where the aggregate expanded and developed tendril like structures (Figure XD).

\begin{figure}
\includegraphics[scale=.77]{cultures.jpg}
\caption{Impact of GT addition to activated sludge cultures; A) the comparison between 2 \% GT enrichment and no treatment; B) lipid droplet formation in the presence of 1 \% GT; C) the lack of settleability of sludge after incubation with 2 \% GT; D) secondary enrichment by transferral of aggregate into sludge supernatant and 2 \% GT.}
\end{figure}
\FloatBarrier

\subsubsection{\emph{Association between glyceryl trioleate and activated sludge biomass}}
To examine the spatial relationships between biomass and lipids in 1 \% and 2 \% GT amended sludge samples (Aim 2), lipid droplets and aggregates were isolated and stained with SybrGreen before imaging with epifluorescence microscopy. Figure BB shows that the biomass is closely associated with lipid, rather than the intermediate water space and firmly attached as evidenced by the washing steps applied prior to staining as part of the isolation procedure. While Figure mA and mB show clusters of biomass around lipid, mC, mD, mE and mF are a close up of those clusters which showcase smaller segments of lipids surrounded and colonised by microbes. Owing to the depth of the samples, these images are unavoidably complex with multiple planes visible simultaneously in soft focus. This spatial distribution and cell density is reminiscent of surface associated biofilms.
The fluorescent stain binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black.
\begin{figure}
\includegraphics[scale=.4]{august_microscopy.png}
\caption{Imaging of lipid droplets post SybrGreen staining with fluorescence microscopy. A); B); C); D); E) and F) demonstrate the close association of biomass with lipid compared to the intermediate liquid phase.}
\end{figure}

\FloatBarrier
\subsubsection{\emph{Upregulation of lipase activity in response to glyceryl trioleate}}
For the floc community to utilise and hence remediate the GT in the sludge, lipase needs to be produced as lipase activity is directly proportional to the remediation potential of the sludge. It was expected that lipase production would be upregulated in he presence of GT and hence activity change with exposure time.
After addition of 2 \% (v/v) GT to activated sludge in quadruplicate, samples were separated into supernatant, EPS associated and membrane bound lipase fractions. \\

Lipase activity was detected upon colorimetric change from p-nitrophenyl palmitate cleavage in the substrate solution. 
Considerable variability was observed between replicates. Average lipase activity values for each fraction and time point are presented in Figures 5-8. Data from individual replicates is presented as supplementary information (Supplementary Material,  Figures c, x, z).
An immediate increase of lypolytic activity in the EPS fraction (Figure b) within the first 3 days is evident (p=..) with a gradual decrease in activity observed thereafter, dropping to approximately 30 \% of the peak activity at  the end of the incubation period. 
Conversely, the lipase activity in the supernatant fraction of the enriched cultures increased from day 3 onwards (p=..) (Figure m). The membrane bound lipase fraction displayed the highest lipase activity but did not differ between GT amended sludge samples and untreated controls (p=..) (Figure b). The enriched cultures displayed on average a higher total lipase activity (sum of the fractions) than the control replicates (Figure v).
                                                                                                                                                                                                                                         
\begin{figure}
\includegraphics[scale=1.1]{lip_eps.PNG}
\caption{Average activity of lipase immobilised in the EPS fraction of activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{lip_sn.PNG}
\caption{Average lipase activity in the supernatant fraction of activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{lip_biom.PNG}
\caption{Average activity of cell membrane bound lipase in activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls.}
\end{figure}


\begin{figure}
\includegraphics[scale=1]{lip_av.PNG}
\caption{Total lipase activity of all fractions and replicates in activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls.}
\end{figure}
\FloatBarrier

\subsection{Microbial community changes in the presence of lipids}
To identify bacteria potentially involved in lipid consumption bacterial community fingerprinting (DGGE) was used to monitor changes in activated sludge community composition in response to GT addition over time in one of the four replicates amended with 2 \% (v/v) GT (Aim 3). A reduction in intensity of bands in DGGE profiles over time suggests a decrease in relative abundance of bacterial lineages whilst increases in band intensity over time suggests increases in relative abundance. An increase in relative abundance in response to GT addition implicates specific bacterial lineages in lipid consumption.
Figure m shows DGGE profiles of GT amended and unamended control cultures over 25 days incubation. A clear shift in bacterial community composition is apparent in response to GT addition. Bands of interest (annotated in Figure n) representing distinct bacterial lineages, were excised, sequenced and matched with existing bacterial sequences in the NCBI database (Table m). Initially abundant Bacillus, Dechloromonas, Xanthomonadales, Bacteroidetes and Azospira lineages were replaced with Novosphingobium, Sphingomonas and Roseomonas lineages. These results implicate the latter three in lipid consumption in activated sludge.

\begin{figure}
\includegraphics[scale=2.7]{DGGE_R4_450bp_thesis.jpg}
\caption{Bacterial community fingerprints of GT amended and unamended control cultures over time. Whilst unamended activated sludge fingerprints remained relatively stable over time fingerprints from the GT amended culture shifted dramatically between day 3 and day 9 of incubation. Bands of interest (numbered 1-12) were excised and sequenced to identify the bacterial lineages from which the bands are derived.}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of replicate 4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1   &  \emph{Bacteroidetes} (JX473581.1) & --- & 8e$^{-45}$ \\
\hline
2  & \emph{Bacillus} sp. (GU271888.1) & \emph{Bacilli} & 5e$^{-70}$ \\
\hline
3 & Uncultured \emph{Dechloromonas} sp. (JQ012310.1) & \emph{$\beta$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Azospira oryzae} (KF260987.1) & \emph{$\beta$-proteobacteria} & 1E$^{-50}$ \\
\hline
5 & Uncultured \emph{Xanthmonadales} (KC588330.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
6 & Uncultured \emph{Dechloromonas} sp. (KF003189.1) & \emph{$\beta$-proteobacteria} & 4e$^{-103}$ \\
\hline
7 & \emph{Novosphingobium} sp. (KF544940.1) & \emph{$\alpha$-proteobacteria} & 1e$^{-173}$ \\
\hline
8 & \emph{Novosphingobium} (KF544932.1) & \emph{$\alpha$-proteobacteria} & 4e$^{-61}$ \\
\hline
9 & \emph{Sphingomonas} sp. (AY521009.2) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
10 & \emph{Sphingomonas suberifaciens} (AY521009.2) & \emph{$\alpha$-proteobacteria} & 3e$^{-119}$ \\
\hline
11 & \emph{Sphingomonas} sp. (JQ928361.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$ \\
\hline
12 & \emph{Roseomonas} sp.  (KF254767.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-65}$ \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

To further enrich for lipid degrading bacteria in activated sludge, secondary and subsequent tertiary sub-cultures were established from the primary GT amended cultures in the presence of GT. DGGE was used to investigate further shifts in bacterial community composition in planktonic, aggregated and emulsified sludge fractions (Figure ). Bands were excised as before and sequence matches presented in Table MN. 

A distinct change in microbial composition due to long term GT exposure was evident. Secondary enrichment (t2) was initiated after 10 days of primary culturing and the tertiary transfer (t3) was performed after 11 days of t2 and conducted for 45 days before DNA extraction. Bands which indicated abundant bacterial lineages (annotated in Figure bb) were excised, sequenced and matched against the NCBI database (Table,,). 
Considering band intensity to correlate to abundance, the dominant microbes in t3 aggregate and aggregate supernatant add up to resemble the banding pattern for the emulsion. 
Key players in this setting were \emph{Sphingomonas}, resembling the primary enrichment, and \emph{Bradyrhizobium}. \emph{Oleomonas sagarensis} and \emph{Rhodovarius lipocyclicus} were represented in t3, in emulsion and the planktonic state respectively.

%Mike: You need a couple of sentences describing what the data shows. Something along these lines from above… ‘A clear shift in bacterial community composition is apparent in response to GT addition. Bands of interest (annotated in Figure 9) representing distinct bacterial lineages, were excised, sequenced and matched with existing bacterial sequences in the NCBI database (Table 2). Initially abundant Bacillus, Dechloromonas, Xanthomonadales, Bacteroidetes and Azospira lineages were replaced with Novosphingobium, Sphingomonas and Roseomonas lineages. These results implicate the latter three in lipid consumption in activated sludge.’

\begin{figure}
\includegraphics[scale=1.2]{DGGE_misc_450bp_thesis.jpg}
\caption{Community structure of aggregates and emulsions from various time points (D7, D12, D18) as well as secondary (t2) and tertiary (t3) enrichments, which are uniformly taken after 45 days incubation, and community comparison between 1 and 2 \% GT enrichment}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of various further enrichments with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 & \emph{Nevskia} sp. (GQ845011.1) & \emph{$\gamma$-proteobacteria} & 0.0  \\
\hline
2 & \emph{Sphingomonas} sp. (KC172307.1) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
3 & \emph{Sphingobium} sp. (KF437579.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-74}$ \\
\hline
4 & \emph{Sphingomonas} sp. (KF544924.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
5 & \emph{Rhodovarius lipocyclicus} (NR\_025629.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-114}$ \\
\hline
6 & \emph{Xanthobacter} sp. (AB847934.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$  \\
\hline
7 & \emph{Xanthobacter} sp. (AB245351.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-45}$  \\
\hline
8 & \emph{Sphingomonas} sp.(KF551133.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-100}$  \\
\hline
9 & \emph{Bradyrhizobium} sp. (JX505076.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-165}$  \\
\hline
10 & \emph{Sphingomonas} sp. (EF636068.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
11 & Candidatus \emph{Competibacter} sp. (JQ480426.1) & \emph{$\gamma$-proteobacteria} & 1e$^{-123}$  \\
\hline
12 & \emph{Sphingomonas} sp. (HE974351.1) & \emph{$\alpha$-proteobacteria} &  1e$^{-148}$ \\
\hline
13 & \emph{Stakelama pacifica} (HE662817.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-77}$  \\
\hline
14 & \emph{Oleomonas sagaranensis} (AJ784808.1) & \emph{$\alpha$-proteobacteria} & 6e$^{-101}$  \\
\hline
\end{tabular}

\end{table}
\FloatBarrier
Approximately 92 \% of identified key organisms from DGGEs belonged to the proteobacteria. Of these 75 \% were \emph{$\alpha$}-proteobacteria  and each \emph{$\gamma$}- and \emph{$\beta$}- proteobacteria were 12.5 \%.


\subsection{Isolation and characterization of lipid degrading bacteria}

The development of an activated sludge floc formation model based on bacterial interactions with lipids necessitates isolation of model lipid degrading bacteria (Aim 4). Lipid droplets from sludge amended with 1 \% GT were washed and plated on solid media with agar and GT as the only available carbon source. Colonies were subsequently subcultured to isolate bacteria that use the lipid as a carbon and energy source. Five pure bacterial isolates were obtained (LC1 - LC5) and lipid degrading ability was confirmed by culturing in liquid media (without agar) with and without lipid. All five cultures grew in the presence but not in the absence of lipid. 
Sequencing of lipid colonising isolates resulted in identification of LC1 - LC4 (Table 4). The sequence quality for LC5 was not sufficient to compare with the NCBI database.

\begin{table}
\caption{Sequencing results for 16S rRNA fragments of lipid colonising isolates LC1 - LC4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
Isolate & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 &  \emph{Pandoraea} sp. (KF378759.1) & \emph{$\beta$-proteobacteria} & 2e$^{-79}$ \\
\hline
2 & Uncultured \emph{Achromobacter} sp. (KF448091.1) & \emph{$\beta$-proteobacteria} & 1e$^{-139}$ \\
\hline
3 & \emph{Enterobacter} sp. (KF411353.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Pseudomonas} sp. (KC822768.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
\end{tabular}
\end{table}



Lipid colonising isolates were then tested for the production of AHLs (Aim 5). AHL mediated gene expression has been linked to lipase activity in proteobacteria and therefore may play a role in lipid based floc formation. A LuxR based bioassay in which expression of \emph{gfp} is upregulated in the presence of AHLs was used for the detection of AHLs in culture supernatants. Figure B shows that three of the five isolates (LC2, LC3 and LC5) activated the assay suggesting that they produce AHLs or molecules with AHL-like activity. A standard curve is presented in supplementaray information (Supplementary Material, Figure XX) to enable comparison with pure N-3-oxohexanoyl-L-homoserone lactone. LC1 exhibits minimal AHL activity detectable by LuxR, while LC4 displays no activity.
Interestingly, both LC2 and LC3 evoked markedly higher LuxR responses when cultured in the presence of lipid, while LC5 elicited the highest LuxR response but only marginally more so when cultured in the presence of lipids (Figure x). Another interesting lipid dependent phenomenon observed was the production of a pink pigment by LC1 when grown in the presence of lipid (Figure m).

\begin{figure}
\includegraphics[scale=.65]{LC1_comp.png}
\caption{Lipid colonising isolate LC1 cultured in the presence (A) and absence (B) of lipid in LCM.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{LuxR.PNG}
\caption{Response to LuxR bioassay by lipid colonising isolates LC1 - LC5, comparatively cultured in the presence and absence of lipid.}
\end{figure}

\FloatBarrier
