\documentclass[twoside,12pt]{article}

\usepackage{placeins}
\usepackage[numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\usepackage{tabularx}
\linespread{1.3}
%\fontsize{8cm}{1.3em}\selectfont

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\setlength{\parindent}{0pt}

\title{\textbf{Development of an experimental model system for activated sludge floc formation based on bacterial interactions with lipids}}
\author{Anna Liza Kretzschmar\\
        z3218219\\
        Supervisor: Mike Manefield}
\date{}
%\usepackage{cite}
%something in reamble is fucked.. independent of line 17

\begin{document}
\thispagestyle{plain}
\pagestyle{plain}
\setcounter{page}{1}
\pagenumbering{roman}


\section{Abstract}
Every urban civilisation faces the challenge of dealing with wastewater.  Treatment of wastewater is currently carried out in an environmentally responsible manner. The biological aspect of wastewater treatment plants employs microbial flocs to oxidise organic compounds in the waste influent, specifically in activated sludge. This is combined with chemical and physical remediation strategies to yield remediated effluent that can be released into the environment without posing a  threat. \\

It is established that municipal wastewater activated sludge comprises a core microbiome that is adaptable to deal with fluctuating wastewater influent. When the influent contains a high lipid influent, the flocculation process is prevented and the biological aspect of wastewater remediation stalls. The mechanism underlying this phenomenon is not known. This study seeks to establish an experimental system to further the understanding of this knowledge gap in order to aid in establishing prevention of system failure. \\

The changes in partitioning behaviour of lipids in activated sludge and in microbial composition due to lipid exposure were assessed using denaturing gradient gel electrophoresis and lipase activity was also examined. Furthermore, lipid colonising microbes are isolated, identified and screened for \emph{N}-acyl-L-homoserine lactone (AHL) activity. Overall, the data suggests lipid colonisation and consumption plays a role in activated sludge floc formation and the foundation of an experimental model to examine the role of intercellular signalling in the process has been established.
 
\newpage
\section{Acknowledgements}
\thispagestyle{plain}
Thank you to my supervisor, Mike Manefield, for advice, guidance and time. But most of all, his infectious enthusiasm.\\

My gratitude also extends to my in no way co-supervisor, Tim Williams for his help and boundless encouragement.\\

The members of the Manefield Lab have been amazing -  by answering my many questions, showing me the ropes and their support.\\ 

Credit to Drew and my friends, for helping me conserve a semblance of sanity and health with their care and shenanigans.

\newpage
\thispagestyle{plain}
\tableofcontents

\newpage


\listoffigures

\newpage

\listoftables

\newpage
\section{List of Abbreviation}
\thispagestyle{plain}
\begin{tabularx}{\textwidth}{ p{4cm} | p{8cm} }
\hline
AHL & \emph{N}-acyl-L-homoserine lactone \\
BLAST & Basic Local Alignment Search Tool  \\
bp & Base pair \\
D & Day \\
$\,^{\circ}\mathrm{C}$ & Degrees celsius \\
DGGE & Denaturing gradient gel electrophoresis \\
DNA & Deoxyribonucleic acid  \\
 EPS & Extracellular polymeric substances \\
FISH & Fluorescence in situ hybridisation \\
 GC & Guanine and cytosine pair \\
GT & Glyceryl trioleate \\
LC & Lipid coloniser \\
$\mu$l & Microlitre \\
$\mu$m & Micrometre \\
ml & Millilitre \\
mM & Micromolar \\
M & Molar \\
NCBI &  National Centre for Biotechnology Information \\
PBS & Phosphate buffered saline \\
PCR & Polymerase chain reaction \\
 p-npp & p-Nitrophenyl palmitate \\
rcf & Relative centrifugal force \\
rRNA & Ribosomal ribonucleic acid \\
SDS & Sodium dodecyl sulfate \\
t & Time \\
 t2 & Secondary transfer \\
t3 & Tertiary transfer \\
U & Enzyme activity \\
V & Volt \\
v/v & volume per volume \\
w/v & Weight per volume \\
XS & xanthogenate-SDS \\
  \hline
\end{tabularx}


\newpage
%\pagestyle{headings}
\setcounter{page}{1}
\pagenumbering{arabic}
\section{Introduction}
\thispagestyle{plain}
Wastewater treatment plants are common in the global urban setting and a necessity for the continued function of society. In such systems, fluid waste entering the plant, is purified largely through microbiological processes and the effluent can then be released into the environment. It is necessary to understand these microbiological processes underlying wastewater treatment in order to optimise the functionality of the plant, as the global human population expands and strains the Earth's finite water resources. In 1914 Arden and Lockett pioneered the development of wastewater plants using activated sludge \cite{ardern1914experiments}.\\

Activated sludge consists of complex microbial communities living in cellular aggregates called flocculates (flocs). The formation of flocs in activated sludge is crucial for global wastewater treatment. Without floc formation activated sludge would not settle under gravity and the treated water could not be decanted or separated from the microbial biomass. Despite the importance of the formation of activated sludge flocs to human civilization, very little is known about how they arise, how they mature and how they ultimately lose activity as sludge age increases. These major scientific knowledge gaps prevent the rational engineering of microbial aggregates for wastewater treatment. The present study represents a stepping-stone towards addressing these knowledge gaps on floc formation through development of an experimental model of floc formation based on the interactions between activated sludge bacteria and lipids. \\


\subsection{Wastewater treatment is essential to civilisation}
The development of the current municipal wastewater treatment system has evolved over millennia to combat the waste arising from congregated human habitation. The Mesopotamian empire, from between 3500 to 2500 BC, built connections from some homes to storm water drains to remove wastes, while the Babylonians developed clay pipes to fulfil the same purpose \cite{lofrano2010}. The logical necessity to develop such a system to prevent disease and minimise environmental impact globally has been continuously improved. In 1914 Arden and Lockett pioneered the development of activated sludge, which is one of the most commonly employed biotechnologies today \cite{jenkins2004manual,muchie2010bioremediation}\\


Current plants combine physical, chemical and biological treatment strategies to the polluted influent in order to reduce environmental and health impacts. These treatments are in place to minimise the presence of nitrogen, phosphorous and carbon, reduce the biological oxygen demand and chemical oxygen demand and filter out harmful substances and pathogens \cite{mayhew1997low}.
Water quality is in part assessed by the dissolved oxygen concentration. The parameters for this assessment are measured by the concentration of substances present that can be oxidised chemically or biologically to inorganic end products \cite{pisarevsky2005chemical}.

\subsubsection{\emph{Microbial floc formation underlies successful wastewater treatment}}
Initially, large solids in the influent are either filtered out or broken down into smaller particulates to prevent blockages or excessive mechanical strain during the process. During the primary treatment phase, the influent is deposited in a large settlement tank where the majority of the particles settle to the bottom due to gravity. The weir controlled outlet flow is directed to the aeration tank for secondary treatment. The activated sludge in the aeration tank comprises the biological aspect of the treatment. It is an oxygenated system where the influent is incubated with microbial consortia, predominantly aggregated as flocs, which live by oxidising the soluble and fine sewage in the mixed liquor \cite{mayhew1997low}. The suspension is drawn off into a final settlement tank where the sludge flocs settle at the bottom and the treated water is then discharged. Depending on the required effluent quality, tertiary treatment may also be required to remove residual solids, compounds or planktonic bacteria that could pollute or enrich the effluent destination \cite{Price_95}. The critically important sludge flocs are either recycled back into the mixed liquor to do more work or discarded.\\


Bioflocculation is essential for wastewater treatment, as the settlement of the flocs is key to drawing off remediated water. Flocs are often compared with biofilms. Biofilms are traditionally thought to constitute an aggregated community of microbes at the liquid-solid interface surrounded by secreted extracellular polymeric substances (EPS), consisting of polysaccharides, nucleic acids, lipids and proteins \cite{wingender1999}, which represents 85-90 \% of biofilm dry weight \cite{Frolund_96}. Flocs are traditionally thought to be biofilms lacking the surface substrate.\\


The composition of biofilms are dependent on the participating species coupled with prevailing environmental conditions, and are sites of rapid adaptation through continuous natural selection for survival \cite{boles2008,matz2005,palmer2001}.  
The EPS offers the advantages of defence against predation by bacteria \cite{rao2005} and protozoa \cite{matz2005}; increased resistance against some chemicals such as antibiotics and hydrogen peroxide \cite{burmolle_06}; a stable matrix for the cells to reside in \cite{Flemming_10}; and to immobilise extracellular enzymes offering proximity while protecting them from proteolysis and conferring resistance to higher temperatures \cite{wingender2002extracellular,Flemming_10,skillman1998}.\\

\subsubsection{\emph{Microbial composition determines wastewater treatment efficiency}}
To optimise the efficiency of wastewater treatment, understanding the composition of the microbial communities at work in activated sludge and how their interactions influence their capabilities of remediation is essential \cite{daims2006}.
Single celled organisms from all three domains of life reside in this system including eukaryotes (protozoa, nematodes and fungi) in amongst the more abundant bacteria and archaea as well as viruses. All of these play a role in degradation of solids and nutrient cycling and hence in wastewater remediation \cite{muchie2010bioremediation}. The domain most active in remediation are the bacteria \cite{spellman2008handbook}.\\


The advance of community analysis techniques, which exclude the necessity to culture microbes in the laboratory, has made it possible to investigate the composition of microbial communities, including in activated sludge. The most frequent target for operational taxonomic unit analysis centres around the 16S ribonucleic acid (rRNA) gene, which is then compared to expansive 16S databases such as NCBI. Several techniques can be utilised around 16S rRNA, such as constructing a 16S rRNA gene library \cite{McGarvey_04}, analysing ribosomal intergenic spacer sequences \cite{Yu_01}, 16S-restriction fragment length polymorphism \cite{Gilbride_06} and comparing community structures via denaturing gradient gel electrophoresis (DGGE) \cite{Hesham_11} or pyrosequencing \cite{wang2012pyrosequencing}.\\


In wastewater treatment plants, the microbial community composition is strongly dependent on what type of waste influent the microbes are exposed to, as well as the operational settings, such as chemical oxygen demand \cite{Gilbride_06,wang2012pyrosequencing,hu2012microbial}. Municipal wastewater treatment plant influent varies in composition \citep{henze2002wastewater}.\\


\begin{sidewaystable}[!htbp]
\caption{Studies characterising core microbiomes found in activated sludge.}
\begin{tabular}{ | l | l | p{4.5cm} | p{7cm} | l | }
\hline
Study & Method & Sample size & Core microbiome composition & Refernce\\
\hline
Hesham et al. & DGGE & One plant with two different operational modes over 6 months, China & 2x \emph{$\alpha$-} \& 2x \emph{$\beta$}- proteobacteria; 3x Bacteroidetes; 2x Actinobacteria; 2x Firmicutes & \cite{Hesham_11} \\
\hline
Wagner et al. & DGGE & 750 16S rRNA sequences from wastewater treatment plants and reactors & \emph{$\alpha$-, $\beta$-} \& \emph{$\gamma$}- proteobacteria; Bacteroidetes; Actinobacteria & \cite{Wagner_02} \\
\hline
Wang et al. & Pyrosequencing & 14 plants \& pilot/benchtop operations in China & 21 - 53 \% Proteobacteria, where \emph{$\beta$-, $\alpha$-} \& \emph{$\gamma$}-proteobacteria were present 21 - 52 \%, 7 - 48 \% and 8 - 34 \% respectively & \cite{wang2012pyrosequencing} \\
\hline
Hu et al. & Pyrosequencing & 12 plants where 4 were similar to ST. Mary's, China & In similar plants, Proteobacteria dominated with up to 2:1 ratio to Bacteroidetes. In other types of plants the dominant phylum Proteobacteria was occasionally replaced by Bacteroidetes & \cite{hu2012microbial} \\
\hline
Ranasinghe et al. & Pyrosequencing & 12 plants over two years, Japan & Proteobacteria represented 38 \% of total assigned reads where 15 \% belonged to \emph{$\beta$}-proteobacteria & \cite{ranasinghe2012revealing} \\
\hline
Xia et al. & Microarray & Three plants in China and two in USA & Proteobacteria wast the dominant phylum at 50 \% to 62 \%, where \emph{$\gamma$}-proteobacteria represented  31 \% to 38 \% and \emph{$beta$}-proteobacteria 30 \% to 35 \% & \cite{xia2010bacterial} \\
\hline
\end{tabular}
\end{sidewaystable}


Table 1 summarises methods used to analyse activated sludge microbial communities across geographical and operational settings, showing a core microbiome. Pyrosequencing and microarrays studies give a snapshot of the microbial community, including low level abundance organisms \cite{ranasinghe2012revealing}. DGGE gives an insight into the change in abundance of dominant organisms active under certain conditions and over time. The wealth of information available suggests community composition varies widely between different plants, and hence there is a need to describe treatment plant details specific to the present study (Supplementary Material, Figure 21).
\FloatBarrier

\subsection{Lipids in wastewater need to be degraded by lipase}
Municipal wastewater contains lipids, at a concentration between 40 and 100 mg/m$^{3}$ \cite{Forster_92} representing 31 \% of the chemical oxygen demand of domestic wastewater \cite{Raunkjaer_94}. Oleic acid is the most common fatty acid found in native as well as used olive and sunflower oils, which are commonly used in domestic settings \cite{haba2000isolation}.  An excess in lipid content has been shown to inhibit flocculation and promote the growth of filamentous bacteria, which are linked to sludge bulking - an event that induces foaming and reduces effluent quality \cite{Forster_92}.\\


Lipids are broken down by the enzyme lipase. Due to the hydrophobic nature of lipids, they may be associated with other particulates in the activated sludge or form a separate phase. In the former case, membrane or EPS bound lipases are likely expressed for degradation, whereas lipids in a separate phase could be targeted with extracellular lipases. It is advantageous for consortia to express lipase to utilise this carbon source and lipase is commonly expressed in activated sludge \cite{gessesse2003lipase}. Investigations as to the most effective strategy to remove lipids in wastewater have been conducted extensively. Strategies including pretreatment with large concentrations of lipases and introducing pure or mixed lipid degrading cultures to activated sludge \cite{Wakelin_97}. 

\subsubsection{\emph{Lipases are extracellular degradation units}}
Triacylglycerol acylhydrolase (Enzyme Class 3.1.1.3) catalyses the reversible hydrolysis of triacylglycerols by targeting the ester bonds that attach the fatty acid side chain to the glycerol backbone. The model lipid Glyceryl trioleate (GT) used in this study consists of three oleic acid side chains.
Lipase activity is highly chemo-, enantio- and regioselective. Common to this class of enzyme is an extruding loop that extends over the active site, called the lid. The aggregated, hydrophobic nature of the substrate causes interfacial activation, which causes the lid to move to reveal the underlying active site to the substrate \cite{derewenda1992,van_Tilbeurgh1993}. These features contribute to lipases being highly efficient biocatalysts in organic chemistry and their extensive representation in the industrial setting. 
They are found within the textile, detergent, food processing, leather, pharmaceutical, pulp and paper industries \cite{hasan_06}; are essential for the production of fine chemicals such as flavours, cosmetics, agrochemicals and therapeutics \cite{jaeger2002}; and are under investigation for their potential to produce biodiesel to replace fossil fuels \cite{hasan_06,iso2001}. \\


Microbial lipases are secreted into the extracellular space, where they can catalyse the liberation of fatty acids at the lipid-water interface, which are then absorbed and utilised as a C source. In Gram negative bacteria, including Proteobacteria, the lipase zymogen passes two membranes separated by the periplasm, before secretion and folding into its active conformation \cite{bos2007,michel2009}. There are two pathways by which this can occur: the type I or type II secretory pathways. \\


\subsection{The role of intercellular signalling in activated sludge flocs and regulation of lipase production}
Bacteria monitor their environment through the production of and response to specific metabolites. Such molecules occupy the intercellular space in microbial aggregates and at elevated concentrations trigger a change in transcriptional regulation depending on their concentration. This allows the consortia, whether multi-species or cross-domain \citep{williams2007quorum}, to engage in a concerted effort that resembles multi-cellularity \cite{kjelleberg2002}. While there are several bacterial communication systems, this study focuses on \emph{N}-acyl-L-homoserine lactone (AHL) dependent gene expression orchestration. The bacteria that actively participate in this style of communication are predominately from the phylum Proteobacteria, which are diverse and well represented in the environment, especially so in municipal activated sludge \cite{Hesham_11,Wagner_02}.  \\


The machinery facilitating AHL-mediated gene expression consists of two major constituents, - LuxR and LuxI, or their respective homologs. The latter is an AHL synthase, while the former is a receptor that binds AHLs at a threshold concentration. The LuxR-AHL complex functions as a transcription factor for species specific functional genes. These genes typically play a role in bacterial cell adhesion to surfaces, biofilm formation and the production of extracellular enzymes \cite{Flemming_10}.
These traits are usually under further control outside of AHL-mediated transcription factors as well as potential cross regulation between several QS systems within the organism \cite{juhas2005}.


\subsubsection{\emph{AHL responsive gene circuits are found in Proteobacteria}}
Proteobacteria is the most prevalent phylum containing AHL mediated expression circuits \cite{gelencser_12}. In 2008 Case et al. (2008) compiled a list of isolates containing \emph{luxR} and \emph{luxI} homologs from 512 completed genomes on the NCBI platform, where 13 \% of those contained both \emph{luxR} and \emph{luxI}. These were exclusively from the phylum Proteobacteria and were found in 26 \% of all finished proteobacterial genomes \cite{case_08}.
The type II secretion system is only found in this phylum, usually associated with delivering QS influenced virulence factors and enzymes, such as lipase, into extracellular space \cite{sandkvist2001}. 

The frequency in which Case et al. found \emph{luxR} and \emph{lux} in the Proteobacteria \cite{case_08} coupled with the prevalence of this phylum in activated sludge \cite{Wagner_02,Hesham_11}, raises the fascinating proposition that AHL mediated gene expression plays a role in activated sludge floc formation.



\subsubsection{\emph{AHL mediated lipase production in Proteobacteria}}
The influence of AHL mediated gene expression on lipase production has been documented in several genera, discussed in this section with their respective classification and \emph{luxRI} homologs.\\


\emph{\underline{Burkholderia spp.}} 
\\In \emph{Burkholderia cepacia}, Lewenza et al. have demonstrated that the expression of \emph{lipA}, which encodes triacylglycerol lipase, is linked to CepR, the LuxR homolog \cite{lewenza1999}, which is in direct opposition to the results obtained by Huber et al. \cite{huber2001}. In Lewenza et al.'s study, lipase production in \emph{cepR} mutants was reduced by up to 45 \%. However the supplementation of CepR via a plasmid did not restore wild type lipase expression, which suggests that the link between \emph{cepR} and \emph{lipA} is other than transcriptional activation. The lack of a \emph{lux} box within \emph{lipA}, to which the CepR:AHL transcription factor would bind, supports the postulation that \emph{lipA} is situated downstream of \emph{cepR} and under same operon control \cite{lewenza1999}.\\



\emph{B. glumae}, an emerging rice pathogen, regulates it's lipase production with AHL mediated gene expression and lipase expression is innately linked with the pathogenic phenotype. TofR, the LuxR homolog, activates \emph{lipA} transcription \cite{devescovi_07}. \\


A study by Suarez-Moreno et al. which compared the plant associated species \emph{B. kururiensis}, \emph{B. unamae} and \emph{B. xenovorans} found no discernible impact of disabling the respective LuxR homologs on lipase production, in agreement with Huber et al. \cite{huber2001,suarez2010}. For these species tested, which live in symbiosis with the plant host, it follows that lipase expression, which is generally considered a virulent attribute, should not be population density dependent as increasing cell density would imply successful symbiosis. \\



In \emph{B. vietnamiensis}, the literature on how AHLs influence lipase secretion are contradictory. Conway et al. recorded no detectable influence of AHLs on lipase production whether in the wild type or an AHL synthase (\emph{bviI}) deficient mutant \cite{conway_02}. On the other hand Ulrich et al. conducted a they found three separate AHL synthases referred to as \emph{btaI123} and five separate \emph{luxR}-like transcriptional regulators named \emph{btaR12345}, which is a nomenclature system distinct to other nomenclature reported here. They found that \emph{btaR1}, \emph{btaR3}, \emph{btaR4} and \emph{btaR5} acted as repressors on lipase production \cite{ulrich2004}. \emph{btaI1} and \emph{btaI3} inhibit lipase production while \emph{btaI2} enhances it. Even though the data in regard to lipase activation and repression implies that \emph{luxR123} and \emph{luxI123} represent complete circuit systems, the evidence was not compelling to Ulrich et al. to draw that conclusion without further investigation \cite{ulrich2004}.
It is unclear whether the studies by Conway et al. and Ulrich et al. refer to the same AHL-mediated systems, if the proposed genes \emph{bviRI} and \emph{btaRI} are synonymous. If this is the case, a possible explanation for Conway et al. not detecting any impact of the \emph{bviR} mutant on lipase production could be given by them investigating what Ulrich et al. termed \emph{btaR2}, in which they also reported no discernible impact on lipase production. 
The complicated and cross-regulatory system surrounding lipase expression within the Burkholderia genus may indicate the complexity of lipase production in other Proteobacteria, including the role that AHLs play to influence the expression.
\\
\\ \emph{\underline{Pseudomonas spp.}}
\\ \emph{P. aeruginosa} has two AHL dependent systems: \emph{las} and \emph{rhl} which each represent \emph{lux} homologs and respond to separate AHLs. The LasR:AHL complex controls \emph{rhlIR} expression, while RhlR activates \emph{lipA} transcription. Hence RhlR directly influences lipase production while LasR's influence is indirect. However a third non-AHL mediated system, called the \emph{P. aeruginosa} quinilone signal, regulates the other two systems \cite{juhas2005}. 


The AHLs are also influenced by other factors such as GacA, which promotes the AHL regulatory cascade \cite{reimmann1997}, as well as RsmA which degrades AHLs \cite{pessi2001}. However even though RsmA degrades AHLs, it positively affects lipase production as it binds to the \textit{lip} mRNA and stabilises the transcript to reduce the transcript's rate of degradation. RsmA in turn is regulated by RsmZ, whose suppression of RsmA therefore negatively affects lipase production \cite{heurlier2004}. 
\\
\\ \emph{\underline{Xenorhabdus nematophilus}} 
\\The insect pathogen \emph{Xenorhabdus nematophilus} resides in the intestinal tract of the symbiotic host, the nematode worm \emph{Steinernema carpocapsae}. It aids the host in reproducing while the nematode represents a reservoir for the bacterium to infect insects \cite{herbert2007}. Lipase production is upregulated by  AHL mediated transcriptional activation \cite{dunphy_97} if \emph{flhDC}, which is the flagella master operon, is also expressed \cite{rosenau2000}. Another regulator is Lrp, which controls LrhA, and positively impacts lipase production and motility \cite{richards2008} 
The link between lipase expression, motility and virulence is established \cite{givaudan_00}, which suggests that the lipase contributes to the feeding mechanism while infecting rather than being a virulent agent itself \cite{richards2010}. Considering the organism's life style, which involves the necessity to leave the host in order to access the target, the controls associated with lipase expression are consistent with the requirement to move to attack the food source before releasing extracellular enzymes.
\\
\\ \emph{\underline{Serratia spp.}} 
\\Lipase production in both \emph{S. marcescens} and \emph{S. proteamaculanss} is influenced by AHL- mediated gene expression \cite{horng2002,shibatani2000,christensen_03}. Furthermore, in \emph{Serratia marcescens} surface attachment and biofilm formation in is influenced by the same mechanism \cite{labbate2007} and all contribute to the virulent phenotype \cite{hejazi_97}.

\newpage
\section{The development of an experimental model of activated sludge floc formation based on lipid as a substrate}
\thispagestyle{plain}
Major knowledge gaps exist in the scientific literature regarding the mechanistic processes governing floc formation and activity. For example, it is unclear how floc formation is initiated, how enzymatic activity is regulated within flocs, what role intercellular signalling plays in floc formation and activity or why flocs lose activity as activated sludge age increases. Given how important activated sludge floc formation and activity are to human civilization it is surprising such knowledge gaps exist. A promising approach to addressing these knowledge gaps is through development and interrogation of experimental models of floc formation. The absence of such models curtails scientific endeavour in the context of wastewater treatment.  \\

The present study seeks to develop an experimental model system based on interactions between activated sludge bacteria and lipids. The choice of lipids as a substrate is justified by the high lipid content of wastewater and the high lipase activity in activated sludge. Further, the link between Proteobacteria, which are abundant in activated sludge, and AHL dependent regulation of lipase activity offers an opportunity to explore the role of AHL mediated gene expression in activated sludge floc formation. The lipid chosen to develop the model is glyceryl trioleate, which is the most abundant lipid in vegetable oil. The study is by necessity developmental with very little experimental information available as a starting point.\\

This project directly addresses the hypothesis that the lipid glyceryl trioleate serves as a surface substratum for activated sludge floc formation. 


Specifically, the aims are:
\begin{enumerate}
\item To document the partitioning behaviour of glyceryl trioleate in activated sludge.
\item To document spatial relationships between glyceryl trioleate and activated sludge bacteria.
\item To describe the bacterial community composition of flocs in the presence of glyceryl trioleate.
\item To isolate and identify glyceryl trioleate degrading bacteria from activated sludge.

\markright{7 EXPERIMENTAL MODEL OF FLOC FORMATION}
\item To test glyceryl trioleate degrading bacteria from activated sludge for AHL production.
\end{enumerate}
\newpage

\section{Methods}
\thispagestyle{plain}
\subsection{Enrichment cultures}

Activated sludge samples were acquired from activated sludge in St. Mary’s municipal wastewater treat­ment plant for establishment of cultures in the laboratory. The treatment plant is located in the suburb of St Mary's in western Sydney serving a population of app. 160000 in a catchment area of 84 km$^{2}$ and discharges into the Hawkesbury-Nepean River. On a daily basis the plant processes 35 million litres of wastewater. Wastewater can enter two influent streams - stage 1 and 2 or stage 3 (Supplementary Material, Figure 21). The streams are subjected to differing primary treatments, however secondary and tertiary treatment is the same. The biological treatment is part of the secondary treatment stage. The biological process is conducted in three stages: 1. Anaerobic zone, where microbes take up carbon and release phosphates; 2. Anoxic zone, where carbon is consumed and nitrates are released as nitrogen gas; 3. Aerobic zone, where organic matter including ammonia is oxidized, thereby reducing oxygen demand \cite{stmarys}. The sludge samples collected for this study were taken from the Aerobic zone.\\


Enrichment cultures consisted of 50 ml activated sludge collected from St. Mary's wastewater plant in each of eight identical 250 ml Boeco flasks. The experiment was conducted in quadruplicates of 2 \% GT enrichment (Glyceryl trioleate, Sigma-Aldrich T7140-10G) and no treatment. The cultures were incubated at $37\,^{\circ}\mathrm{C}$ on a shaker set at 140 rpm. Samples were taken over a 25 day period and consisted of 0.5 ml for deoxyri­bonucleic Acids (DNA) extraction and 0.5 ml for lipase assays. The 1 ml removed was replaced with sterile, syringe filtered (0.22 $\mu$m Millipore) activated sludge supernatant. 
A trial enrichment run was conducted 1 \% GT enrichment rather than 2 \%.

\subsubsection{\emph{Microscopy}}
Lipid droplets were removed with tweezers and washed in PBS in two subsequent crystallising dishes. After transferral to a glass slide, the droplet was stained with 5 $\mu$l SybrGreen, a DNA binding dye, and covered with a cover slide. The droplets were imaged with an Olympus Fluorescence Microscope which elicits a visual response from DNA bound dye.

\subsubsection{\emph{DNA extraction}}
DNA was extracted by the amended xanthogenate-SDS (XS) method \cite{tillett2000xanthogenate}. The samples for DNA extraction were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube, supernatant removed and the pellets frozen . In the case of DNA extraction from GT droplets, 0.5 ml from GT enrichment cultures were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube and the pellet was decanted into sterile MilliQ water in a crystallising dish. Semi-solid lipid droplets were removed with tweezers and washed in a second crystallising dish, transferred to a 1.5 ml eppendorf tube and frozen. Thawed pellets were re-suspended in 0.2 ml phosphate buffered saline (PBS per Supplementary Material, Table 9).\\


For each sample, 0.9 ml phenol solution and 0.9 ml XS-buffer (as per Supplementary Material, Table 10) was  heated to $70\,^{\circ}\mathrm{C}$ in a 2 ml microcentrifuge tube in a water bath for 5 min. The 0.2 ml cell suspension was added to the 2 ml tubes, mixed by inversion and left at $70\,^{\circ}\mathrm{C}$ for 15 min. Then the samples were vortexed for 10 sec and frozen for 2 min, followed by centrifugation at 16.1 rcf for 5 min. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 0.9 ml phenol:chloroform:isoamyl alcohol mix and centrifuged at 16.1 rcf for 5 min. This step was repeated. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 1 ml ice cold isopropanol, 50 $\mu$l 3 M sodium acetate pH 7.5 and 1.3 $\mu$l glyco blue, mixed by inversion and frozen over night. Subsequent centrifugation at 16.1 rcf for 60 min at $0\,^{\circ}\mathrm{C}$ was followed by removal of isopropanol and addition of 1 ml 80 \% ethanol. Centrifugation at 16.1 rcf for 20 min at $0\,^{\circ}\mathrm{C}$ followed, then ethanol was removed and samples were left to air dry for 30 min before re-suspension in 0.1 ml molecular water.
\\ 
Extracted DNA concentration was analysed via Nanodrop (NanoDrop ND-1000 Spectrophometer) as well as Qubit (Invitrogen, Qubit 2.0 Flourometer), as per standard protocol for broad range DNA detection.

\subsubsection{\emph{Lipase assay}}
The lipase assay was adapted from Christensen et al. \cite{christensen_03} as demonstrated in Figure 1. From each replicate 0.5 ml was taken and centrifuged at 16.1 rcf for 5 min or until phases separated. The supernatant was removed and syringe sterilised (0.22 $\mu$m Millipore filter) to test for suspended EC lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water with the addition of 0.1 ml Zirkonium beads (Daintree Scientific). The samples were bead beaten for three 45 sec cycles and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and and syringe sterilised (0.22 $\mu$m Millipore filter) to test for EPS associated lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water to test for membrane bound lipase.\\


\begin{figure}
\begin{center}
\includegraphics[width=.65\textwidth]{lipase_assay.png}\\
\includegraphics[width=.65\textwidth]{lipase_assay2.png}
\caption{Fractionation of each sample into three fractions, each subjected to lipase activity assay.}
\end{center}
\end{figure}

For each sample 0.1 ml were incubated with 0.9 ml of freshly mixed p-nitrophenyl palmitate containing substrate solutions A and B (as per Supplementary Material, Tables 6 and 7) for one hour. Lipase cleaves the p-nitrophenyl group off the plamitate which can be quantified at an absorbance of 410. The p-nitrophenyl concentration is directly correlative with lipase activity. The reaction is terminated by alkaline pH inactivation of lipase with 1 ml 1 M sodium carbonate. Samples were centrifuged at 16.1 rcf for 5 min. From each terminated reaction, 0.3 ml supernatant was transferred to 96 well micro titre plate and the absorbance read at a wavelength of 410 with a Microtitre reader (Molecular Devices, Spectra Max 340).
\FloatBarrier
\subsection{Community analysis of enriched cultures}
DNA was extracted from enrichment culture isolates, verified with DGGE polymerase chain reaction (PCR). The DGGE PCR assay was set up with the primers 357FGC and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq (Lucigen) Master mix, 15.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 2 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.
Bands of interest were selected from the DGGE and excised and re-run on DGGE PCR to test their integrity, and if satisfactory underwent sequencing PCR. 

\subsubsection{\emph{Further enrichment}}
To investigate the change of microbial composition with extended exposure to GT, secondary (t2) and tertiary (t3) enrichments were conducted by transferral into 25 ml filtered sludge supernatant (0.22 $\mu$m Millipore filter) and 2 \% GT (Supplementary Material, Figures 12 and 13). t2 was conducted with an aggregate and 5 \% emulsion after 10 days of incubation. After 11 days of t2, the aggregate and emulsion were transferred again as well as 5 \% of the planktonic dispersal phase from the aggregate. t3 was run for 45 days before DNA extraction.\\

To confirm the difference in morphology of activated sludge in the presence of 1 vs. 2 \% GT rather than the cause being seasonal variation in the microbial community, a comparative enrichment was conducted. DNA extracted after 7 days and processed for DGGE analysis.
 
\subsubsection{\emph{Denaturing Gradient Gel Electrophoresis}}
DGGE was conducted using the BioRad DCode system and protocol for 50 \% to 70 \% denaturing gradient on a 6.7 \% arcylamide gel. Electrophoresis was conducted at 75V and $60\,^{\circ}\mathrm{C}$ for 16.5 hrs in 1x TAE. The gel was stained with SybrGold for 20 min and imaged on a BioRad Gel Doc XR+ Imaging system and BioRad Image Lab software. Bands of interest were excised and suspended in 40 $\mu$l molecular water over night, which was used as template for nested PCR set up with 357F and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 12.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 5 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$. Products for sequencing were cleaned up with DNA Clean \& Concentrator-5 kit (Zymo Research), following the PCR product protocol.

\subsubsection{\emph{Sequencing}}
The sequencing PCR assay, protocol from the Ramaciotti centre, was set up for the 357F primer at 20 $\mu$l as follows: 1 $\mu$l BigDye terminator V3.1, 20 - 50 ng PCR product, 0.32 $\mu$l primer, 3.5 $\mu$l 5x sequencing buffer and up to 20 $\mu$l with molecular water. The PCR protocol was the following: 26 cycles of denaturing phase: 10 sec at $96\,^{\circ}\mathrm{C}$, annealing phase:  at 5 sec $50\,^{\circ}\mathrm{C}$, extension phase: 4 min at $60\,^{\circ}\mathrm{C}$, followed by and storage at $4\,^{\circ}\mathrm{C}$.\\

The samples were cleaned up by addition of 5 $\mu$l of 125 mM EDTA and 60 $\mu$l 100 \% ethanol followed by vortexing and 15 min precipitation period. The samples were then spun at 14000 rcf for 20 min and supernatant was removed. After addition of 160 $\mu$l fresh 70 \% ethanol, samples were spun at 14000 rcf for 10 min and supernatant discarded. This step was repeated with addition of 80 $\mu$l 70 \% ethanol. The samples were dried in the dark and submitted to the Ramaciotti Centre at UNSW for Sanger Sequencing.

\subsection{Lipid coloniser isolation}
Sludge isolates  were isolated from lipid droplets, washed twice in PBS, by three subsequent transfers on Lipid coloniser medium (LCM per Supplementary Material, Table 8) plates. Isolates were prepared for 16S rRNA PCR for the primers 27F and 1492R at 20 $\mu$l as follows: 10 $\mu$l EconoTaq Master mix, 7.4 $\mu$l molecular water, 0.8 $\mu$l of each primer and 1 $\mu$l template. The PCR protocol was the following: 2 min at $94\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 1 min at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $59\,^{\circ}\mathrm{C}$, extension phase: 1.40 min at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$, then prepared for Sanger sequencing by the Ramaciotti centre as previously outlined.
% incl diff primers and hene length, PCR protocol.

\subsubsection{\emph{Lipid coloniser behaviour in the presence and absence of olive oil}}
Isolates were pre-cultured in 25 ml LCM broth, either with or without olive oil. 

\subsubsection{\emph{Screening lipid colonising isolates for AHL production}}
Pre-cultures were centrifuged at 16.1 rcf for 10 min and 20 ml supernatant was transferred to a 50 ml falcon tube. After addition of 20 ml Ethyl acetate with 0.01 \% glacial acetic acid, the mixture was agitated and left to separate. The top  layer was transferred to an 100 ml beaker. This was repeated twice and left over night for evaporation in a chemical fume hood. The AHLs were resuspended in 100  $\mu$l methanol and filter sterilised with PVDF Filter Vials (CP-ANALYTICA GmbH). \\

 The reporter strain \emph{Escherichia coli} carrying the plasmid pJBA357 was pre-cultured in Lysogeny Broth (LB10 per Supplementary Material, Table 5) \cite{bertani1951studies}. The plasmid contains \emph{gfp}, which  is preceded by the \emph{luxR} promoter, activated by OHHL and the response is directly proportionate to the AHLs present.
The assay was conducted in triplicate with \emph{Aeromonas hydrophila} GC1 as the positive control, LB10 as a blank and 2.5 nm, 1 nm and 0.5 nm OHHL as a comparative reference. Per microtitre plate well 20 $\mu$l of extract was added and the methanol left to evaporate, followed by addition and mixing of 100 $\mu$l \emph{E. coli} in it's exponential growth phase. After 4 hrs of incubation OD600 was read to verify \emph{E. coli} growth as well as fluorescence by excitation at 485 nm and then emission at 535 nm.
\newpage
\section{Results}
\thispagestyle{plain}
\subsection{Partitioning behaviour of glyceryl trioleate in activated sludge}

To establish an experimental model of floc formation based on interactions between lipids and bacteria in activated sludge it was first essential to examine the physical partitioning of lipids in sludge (Aim 1). Glyceryl trioleate (GT) was added to activated sludge samples in quadruplicate at 1 \% v/v. On addition GT formed a transparent and hydrophobic layer on the surface of the enrichment cultures, which was replaced with white droplets suspended below the culture surface within 7 days (Figure 2B). While varying in size, the droplets consistently decreased in diameter with increased incubation period until they were indistinguishable from the sludge after 21 days. Droplet formation decreased floc settlement for the duration of their presence.\\

A second experiment was conducted with 2 \% GT added to activated sludge samples. The increased GT concentration did not result in lipid droplet formation. Instead the cultures became increasingly viscous and lighter in colour (Figure 2A and 2C) with lipid apparently adsorbed to the sludge flocs. No settlement of sludge flocs was observed, however large dense aggregates appeared in the viscous cultures. One of these was targeted for secondary enrichment by moving the aggregate into sludge supernatant, where the aggregate expanded and developed tendril like structures (Figure 2D).

\begin{figure}
\includegraphics[scale=.77]{cultures.jpg}
\caption{Impact of GT addition to activated sludge cultures; A) the comparison between 2 \% GT enrichment and no treatment; B) lipid droplet formation in the presence of 1 \% GT; C) the lack of settleability of sludge after incubation with 2 \% GT; D) secondary enrichment by transferral of aggregate into sludge supernatant and 2 \% GT.}
\end{figure}
\FloatBarrier

\subsubsection{\emph{Association between glyceryl trioleate and activated sludge biomass}}
To examine the spatial relationships between biomass and lipids in 1 \% and 2 \% GT amended sludge samples (Aim 2), lipid droplets and aggregates were isolated and stained with SybrGreen before imaging with epifluorescence microscopy. The fluorescent stain binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black. Figure 3 shows that the biomass is closely associated with lipid, rather than the intermediate water space and firmly attached as evidenced by the washing steps applied prior to staining as part of the isolation procedure. While Figure 3A and 3B show clusters of biomass around lipid, 3C, 3D, 3E and 3F are a close up of those clusters which showcase smaller segments of lipids surrounded and colonised by microbes. Owing to the depth of the samples, these images are unavoidably complex with multiple planes visible simultaneously in soft focus. This spatial distribution and cell density is reminiscent of surface associated biofilms.
\begin{figure}
\includegraphics[scale=.4]{august_microscopy.png}
\caption{Imaging of lipid droplets post SybrGreen staining with fluorescence microscopy. A); B); C); D); E) and F) demonstrate the close association of biomass with lipid compared to the intermediate liquid phase.
The fluorescent stain binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black.}
\end{figure}

\FloatBarrier
\subsubsection{\emph{Upregulation of lipase activity in response to glyceryl trioleate}}
For the floc community to utilise and hence remediate the GT in the sludge, lipase needs to be produced as lipase activity is directly proportional to the remediation potential of the sludge. It was expected that lipase production would be upregulated in the presence of GT.
After addition of 2 \% (v/v) GT to activated sludge in quadruplicate, samples were separated into supernatant, EPS associated and membrane bound lipase fractions. \\

Lipase activity was detected upon colorimetric change from p-nitrophenyl palmitate cleavage in the substrate solution. 
Considerable variability was observed between replicates. Average lipase activity values for each fraction and time point are presented in Figures 4-7. Data from individual replicates is presented as supplementary information (Supplementary Material,  Figures 12 to 17).
An significant increase (p=0.0001) of lypolytic activity in the EPS fraction (Figure 4) within the first 3 days is evident with a gradual decrease in activity observed thereafter, dropping to approximately 30 \% of the peak activity at  the end of the incubation period. 
Conversely, the lipase activity in the supernatant fraction of the enriched cultures increased from day 3,  significantly (p=0.0436) from day 9 to 22 (Figure 5). The membrane bound lipase fraction displayed the highest lipase activity but did not differ between GT amended sludge samples and untreated controls (p=0.0815, day 12) (Figure 6). The enriched cultures displayed on average a higher total lipase activity (sum of the fractions) than the control replicates (Figure 7).

\begin{figure}
\includegraphics[scale=1.1]{lip_eps.PNG}
\caption{Average activity (n=4) of lipase immobilised in the EPS fraction of activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls. Error bars represent the standard deviation between the replicates. GT amendment increased lipase activity in the EPS fraction within three days incubation.}
\end{figure}
% need shit like that description for all graphs!
\begin{figure}
\includegraphics[scale=1.1]{lip_sn.PNG}
\caption{Average lipase activity (n=4) in the supernatant fraction of activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls. Error bars represent the standard deviation between the replicates. GT amended replicates increased lipase activity within 6 days.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{lip_biom.PNG}
\caption{Average activity (n=4) of cell membrane bound lipase in activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls. Error bars represent the standard deviation between the replicates. Lipase activity between amended and unamended replicates is insignificant.}
\end{figure}


\begin{figure}
\includegraphics[scale=1]{lip_av.PNG}
\caption{Total lipase activity (n=4) of all fractions and replicates in activated sludge amended with 2 \% glyceryl trioleate over time in comparison with unamended controls. On average, GT amended replicates display a higher lipase activity than unamended replicates.}
\end{figure}
\FloatBarrier

\subsection{Microbial community changes in the presence of lipids}
To identify bacteria potentially involved in lipid consumption bacterial community fingerprinting (DGGE) was used to monitor changes in activated sludge community composition in response to GT addition over time in one of the four replicates amended with 2 \% (v/v) GT (Aim 3). A reduction in intensity of bands in DGGE profiles over time suggests a decrease in relative abundance of bacterial lineages whilst increases in band intensity over time suggests increases in relative abundance. An increase in relative abundance in response to GT addition implicates specific bacterial lineages in lipid consumption.\\

Figure 8 shows DGGE profiles of GT amended and unamended control cultures over 25 days incubation. A clear shift in bacterial community composition is apparent in response to GT addition. Bands of interest (annotated in Figure 8) representing distinct bacterial lineages, were excised, sequenced and matched with existing bacterial sequences in the NCBI database (Table 2). Initially abundant \emph{Bacillus}, \emph{Dechloromonas}, \emph{Xanthomonadales}, \emph{Bacteroidetes} and \emph{Azospira} lineages were replaced with \emph{Novosphingobium}, \emph{Sphingomonas} and \emph{Roseomonas} lineages. These results implicate the latter three in lipid consumption in activated sludge.\\

\begin{figure}
\includegraphics[scale=2.7]{DGGE_R4_450bp_thesis.jpg}
\caption{Bacterial community fingerprints of GT amended and unamended control cultures over time. Whilst unamended activated sludge fingerprints remained relatively stable over time fingerprints from the GT amended culture shifted dramatically between day 3 and day 9 of incubation. Bands of interest (numbered 1-12) were excised and sequenced to identify the bacterial lineages from which the bands are derived.}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of replicate 4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1   &  Bacteroidetes (JX473581.1) & --- & 8e$^{-45}$ \\
\hline
2  & \emph{Bacillus} sp. (GU271888.1) & Bacilli & 5e$^{-70}$ \\
\hline
3 & Uncultured \emph{Dechloromonas} sp. (JQ012310.1) & \emph{$\beta$}-proteobacteria & 0.0 \\
\hline
4 & \emph{Azospira oryzae} (KF260987.1) & \emph{$\beta$}-proteobacteria & 1E$^{-50}$ \\
\hline
5 & Uncultured Xanthmonadales (KC588330.1) & \emph{$\gamma$}-proteobacteria & 0.0 \\
\hline
6 & Uncultured \emph{Dechloromonas} sp. (KF003189.1) & \emph{$\beta$}-proteobacteria & 4e$^{-103}$ \\
\hline
7 & \emph{Novosphingobium} sp. (KF544940.1) & \emph{$\alpha$}-proteobacteria & 1e$^{-173}$ \\
\hline
8 & \emph{Novosphingobium} sp. (KF544932.1) & \emph{$\alpha$}-proteobacteria & 4e$^{-61}$ \\
\hline
9 & \emph{Sphingomonas} sp. (AY521009.2) & \emph{$\alpha$}-proteobacteria & 0.0 \\
\hline
10 & \emph{Sphingomonas suberifaciens} (AY521009.2) & \emph{$\alpha$}-proteobacteria & 3e$^{-119}$ \\
\hline
11 & \emph{Sphingomonas} sp. (JQ928361.1) & \emph{$\alpha$}-proteobacteria & 5e$^{-86}$ \\
\hline
12 & \emph{Roseomonas} sp.  (KF254767.1) & \emph{$\alpha$}-proteobacteria & 5e$^{-65}$ \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

To further enrich for lipid degrading bacteria in activated sludge, secondary and subsequent tertiary sub-cultures were established from the primary GT amended cultures in the presence of GT. DGGE was used to investigate further shifts in bacterial community composition in planktonic, aggregated and emulsified sludge fractions (Figure 9). 
A distinct change in microbial composition due to long term GT exposure was evident. Secondary enrichment (t2) was initiated after 10 days of primary culturing and the tertiary transfer (t3) was performed after 11 days of t2 and conducted for 45 days before DNA extraction. Bands which indicated abundant bacterial lineages (annotated in Figure 9) were excised, sequenced and matched against the NCBI database (Table 3). 
Considering band intensity to correlate to abundance, the dominant microbes in t3 aggregate and aggregate supernatant add up to resemble the banding pattern for the emulsion. 
Key players in this setting were \emph{Sphingomonas} sp., resembling the primary enrichment, and \emph{Bradyrhizobium} sp. \emph{Oleomonas sagarensis} and \emph{Rhodovarius lipocyclicus} were represented in t3, in emulsion and the planktonic state respectively.

\begin{figure}
\includegraphics[scale=1.2]{DGGE_misc_450bp_thesis.jpg}
\caption{Community structure of aggregates and emulsions from various time points (D7, D12, D18) as well as secondary (t2) and tertiary (t3) enrichments, which are uniformly taken after 45 days incubation, and community comparison between 1 and 2 \% GT enrichment}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of various further enrichments with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 & \emph{Nevskia} sp. (GQ845011.1) & \emph{$\gamma$}-proteobacteria & 0.0  \\
\hline
2 & \emph{Sphingomonas} sp. (KC172307.1) & \emph{$\alpha$}-proteobacteria & 0.0 \\
\hline
3 & \emph{Sphingobium} sp. (KF437579.1) & \emph{$\alpha$}-proteobacteria & 2e$^{-74}$ \\
\hline
4 & \emph{Sphingomonas} sp. (KF544924.1) & \emph{$\alpha$}-proteobacteria & 0.0  \\
\hline
5 & \emph{Rhodovarius lipocyclicus} (NR\_025629.1) & \emph{$\alpha$}-proteobacteria & 3e$^{-114}$ \\
\hline
6 & \emph{Xanthobacter} sp. (AB847934.1) & \emph{$\alpha$}-proteobacteria & 5e$^{-86}$  \\
\hline
7 & \emph{Xanthobacter} sp. (AB245351.1) & \emph{$\alpha$}-proteobacteria & 3e$^{-45}$  \\
\hline
8 & \emph{Sphingomonas} sp.(KF551133.1) & \emph{$\alpha$}-proteobacteria & 2e$^{-100}$  \\
\hline
9 & \emph{Bradyrhizobium} sp. (JX505076.1) & \emph{$\alpha$}-proteobacteria & 3e$^{-165}$  \\
\hline
10 & \emph{Sphingomonas} sp. (EF636068.1) & \emph{$\alpha$}-proteobacteria & 0.0  \\
\hline
11 & Candidatus \emph{Competibacter} sp. (JQ480426.1) & \emph{$\gamma$}-proteobacteria & 1e$^{-123}$  \\
\hline
12 & \emph{Sphingomonas} sp. (HE974351.1) & \emph{$\alpha$}-proteobacteria &  1e$^{-148}$ \\
\hline
13 & \emph{Stakelama pacifica} (HE662817.1) & \emph{$\alpha$}-proteobacteria & 3e$^{-77}$  \\
\hline
14 & \emph{Oleomonas sagaranensis} (AJ784808.1) & \emph{$\alpha$}-proteobacteria & 6e$^{-101}$  \\
\hline
\end{tabular}

\end{table}
\FloatBarrier
Approximately 92 \% of identified key organisms from DGGEs belonged to the Proteobacteria. Of these 75 \% were \emph{$\alpha$}-proteobacteria, \emph{$\gamma$}- and \emph{$\beta$}- proteobacteria were 12.5 \% each.


\subsection{Isolation and characterization of lipid degrading bacteria}

The development of an activated sludge floc formation model based on bacterial interactions with lipids necessitates isolation of model lipid degrading bacteria (Aim 4). Lipid droplets from sludge amended with 1 \% GT were washed and plated on solid media with agar and GT as the only available carbon source. Colonies were subsequently subcultured to isolate bacteria that use the lipid as a carbon and energy source. Five pure bacterial isolates were obtained (LC1 - LC5) and lipid degrading ability was confirmed by culturing in liquid media (without agar) with and without lipid. All five cultures grew in the presence but not in the absence of lipid. \\

Sequencing of lipid colonising isolates resulted in identification of LC1 - LC4 (Table 4). The sequence quality for LC5 was not sufficient to compare with the NCBI database.

\begin{table}
\caption{Sequencing results for 16S rRNA fragments of lipid colonising isolates LC1 - LC4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
Isolate & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 &  \emph{Pandoraea} sp. (KF378759.1) & \emph{$\beta$}-proteobacteria & 2e$^{-79}$ \\
\hline
2 & Uncultured \emph{Achromobacter} sp. (KF448091.1) & \emph{$\beta$}-proteobacteria & 1e$^{-139}$ \\
\hline
3 & \emph{Enterobacter} sp. (KF411353.1) & \emph{$\gamma$}-proteobacteria & 0.0 \\
\hline
4 & \emph{Pseudomonas} sp. (KC822768.1) & \emph{$\gamma$}-proteobacteria & 0.0 \\
\hline
\end{tabular}
\end{table}

\FloatBarrier

Lipid colonising isolates were then tested for the production of AHLs (Aim 5). AHL mediated gene expression has been linked to lipase activity in Proteobacteria and therefore may play a role in lipid based floc formation. A LuxR based bioassay in which expression of \emph{gfp} is upregulated in the presence of AHLs was used for the detection of AHLs in culture supernatants. Figure 11 shows that three of the five isolates (LC2, LC3 and LC5) activated the assay suggesting that they produce AHLs or molecules with AHL-like activity. A standard curve is presented in supplementaray information (Supplementary Material, Figure 20) to enable comparison with pure N-3-oxohexanoyl-L-homoserine lactone. LC1 exhibits minimal AHL activity detectable by LuxR, while LC4 displays no activity.\\


\begin{figure}
\includegraphics[scale=.65]{LC1_comp.png}
\caption{Lipid colonising isolate LC1 cultured in the presence (A) and absence (B) of lipid in LCM.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{LuxR.PNG}
\caption{Response to LuxR bioassay by lipid colonising isolates LC1 - LC5, comparatively cultured in the presence and absence of lipid.}
\end{figure}

Despite not activating the bioassay, a \emph{Pandoraea} sp. soil isolate has been shown to secrete the AHL \emph{N}-octanoylhomoserine lactone, which has eight carbon atoms attached to the homoserine lactone ring \cite{han2013pandoraea}. A search of NCBI Protein BLAST returns a LuxR homolog but not a LuxI homolog. The \emph{Achromobacter} sp. genome was searched for a \emph{luxI} homolog by using a \emph{Burkholderia} sp. homolog as  this is a \emph{$\beta$}- proteobacterium known to contain an AHL synthase. The databases UniProt and SwissProt were used through EXPASY in order to search annotated genomes only and exclude draft genomes. This returned  22 \% similarity to an \emph{Agrobacterium} sp. LuxI homolog.
Interestingly, both \emph{Achromobacter} and \emph{Enterobacter} evoked markedly higher LuxR responses when cultured in the presence of lipid, while LC5 elicited the highest LuxR response but only marginally more so when cultured in the presence of lipids (Figure 10). Another interesting lipid dependent phenomenon observed was the production of a pink pigment by \emph{Pandoraea} when grown in the presence of lipid (Figure 10).
\FloatBarrier    
 

\newpage
\section{Discussion}
\thispagestyle{plain}
The function of our society depends on safe and efficient municipal wastewater remediation while sustainably recycling water and reducing environmental impact. Waste influent created by domestic households is extensive and fluctuates in composition. Hence the wastewater treatment system for influent remediation needs to be adaptable to deal with the volume and variability. The biological component employed by the treatment facilities is the least well understood aspect. This study sought to develop an experimental system to explore the interaction between lipids and activated sludge. Specifically, five aims were addressed relating to spatial aspects of lipid partitioning and colonisation, impacts on bacterial community composition, isolation of lipid degrading bacteria and testing for AHL production.

\subsection{Physical behaviour of lipids in activated sludge}
The physical partitioning behaviour of lipids upon introduction to activated sludge is poorly described. As a starting point it was unclear whether the addition of GT to activated sludge would result in the formation of a single free phase lipid droplet or whether sludge flocs and the incubation vessel would be coated in lipid with an absence of free phase or if multiple small lipid droplets or an emulsion would form. This lack of information on the physical partitioning behaviour in activated sludge hampers development of an experimental model for lipid dependent floc formation. To this end, 1 and 2 \% (v/v) GT was incubated in the presence of activated sludge and direct observations were made (Aim 1). \\

During the enrichment on 1 \% GT (v/v), multiple lipid droplets formed which were suspended throughout the sludge. Objects that resemble droplets in oleic acid challanged anaerobic sludge have been reported previously \cite{pereira2005anaerobic}. It stands to reason that such lipid droplets represent an attractive surface substratum for colonisation by activated sludge bacteria enabling evasion of predation and ready access to a carbon and energy source. Biofilm formation on lipid droplets would ultimately generate aggregated biomass resembling a floc. The result was encouraging in that it suggested lipid colonisation may represent a realistic model system for floc formation and development. \\

When the GT concentration was increased to 2 \% (v/v), lipid droplets did not form and the sludge increased in viscosity and lightened in colour, resembling an emulsion. This is likely due to the excessive release of fatty acids released from the high concentration of triacylglyceride applied. The 2 \% (v/v) lipid concentration applied to the activated sludge was adapted from a study conducted by Haba et al. \cite{haba2000isolation} on isolation of lipase producing bacteria. However the typical lipid concentration found in municipal wastewater is approximately 0.01 \% (v/v) \cite{Forster_92}. \\

Continued enrichment of the emulsion through sub-culturing resulted in increased viscosity to an almost solid state and white appearance in the tertiary sub-culture after 45 days. No incidence of an emulsion like phenotype in the wastewater treatment plant setting has been found. Hence, these phenomena are considered to be artifacts of the unrealistically high GT concentration applied.

\subsection{Bacteria colonise lipid droplets in activated sludge}
Addition of 1 \% (v/v) GT to activated sludge resulted in lipid droplet formation. Epifluorescence microscopy was then used to observe physical interactions (spatial relationships) between the activated sludge biomass and the lipid droplets (Aim 2). \\

The microscopy images obtained clearly showcase the close association of biomass with the lipids. Whilst the aggregation around the droplets resembles biofilm formation, it is not clear if the physicochemical properties of cells and lipid droplets drive the association or if active energy dependent processes result in microbial colonisation of the lipid droplets. \\

Whether the dense mass of cells observed is surrounding and growing on the lipid or whether it was embedded within the lipid phase is unclear from the microscopy images generated. Cells embedded in the lipid, with no access to water, are unlikely to proliferate. If lipids seed flocs, the buoyancy of the lipid core could be responsible for lack of settlement rather than the presence of lipids preventing flocculation as suggested by Forster et al. \cite{Forster_92}. Determining which microbial communities dominate during lipid induced floc settlement failure could provide another aspect for monitoring and anticipation of system failure.

\subsection{Activated sludge bacteria produce lipases and degrade lipid droplets}

Lipid droplet size was observed to decrease over time until they were no longer visible. This is consistent with microbes colonising the droplets and degrading the surface substratum as a carbon and energy source. Lipase activity was also observed to increase in activated sludge samples amended with GT. An increase in lipase activity was observed in the EPS over the first 3 days of incubation but the greatest fold increase in lipase activity was observed in the culture supernatant between 3 and 6 days of incubation as the EPS based lipase activity decreased. It has previously been shown that lipases are weakly bound to the EPS of flocs through hydrogen bonding \cite{mayer1999role,wicker1987}. The sequential appearance of lipase first in the EPS and then in the supernatant fraction is likely due to release of lipases from the EPS.\\

While lipase activity was observed in the biomass fraction, there was little difference in activity when comparing GT amended treatments and unamended controls. The unamended controls displayed a high background level of lipase activity presumably owing to the presence of background levels of lipid in activated sludge. The decline in lipase activity observed in the EPS fraction in unamended controls is concordant with this, with the background lipid component being consumed. It should also be noted that as the viscosity of samples increased it was difficult to separate the three distinct fractions.

\subsection{Sphingomonads dominate in GT amended activated sludge.}
It is not known which bacteria are responsible for the degradation of lipids in activated sludge. This information is crucial in the development of an activated sludge floc formation model based on lipid colonisation. As a starting point, the response of bacteria in activated sludge to the addition of GT was assessed using DGGE (Aim 3). It is evident from the DGGE community analysis that lipids have a profound impact on the microbial community structure in activated sludge. Whilst bacterial lineages enriched in the presence of GT can be implicated in lipid degradation in activated sludge this does not represent unambiguous evidence of lipid biodegradation by these bacteria.\\

Based on band intensity the most abundant bacterial lineages in the untreated sludge samples belonged to the genera \emph{Bacillus}, \emph{Dechloromonas} and \emph{Azospira}. \emph{Bacillus} species have been observed in sludge previously and the sequence retrieved here was associated with lipolytic activity in olive mill wastewater \cite{ertuugrul2007isolation}. \emph{Bacillus} species have also been associated with bioflocculation in starch wastewater treatment \cite{deng2003characteristics}.\\

\emph{Dechloromonas} species were observed in membrane fouling biofilms of municipal wastewater treatment plants in Japan, with over 30 \% of clones belonging to this genus \cite{miura2007membrane}. \emph{Azospira} are also common activated sludge occupants \cite{tan2003dechlorosoma,reinhold2000reassessment,hunter2007azospira,wilhelmus2013microbiological}.
Overall, the DGGE profiles of the unamended sludge community was typical of activated sludge and did not differ greatly over the incubation period.\\

In sludge samples exposed to GT the DGGE profiles over time shifted dramatically. The most abundant bacteria observed in the unamended sludge were replaced within 9 days of incubation with \emph{Sphingomonas}, \emph{Novosphingobium} and \emph{Roseomonas} species.
\emph{Sphingomonas} species are present at about 5 - 10 \% relative abundance in sludge as shown by FISH \cite{neef1999detection}, and play an important role in wastewater remediation. Members of this genera degrade testosterone and sterol hormones as well as the pollutant nonylphenol \cite{fujii2001sphingomonas,roh201017beta}. \emph{Novosphingobium} species have been shown to play an important role in wastewater remediation by degrading toxic dyes and estrogen \cite{addison2007novosphingobium,hashimoto2009contribution}.
\emph{Roseomonas} species have been found at about 5 \% relative abundance in activated sludge and are known to degrade organophosphate pesticides \cite{jiang2008bacterial,jiang2006isolation}. None of these lineages have previously been associated with lipid consumption in wastewater treatment plants. From the data generated here bacteria belonging to the \emph{Sphingomonas}, \emph{Novosphingobium} and \emph{Roseomonas} genera can be considered candidates for inclusion in an experimental system for investigating lipid based floc formation. \\


The emulsions contained an unrealistically high lipid concentration, facilitating long term monitoring of community changes in response to GT exposure.
Tertiary enrichment samples were separated to monitor the community in an aggregate, the cells dispersed into the supernatant by the aggregate and the emulsion. \\



\emph{Sphingomonas} species were identified for 3 out of the 6 main bacterial lineages in the tertiary enrichments. Bands 2 and 10 from the aggregate's  planktonic phase (Figure DGGEMISC), \emph{Sphingomonas} sp., is on the same position in the gel as band 13 for the t3 emulsion but returned as \emph{Stakelama pacifica}, a close relative of Sphingomonas \cite{chen2010stakelama}.
NCBI assigned the latter designation with and E-value of 3e$^{-77}$ from a 163 bp sequence, while the designation for \emph{Sphingomonas} sp. was with an E-value of 0.0, based on a 445 bp sequence from band 2. The two sequences were aligned with NCBI BLAST to investigate sequence similarity. The sequence alignment of band 13, which covers the 269 to 431 bp region of the amplified 16S rRNA fragment, aligns from the 269$^{th}$ bp of the \emph{Sphingomonas} with a sequence identity was 100 \% with an E-value of 7e$^{-87}$. It is concluded that band 13 represents the \emph{Sphingomonas} rather than \emph{Stakelama pacifica}.
The high variability of \emph{Sphingomonas} GC content is a common observation in DGGE \cite{qiao2012effect}.\\

%why talk about these dudes?
Members of the \emph{Bradyrhizobium} genus are slow growing \cite{rebah2002wastewater} and usually associated with plant nodules. In plant symbiosis, they contribute by fixing nitrogen - an attribute required for successful activated sludge mediated remediation. When exposed to activated sludge, these microbes have been shown to become highly antibiotic and heavy metal resistant \cite{ahmad17samiullah}.\\


\emph{Oleomonas sagarensis} becomes prevalent in the emulsion but not in the aggregate or it's supernatant. This species is involved in breaking down urea \cite{kanamori2005allophanate,kanamori2004enzymatic} which is essential for nitrification of ammonia in activated sludge.
\emph{Rhodovarius lipocyclicus} was abundant in the aggregate supernatant and emulsion t2, but disappeared by the t3 sampling point. Information about this species is scarce \cite{kampfer2004rhodovarius} and it has not been previously implicated in lipid degradation. This species could be of interest when dealing with microbiomes subject to extended lipid exposure. \\

While the majority of the microbes identified in the further enrichments were not found in the enrichment or control cultures for the 25 day duration of the enrichment experiment, these microorganisms are present in low abundance and became abundant during the next 56 days that t2 and t3 enrichments ran for. Also the detection of organisms was limited as several bands were not sequenceable.\\


Wagner et al. suggest FISH with 16S rRNA group-specific probes is necessary to accompany 16S rRNA sequencing sampling to ensure the accuracy of the OTU representation. However as this is a time consuming process, the majority of the community analysis is on 16S rRNA only \cite{Wagner_02} . A skewered representation of the operational taxonomic units in the community can arise from 16S primer bias which should be checked and accounted for.


\subsection{Lipid colonising isolates behave differently in the presence of lipids}

Five bacterial strains isolated directly from lipid droplets were screened for AHL activity and four isolates were sequenced. All four identified isolates belong to the Proteobacteria, which are known for AHL mediated gene expression of lipase. However despite their response to the LuxR bioassay, these results were not conclusive when trying to establish LuxI homologs from the NCBI database. Only \emph{Pseudomonas} species have had their lipase activity definitively linked to AHL mediated gene expression. \\

\emph{Achromobacter} and \emph{Enterobacter} elicited a markedly higher LuxR response when cultured in the presence yeast extract and starch without lipid supplement, while \emph{Pseudomonas} only caused a response when cultured in the presence of lipids. Further investigation as to whether the presence of lipids impacts the the AHL profile produced directly should be conducted. \\

Isolate LC5 was of interest due to it's high LuxR activation, in either culture condition. The organism could not be identified as the sequencing data was inconclusive for both attempts.\\

Interestingly LC1, \emph{Pandoraea} sp., produces a pink phenotype when cultured in the presence of lipids but remains white in the absence. This could be due to pigment production whose expression pattern is linked to the presence of lipids. The nature of the pink phenotype and how it relates to lipid degradation should be investigated. 
% ref bioassays
Furthermore characterisation with bioassays targeting a range of AHLs should be conducted, such as \emph{Chromobacterium violacein} CV026 and \emph{Agrobacterium tumefaciens} A136.
Nonetheless a \emph{Pandoraea} sp.  As the activation was negligible in this study, this AHL may be out of the detection range of the LuxR bioassay and should be tested with the  bioassay.
\newpage
\section{Conclusions and Future directions}
\thispagestyle{plain}
Bacterial floc formation is essential for the remediation of domestic wastewater. This study has sought to supplement the limited information available about floc formation and establishing the impact that lipid addition has on activated sludge morphology as well as microbial community structure. Deciphering the nature of colonisation in a biofilm-like manner around lipid droplets would extend our understanding of the mechanism of remediation. Droplets could be fixed cryogenically and sectioned before microscopic analysis. Monitoring the change of colonisation over time could aid in establishing the floc life cycle based on lipid colonisation and consumption. \\


Determining the exact concentration of lipids which elicits droplet formation and hence decrease floc settleability is essential for potentially monitoring wastewater influent and pre-empting failure of the remediation process. An experimental system to analyse the microbiome change with extended exposure to lipids was established. Future research should focus on 0.001 - 1 \% GT addition to mimic sludge behaviour in treatment plants. Highly abundant organisms which were affected by lipid enrichment were analysed with 16S rRNA DGGE, followed by sequencing and characterisation. Approximately 92 \% of the identified microbes belonged to the phylum Proteobacteria, which is well known for using AHL mediated gene expression, including for lipase expression. Describing AHL mediated gene expression systems in isolates and assessment whether they regulate lipase activity should be conducted, possibly by \emph{luxI} knockout mutant construction.\\

Lipid droplet colonising isolates were screened with a LuxR biosensor for AHL production where 3 out of 5 isolates elicited a response from the biosensor. These isolates showed a higher AHL production when cultivated in the presence of lipid than without and the identity of the pink pigment produced by \emph{Pandoraea} sp. cultured with lipids should be investigated.
\newpage 
\thispagestyle{plain}
\bibliographystyle{plain}
\bibliography{thesis_ref.bib}

\newpage 
\FloatBarrier
\section{Supplementary Material}
\begin{figure}
\includegraphics[scale=0.7]{Agr_tr.png}
\caption{Further enrichment process of aggregate from primary enrichment (t1) to secondary (t2) and tertiary (t3) enrichments. Planktonic phase dispersed from aggregate was transfered to t3 seperately.}
\end{figure}
\begin{figure}
\includegraphics[scale=0.7]{EM_tr.png}
\caption{Further enrichment process of emulsion from primary enrichment (t1) to secondary (t2) and tertiary (t3) enrichments.}
\end{figure}

\FloatBarrier
\begin{table}
\caption{LB10 low yeast}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (L) \\
\hline
 Agar, optional & 12 g  \\
Bacto-tryptone    & 10 g  \\
   NaCl   & 10 g  \\
    Yeast extract    & 1.25 g \\
  \hline
\end{tabular}
\end{table}


\begin{table}
\caption{Lipase assay substrate solution, component A at 1x}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (w/v) \\
\hline
 p-nitrophenyl palmitate   & 0.3 \% \\
 Isopropanol &  \\
 \hline
\end{tabular}
\end{table}

\begin{table}
\caption{Lipase assay substrate solution, component B at 9x}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (w/v) \\
\hline
 Gum arabicum & 0.1 \% \\
  Sodium deoxycholate & 0.2 \% \\
  Phosphate buffer, 50 mM pH 8   &  \\
  \hline
\end{tabular}
\end{table}

\begin{figure}
\includegraphics[scale=0.95]{EPS_AS.PNG}
\caption{EPS associated lipase activity of all replicates unamended activated sludge over time.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.15]{EPS_GT.PNG}
\caption{EPS associated lipase activity of all replicates of activated sludge amended with 2 \% glyceryl trioleate over time.}
\end{figure}

\begin{figure}
\includegraphics[scale=0.95]{SN_AS.PNG}
\caption{Lipase activity of the supernatant fraction of all replicates in unamended activated sludge over time.}
\end{figure}
\begin{figure}
\includegraphics[scale=1]{SN_GT.PNG}
\caption{Lipase activity of the supernatant fraction of all replicates in activated sludge amended with 2 \% glyceryl trioleate over time.}
\end{figure}

\begin{figure}
\includegraphics[scale=0.95]{BIOM_AS.PNG}
\caption{Cell membrane associated lipase activity of all replicates unamended activated sludge over time}
\end{figure}

\begin{figure}
\includegraphics[scale=0.95]{Biom_GT.PNG}
\caption{Cell membrane associated lipase activity of all replicates of activated sludge amended with 2 \% glyceryl trioleate over time.}
\end{figure}
\FloatBarrier

\begin{table}
\caption{Lipid coloniser medium}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (L) \\
\hline
Agar, optional  & 15 g  \\
  Di-potassium phosphate  & 0.3 g \\
   Glyceryl Trioleate   & 10 ml \\
      Magnesium sulphate  & 0.024 g  \\
       Proteose peptone   & 0.5 g \\
          Starch  & 0.5 g \\
           Yeast extract   & 0.5 g \\
  \hline
\end{tabular}
\end{table}


\begin{table}
\caption{Phosphate buffered saline, pH adjusted to 7.4}
\begin{tabular}{  p{6.9cm} | p{6.9cm}  }
\hline
Component & Concentration (L) \\
\hline
Di-sodium phosphate  & 1.44 g \\
  Monopotassium phosphate  & 0.24 g  \\
    Potassium chloride  &  0.2 g  \\
      Sodium chloride  & 8 g  \\
  \hline
\end{tabular}
\end{table}

\begin{figure}
\includegraphics[scale=1.1]{Std_curve.PNG}
\caption{Standard curve constructed from fluorescence of LuxR bioassay in response to 3-oxo-hexanoyl-homoserine lactone at concentrations of 0.5, 1 and 2.5 nM.}
\end{figure}
\FloatBarrier

\begin{figure}
\includegraphics[scale=.95]{SMary_process.png}
\caption{Process of wastewater remediation employed by St. Mary's wastewater treatment plant \cite{stmarys}.}
\end{figure}
\FloatBarrier

\begin{table}
\caption{XS buffer}
\begin{tabular}{  p{6.9cm} | p{6.9cm}  }
\hline
Component & Concentration (L) \\
\hline
 4 M Ammonium acetate & 200 ml  \\
  0.45 M EDTA  & 40 ml \\
    20 \% SDS  & 50 ml \\
    1 M Tris-Hydrochloride pH 4    & 100 ml  \\
        Potassium ethyl Xanthogenate  & 10 g  \\
  \hline
\end{tabular}
\end{table}


\end{document}