\documentclass[11pt]{article}

\usepackage{placeins}
\usepackage[nottoc,numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\linespread{1.3}

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\setlength{\parindent}{0pt}

\title{\textbf{Quorum-sensing influence on lipase production in activated sludge}}
\author{Anna Liza Kretzschmar\\
        z3218219\\
        Supervisor: Mike Manefield}
\date{}
%\usepackage{cite}
%something in reamble is fucked.. independent of line 17

\begin{document}

\maketitle
\section{Abstract}
Treatment of wastewater is essential for the function of our civilisation. The biological aspect of wastewater treatment plants employs microbial flocs to oxidise organic compounds in the waste influent, specifically in activated sludge. This is combined with chemical and physical remediation strategies to yield remediated effluent that can be released into the environment without posing a pathological or environmental threat. 
It is established that municipal wastewater activated sludge comprises a core microbiome that is adaptable to deal with fluctuating wastewater influent. When the influent contains a high lipid influent, the flocculation process is prevented and the biological aspect of wastewater remediation stalls. The mechanism underlying this phenomenon is not known. This study seeks to establish an experimental system to further the understanding of this knowledge gap to breach the gap to preventing system failure.
The change in microbial composition due to extended lipid exposure is assessed using denaturing gradient gel electrophoresis as well as change in lipase expression. Furthermore lipid colonising microbes are isolated and screened for \emph{N}-acyl-L-homoserine lactone activity to assess a link with lipase expression. 
\newpage
\section{Acknowledgements}
xx
\newpage

\tableofcontents

\newpage

% remove preamble incl \begin\end{doc} from \input files.


\section{•}
\listoffigures

\newpage
\section{•}
\listoftables

\newpage
\section{List of Abbreviation}
\begin{table}
\begin{tabular}{  p{4cm} | p{8cm} }
\hline
AHL & \emph{N}-acyl-L-homoserine lactone \\
BLAST & Basic Local Alignment Search Tool  \\
bp & Base pair \\
C & Carbon \\
D & Day \\
$\,^{\circ}\mathrm{C}$ & Degrees celsius \\
DGGE & Denaturing gradient gel electrophoresis \\
DNA & Deoxyribonucleic acid  \\
 EPS & Extracellular polymeric substances \\
 GC & Guanine and cytosine pair \\
GT & Glyceryl trioleate \\
L & Litre \\
LB & Lysogeny broth \\
LC & Lipid coloniser \\
$\mu$l & Microlitre \\
$\mu$m & Micrometre \\
min & Minute \\
ml & Millilitre \\
mM & Micromolar \\
M & Molar \\
N & Nitrogen \\
NCBI &  National Centre for Biotechnology Information \\
OHHL & 3-oxo-hexanoyl-homoserine lactone \\
P  &  Phosphate \\
PBS & Phosphate buffered saline \\
PCR & Polymerase chain reaction \\
 p-npp & p-Nitrophenyl palmitate \\
rcf & Relative centrifugal force \\
rRNA & Ribosomal ribonucleic acid \\
sec & Second \\
t & Time \\
 t2 & Secondary transfer \\
t3 & Tertiary transfer \\
U & Enzyme activity \\
V & Volt \\
w/v & Weight per volume \\
XS & xanthogenate-SDS \\
  \hline
\end{tabular}
\end{table}

\newpage
\section{Introduction}
Wastewater treatment plants are common in the global urban setting and a necessity for the continued function of our society. In such systems, fluid waste entering the plant, is purified largely through microbiological processes and the effluent can then be released into the environment. It is necessary to understand these microbiological processes underlying wastewater treatment in order to optimise the functionality of the plant, as the global human population expands and strains the Earth's finite water resources. In 1914 Arden and Lockett pioneered the development of wastewater plants using activated sludge \cite{ardern1914experiments}.

Activated sludge consists of complex microbial communities living in cellular aggregates called flocculates (flocs). The formation of flocs in activated sludge is crucial for global wastewater treatment. Without floc formation activated sludge would not settle under gravity and the treated water could not be decanted or separated from the microbial biomass. Despite the importance of the formation of activated sludge flocs to human civilization, very little is known about how they arise, how they mature and how they ultimately lose activity as sludge age increases. These major scientific knowledge gaps prevent the rational engineering of microbial aggregates for wastewater treatment. The present study represents a stepping-stone towards addressing these knowledge gaps on floc formation through development of an experimental model of floc formation based on the interactions between activated sludge bacteria and lipids. 

The microbes in activated sludge flocs oxidise sewage particulates and soluble compounds in filtered waste influent \cite{Price_95}. Flocs are a form of detached biofilm called a floc, however what seeds floc formation is still unknown. Inter bacterial communications induce transcriptomic change via a system called quorum-sensing, which plays a major role in a consortium's interaction\cite{parsek2005sociomicrobiology}. Hence our understanding of the phenotypes impacted by this communication is essential for optimising the process of water remediation \cite{singh2006biofilms}
%check comments - section needs to be moved, rejiggered and expanded.

\subsection{Wastewater treatment is essential to civilisation}
The development of the current municipal wastewater treatment system has evolved over millennia to combat the waste arising from congregated human habitation. The Mesopotamian empire, from between 3500 to 2500 BC, built connections from some homes to storm water drains to remove wastes, while the Babylonians developed clay pipes to fulfil the same purpose \cite{lofrano2010}. The logical necessity to develop such a system to prevent disease and minimise environmental impact globally and has been continuously improved. In 1914 Arden and Lockett pioneered the development of activated sludge, which is one of the most commonly employed biotechnologies today \cite{jenkins2004manual,muchie2010bioremediation}


Current plants combine physical, chemical and biological treatment strategies to the polluted influent in order to reduce environmental and health impacts. Hence these treatments are in place to minimise the presence of N, P and C, reduce the biological oxygen demand and chemical oxygen demand and filter out harmful substances and pathogens \cite{mayhew1997low}.
Water quality is in part assessed by the dissolved oxygen concentration. The parameters for this assessment are measured by the concentration of substances present that can be oxidised chemically or biologically to inorganic end products \cite{pisarevsky2005chemical}.

\subsubsection{\emph{Microbial floc formation underlies successful wastewater treatment}}
Initially, large solids in the influent are either filtered out or broken down into smaller particulates to prevent blockages or excessive mechanical strain during the process. During the primary treatment phase, the influent is deposited in a large settlement tank where the majority of the particles settle to the bottom due to gravitational force. The weir controlled outlet flow is directed to the aeration tank for secondary treatment. The activated sludge in the aeration tank comprises the biological aspect of the treatment. It is an oxygenated system where the influent is incubated with microbial consortia, predominantly aggregated as flocs, which live by oxidising the soluble and fine sewage in the mixed liquor \cite{mayhew1997low}. The suspension is drawn off into a final settlement tank where the sludge flocs settle at the bottom and the treated water is then discharged. Depending on the required effluent quality, tertiary treatment may also be required to remove residual solids, compounds or planktonic bacteria that could pollute or enrich the effluent destination \cite{Price_95}. The all important sludge flocs are either recycled back into the mixed liquor to do more work or discarded.
% knowledgegap
% jesus fucking christ, more references

Bioflocculation is essential for wastewater treatment, as the settlement of the flocs is key to drawing off remediated water. Flocs are often compared with biofilms. Biofilms are traditionally thought to constitute an aggregated community of microbes at the liquid-solid interface surrounded by secreted extracellular polymeric substances (EPS), consisting of polysaccharides, nucleic acids, lipids and proteins \cite{wingender1999}, which represents 85-90 \% of biofilm dry weight \cite{Frolund_96}. Flocs are traditionally thought to be biofilms lacking the surface substrate.


The composition of biofilms are dependent on the participating species coupled with prevailing environmental conditions, and are sites of rapid adaptation through continuous natural selection for survival \cite{boles2008,matz2005,palmer2001}.  
The EPS offers the advantages of defence against predation by bacteria \cite{rao2005} and protozoa \cite{matz2005}; increased resistance against some chemicals such as antibiotics and hydrogen peroxide \cite{burmolle_06}; a stable matrix for the cells to reside in \cite{Flemming_10}; and to immobilise extracellular enzymes offering proximity while protecting them from proteolysis and conferring resistance to higher temperatures \cite{wingender2002extracellular,Flemming_10,skillman1998}.

\subsubsection{\emph{Microbial composition determines wastewater treatment efficiency}}
To optimise the efficiency of wastewater treatment, understanding the composition of the microbial communities at work in activated sludge and how their interactions influence their capabilities of remediation is essential \cite{daims2006}.
Single celled organisms from all three domains of life reside in this system including protozoa, nematodes and fungi in amongst the more abundant bacteria, archaea and viruses. All of these play a role in degradation of solids and nutrient cycling and hence in wastewater remediation \cite{muchie2010bioremediation}. The domain most active in remediation are the bacteria \cite{spellman2008handbook}.


The advance of community analysis techniques, which exclude the necessity to culture microbes in the laboratory, has made it possible to investigate the composition of microbial communities, including in activated sludge. The most frequent target for operational taxonomic unit (OTU) analysis centres around the 16S ribonucleic acid (rRNA) gene, which is then compared to expansive 16S databases such as NCBI. Several techniques can be utilised around 16S rRNA, such as constructing a 16S rRNA gene library \cite{McGarvey_04}, analysing ribosomal intergenic spacer sequences \cite{Yu_01}, 16S-restriction fragment length polymorphism \cite{Gilbride_06} and comparing community structures via denaturing gradient gel electrophoresis (DGGE) \cite{Hesham_11} or pyrosequencing \cite{wang2012pyrosequencing}.


In wastewater treatment plants, the microbial community composition is strongly dependent on what type of waste influent the microbes are exposed to, as well as the operational settings, such as chemical oxygen demand \cite{Gilbride_06,wang2012pyrosequencing,hu2012microbial}. Municipal wastewater treatment plant influent varies in composition \citep{henze2002wastewater}.


\begin{sidewaystable}[!htbp]
\caption{Studies characterising core microbiomes found in activated sludge.}
\begin{tabular}{ | l | l | p{4.5cm} | p{7cm} | l | }
\hline
Study & Method & Sample size & Core microbiome composition & Refernce\\
\hline
Hesham et al. & DGGE & One plant with two different operational modes over 6 months, China & 2x \emph{$\alpha$-} \& 2x \emph{$\beta$- proteobacteria}; 3x \emph{bacteroidetes}; 2x \emph{actinobacteria}; 2x \emph{firmicutes} & \cite{Hesham_11} \\
\hline
Wagner et al. & DGGE & 750 16S rRNA sequences from wastewater treatment plants and reactors & \emph{$\alpha$-, $\beta$-} \& \emph{$\gamma$- proteobacteria}; \emph{bacteroidetes};\emph{actinobacteria} & \cite{Wagner_02} \\
\hline
Wang et al. & Pyrosequencing & 14 plants \& pilot/benchtop operations in China & 21 - 53 \% Proteobacteria, where \emph{$\beta$-, $\alpha$-} \& \emph{$\gamma$-proteobacteria} were present 21 - 52 \%, 7 - 48 \% and 8 - 34 \% respectively & \cite{wang2012pyrosequencing} \\
\hline
Hu et al. & Pyrosequencing & 12 plants where 4 were A/A/O, China & In A/A/O proteobacteria dominated, with up to 2:1 ratio to bacteroidetes. In other types of plants the dominant phylum proteobacteria was occasionally replaced by bacteroidetes & \cite{hu2012microbial} \\
\hline
Ranasinghe et al. & Pyrosequencing & 12 plants over two years, Japan & Proteobacteria represented 38 \% of total assigned reads where 15 \% belonged to $\beta$-proteobacteria & \cite{ranasinghe2012revealing} \\
\hline
Xia et al. & Microarray & Three plants in China and two in USA & Proteobacteria wast the dominant phylum at 50 \% to 62 \%, where $\gamma$-proteobacteria represented  31 \% to 38 \% and $beta$-proteobacteria 30 \% to 35 \% & \cite{xia2010bacterial} \\
\hline
\end{tabular}
\end{sidewaystable}


Table 1 summarises methods used to analyse activated sludge microbial communities across geographical and operational settings, showing a core microbiome. Pyrosequencing and microarrays studies give a snapshot of the microbial community, including low level abundance organisms \cite{ranasinghe2012revealing}. DGGE gives an insight into the change in abundance of dominant organisms active under certain conditions and over time. The wealth of information available suggests community composition varies widely between different plants, and hence there is a need to describe treatment plant details specific to the present study.
\FloatBarrier

\subsection{Lipids in wastewater need to be degraded by lipase}
Municipal wastewater contains lipids, at a concentration between 40 and 100 mg/m$^{3}$ \cite{Forster_92} representing 31 \% of the chemical oxygen demand of domestic wastewater \cite{Raunkjaer_94}. Oleic acid is the most common fatty acid found in native as well as used olive and sunflower oils, which are commonly used in domestic settings \cite{haba2000isolation}.  An excess in lipid content has been shown to inhibit flocculation and promote the growth of filamentous bacteria, which are linked to sludge bulking - an event that induces foaming and reduces effluent quality \cite{Forster_92}.
%statement of lipase in lipid degr in sludge

Due to the hydrophobic nature of lipids, they may be associated with other particulates in the activated sludge or form a separate phase. In the former case, membrane or EPS bound lipases are likely expressed for degradation, whereas lipids in a separate phase could be targeted with extracellular lipases. It is advantageous for consortia to express lipase to utilise this carbon source and and lipase is commonly expressed in activated sludge \cite{gessesse2003lipase}. Investigations as to the most effective strategy to remove lipids in wastewater have been conducted extensively. Strategies including pretreatment with large concentrations of lipases, introducing pure or mixed lipid degrading cultures and exposure activated sludge \cite{Wakelin_97}. %Activated sludge showed the highest grease removing activity - non funct \cite{Wakelin_97}. 
%need more refs - extensive doesnt equal singular ref.

%While the impact of high lipids influent on activated sludge settleability is documented, the mechanism that facilitates this change is not understood. As the settleability of the biologogical components is key to successful treatment of wastewater, it is essential to understand this process so it can be improved and monitored. -- fit since change?

\subsubsection{\emph{Lipase are extracellular degradation units}}
Triacylglycerol acylhydrolase (Enzyme Class 3.1.1.3) catalyses the reversible hydrolysis of triacylglycerols by targeting the ester bonds that attach the fatty acid side chain to the glycerol backbone. The model lipid Glyceryl trioleate (GT) used in this study consists of three oleic acid side chains. Genes encoding for lipase will be denoted as \emph{lip}.
The activity of lipase is highly chemo-, enantio- and regioselective. Common to this class of enzyme is an extruding loop which extends over the active site, called the lid. The aggregated, hydrophobic nature of the substrate causes interfacial activation, which causes the lid to move to reveal the underlying active site to the substrate \cite{derewenda1992,van_Tilbeurgh1993}. These features contribute to lipases being highly efficient biocatalysts in organic chemistry and their extensive representation in the industrial setting. 
They are found within the textile, detergent, food processing, leather, pharmaceutical, pulp and paper industries \cite{hasan_06}; are essential for the production of fine chemicals such as flavours, cosmetics, agrochemicals and therapeutics \cite{jaeger2002}; and are under investigation for their potentially to produce biodiesel as a replace fossil fuels \cite{hasan_06,iso2001}. 


Microbial lipases are secreted into the extracellular space, where they can catalyse the liberation of fatty acids at the lipid-water interface, which are then absorbed and utilised as a C source. In Gram negative bacteria, including proteobacteria, the lipase zymogen passes two membranes separated by the periplasm, before secretion and folding into it's active conformation \cite{bos2007,michel2009}. There are two pathways by which this can occur: the type I or type II secretory pathways. 


\subsection{The role of intercellular signalling in activated sludge flocs and regulation of lipase production}
Bacteria monitor their environment through the production of and response to specific metabolites. Such molecules occupy the intercellular space in microbial aggregates and at elevated concentrations trigger a change in transcriptomic regulation depending on their concentration. This allows the consortia, whether multi-species or cross-kingdom \citep{williams2007quorum}, to engage in a concerted effort that resembles multi-cellularity \cite{kjelleberg2002}. While there are several bacterial communication systems, this report focuses on \emph{N}-acyl-L-homoserine lactone (AHL) dependent gene expression orchestration. The bacteria that actively participate in this style of communication are predominately from the phylum proteobacteria, which are diverse and well represented in the environment, especially so in municipal activated sludge \cite{Hesham_11,Wagner_02}.  


The machinery facilitating AHL-mediated gene expression consists of two major constituents, - LuxR and LuxI, or their respective homologs. The latter is an AHL synthase, while the former is a receptor that binds AHLs at a threshold concentration. The complex functions as a transcription factor for species specific functional genes. These genes typically play a role in bacterial cell adhesion to surfaces, biofilm formation and the production of extracellular enzymes \cite{Flemming_10}.
These traits are usually under further control outside of AHL-mediated transcription factors as well as potential cross regulation between several QS systems within the organism \cite{juhas2005}.



\subsubsection{\emph{AHL responsive gene circuits are found in Proteobacteria}}
The proteobacteria is the most prevalent phylum containing AHL dependent QS circuits \cite{gelencser_12}. In 2008 Case et al. compiled a list of isolates containing \emph{luxRI} homologs from 512 completed genomes on the NCBI platform, where 13 \% of those contained both \emph{luxR} and \emph{luxI}. These were exclusively from the phylum \emph{Proteobacteria} and were found in 26 \% of all finished proteobacterial genomes \cite{case_08}.
The type II secretion system is only found in this phylum, usually associated with delivering QS influenced virulence factors and enzymes, such as lipase, into extracellular space \cite{sandkvist2001}. 
% explain lux box
The frequency in which Case et al. found \emph{luxR} and \emph{lux} in the proteobacteria \cite{case_08} coupled with the prevalence of this phylum in activated sludge \cite{Wagner_02,Hesham_11}, raises the fascinating proposition that AHL mediated gene expression plays a role in activated sludge floc formation.



\subsubsection{\emph{AHL mediated lipase production in proteobacteria}}
The influence of AHL mediated gene expression on lipase production has been documented in several genera, discussed in this section with their respective classification and \emph{luxRI} homologs.


\emph{\underline{Burkholderia spp.}} 
\\In \emph{Burkholderia cepacia}, Lewenza et al. have demonstrated that the expression of \emph{lipA} is linked to CepR, the LuxR homolog \cite{lewenza1999}, which is in direct opposition to the results obtained by Huber et al. \cite{huber2001}. In Lewenza et al.'s study, lipase production in \emph{cepR} mutants was reduced by up to 45 \%. However the supplementation of CepR via a plasmid did not restore wild type lipase expression, which suggests that the link between \emph{cepR} and \emph{lipA} is other than transcriptomal activation. The lack of a \emph{lux} box within \emph{lipA}, to which the CepR:AHL transcription factor would bind, supports the postulation that \emph{lipA} is situated downstream of \emph{cepR} and under same operon control \cite{lewenza1999}. 



\emph{B. glumae}, an emerging rice pathogen, regulates it's lipase production with AHL mediated gene expression and lipase expression is innately linked with the pathogenic phenotype. TofR, the LuxR homolog, activates \emph{lipA} transcription \cite{devescovi_07}. 


A study by Suarez-Moreno et al. which compared the plant associated species \emph{B. kururiensis}, \emph{B. unamae} and \emph{B. xenovorans} found no discernible impact of disabling the respective LuxR homologs on lipase production, in agreement with Huber et al. \cite{huber2001,suarez2010}. For these species tested, which live in symbiosis with the plant host, it follows that lipase expression, which is generally considered a virulent attribute, should not be population density dependent as increasing cell density would imply successful symbiosis. 



In \emph{B. vietnamiensis}, the literature on how AHLs influence lipase secretion are contradictory. Conway et al. recorded no detectable influence of AHLs on lipase production whether in the wild type or an AHL synthase (\emph{bviI}) deficient mutant \cite{conway_02}. On the other hand Ulrich et al. conducted a they found three separate AHL synthases referred to as \emph{btaI123} and five separate \emph{luxR}-like transcriptional regulators named \emph{btaR12345}, which is a nomenclature system distinct to other nomenclature reported here. They found that \emph{btaR1}, \emph{btaR3}, \emph{btaR4} and \emph{btaR5} acted as repressors on lipase production \cite{ulrich2004}. \emph{btaI1} and \emph{btaI3} inhibit lipase production while \emph{btaI2} enhances it. Even though the data in regard to lipase activation and repression implies that \emph{luxR123} and \emph{luxI123} represent complete circuit systems, the evidence was not compelling to Ulrich et al. to draw that conclusion without further investigation \cite{ulrich2004}.
It is unclear whether the studies by Conway et al. and Ulrich et al. refer to the same AHL-mediated systems, if \emph{bviRI} and \emph{btaRI} are synonymous. If this is the case, a possible explanation for Conway et al. not detecting any impact of the \emph{bviR} mutant on lipase production could be given by them investigating what Ulrich et al. termed \emph{btaR2}, in which they also reported no discernible impact on lipase production. 
The complicated and cross-regulatory system surrounding lipase expression within the \emph{Burkholderia} genus may indicate the complexity of lipase production in other proteobacteria, including the role that AHLs play to influence the expression.
\\
\\ \emph{\underline{Pseudomonas spp.}}
\\ \emph{P. aeruginosa} has two AHL dependent systems: \emph{las} and \emph{rhl} which each represent \emph{lux} homologs and respond to separate AHLs. The LasR:AHL complex controls \emph{rhlIR} expression, while RhlR activates \emph{lipA} transcription. Hence RhlR directly influences lipase production while LasR's influence is indirect. However a third non-AHL mediated system, called the P. aeruginosa quinilone signal, regulates the other two systems \cite{juhas2005}. 


The AHLs are also influenced by other factors such as GacA, which promotes the AHL regulatory cascade \cite{reimmann1997}, as well as RsmA which degrades AHLs \cite{pessi2001}. However even though RsmA degrades AHLs, it positively affects lipase production as it binds to the \textit{lip} mRNA and stabilises the transcript to reduce the transcript's rate of degradation. RsmA in turn is regulated by RsmZ, whose suppression of RsmA therefore negatively affects lipase production \cite{heurlier2004}. 
\\
\\ \emph{\underline{Xenorhabdus nematophilus}} 
\\The insect pathogen \emph{Xenorhabdus nematophilus} resides in the intestinal tract of the symbiotic host \emph{Steinernema carpocapsae}. It aids the host in reproducing while the nematode represents a reservoir for the bacterium to infect insects \cite{herbert2007}. Lipase production is upregulated by  AHL mediated transcriptional activation \cite{dunphy_97} if \emph{flhDC}, which is the flagella master operon, is also expressed \cite{rosenau2000}. Another regulator is Lrp, which controls LrhA, and positively impacts lipase production and motility \cite{richards2008} 
The link between lipase expression, motility and virulence is established \cite{givaudan_00}, which suggests that the lipase contributes to the feeding mechanism while infecting rather than being a virulent agent itself \cite{richards2010}. Considering the organism's life style, which involves the necessity to leave the host in order to access the target, the controls associated with lipase expression are consistent with the requirement to move to attack the food source before releasing extracellular enzymes.
\\
\\ \emph{\underline{Serratia spp.}} 
\\Lipase production in both \emph{S. marcescens} and emph{S. proteamaculanss} is influenced by AHL- mediated gene expression \cite{horng2002,shibatani2000,christensen_03}. Furthermore, in \emph{Serratia marcescens} surface attachment and biofilm formation in is influenced by the same mechanism \cite{labbate2007} and all contribute to the virulent phenotype \cite{hejazi_97}.
\FloatBarrier

\subsubsection{\emph{Samples were aquired from activated sludge in St. Mary's wastewater treatment plant}}
St Mary's municipal waste water treatment plant is where sample collection for this project took place. It is located in St Mary's in western Sydney serving a population of app. 160000 in a catchment area of 84 km$^{2}$ and discharges into the Hawkesbury-Nepean River. On a daily basis the plant processes about 35 million litres of wastewater. Wastewater can enter two influent streams - stage 1 and 2 or stage 3. The streams are subjected to differing primary treatments, however secondary and tertiary treatment is the same. The biological treatment is part of the secondary treatment stage, as can be seen in figure xx. The biological process is conducted in three stages (A/A/O): 1. Anaerobic zone, where microbes take up carbon and release phosphates; 2. Anoxic zone, where carbon is consumed and nitrates are released as nitrogen gas; 3. Aerobic zone, where nitrification of ammonia occurs and organic matter increasing the oxygen demand is reduced (Figure 1) \cite{stmarys}.
\begin{figure}
\includegraphics[scale=0.9]{SMary_process.png}
\caption{Wastewater processing as employed by St Mary's treatment plant \cite{stmarys}}
\end{figure}
\FloatBarrier
\newpage
\section{The development of an experimental model of activated sludge floc formation based on lipid as a substrate}
Major knowledge gaps exist in the scientific literature regarding the mechanistic processes governing floc formation and activity. For example, it is unclear how floc formation is initiated, how enzymatic activity is regulated within flocs, what role intercellular signalling plays in floc formation and activity or why flocs lose activity as activated sludge age increases. Given how important activated sludge floc formation and activity are to human civilization it is surprising such knowledge gaps exist. A promising approach to addressing these knowledge gaps is through development and interrogation of experimental models of floc formation. The absence of such models curtails scientific endeavour in the context of wastewater treatment.  
The present study seeks to develop an experimental model system based on interactions between activated sludge bacteria and lipids. The choice of lipids as a substrate is justified by the high lipid content of wastewater and the high lipase activity in activated sludge. Further, the link between proteobacteria, which are abundant in activated sludge, and AHL dependent regulation of lipase activity offers an opportunity to explore the role of AHL mediated gene expression in activated sludge floc formation. The lipid chosen to develop the model is glycerol trioleate, which is the most abundant lipid in vegetable oil. The study is by necessity developmental with very little experimental information available as a starting point.

This project directly addresses the hypothesis that the lipid glycerol trioleate serves as a surface substratum for activated sludge floc formation. 


Specifically, the aims are:
\begin{enumerate}
\item To document the partitioning behaviour of glycerol trioleate in activated sludge.
\item To document spatial relationships between glycerol trioleate and activated sludge bacteria.
\item To describe the bacterial community composition of flocs in the presence of glycerol trioleate.
\item To isolate and identify glycerol trioleate degrading bacteria from activated sludge.
\item To test glycerol trioleate degrading bacteria from activated sludge for AHL production.
\end{enumerate}
\newpage
\section{Methods}
\subsection{Enrichment cultures}
Enrichment cultures consisted of 50 ml activated sludge collected from St. Mary's wastewater plant in each of eight identical 250 ml Boeco flasks. The experiment was conducted in quadruplicates of 2 \% GT enrichment (Glyceryl trioleate, Sigma-Aldrich T7140-10G) and no treatment. The cultures were incubated at $37\,^{\circ}\mathrm{C}$ on a shaker set at 140 rpm. Samples were taken over a 25 day period and consisted of 0.5 ml for Deoxyribonucleic Acids (DNA) extraction and 0.5 ml for lipase assays. The 1 ml removed was replaced with sterile, syringe filtered (0.22 $\mu$m Millipore) activated sludge supernatant. 
A trial enrichment run was conducted 1 \% GT enrichment rather than 2 \%.

\subsubsection{\emph{Microscopy}}
Lipid droplets were removed with tweezers and washed in PBS in two subsequent crystallizing dishes. After transferral to a glass slide, the droplet was stained with 5 $\mu$l SybrGreen, a DNA binding dye, and covered with a cover slide. The droplets were imaged with an Olympus Fluorescence Microscope which elicits a visual response from DNA bound dye.

\subsubsection{\emph{DNA extraction}}
DNA was extracted by the amended xanthogenate-SDS method \cite{tillett2000xanthogenate}. The samples for DNA extraction were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube, supernatant removed and the pellets frozen . In the case of DNA extraction from GT droplets, 0.5 ml from GT enrichment cultures were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube and the pellet was decanted into sterile MilliQ water in a crystallizing dish. Semi-solid lipid droplets were removed with tweezers and washed in a second crystallizing dish, transferred to a 1.5 ml eppendorf tube and frozen. Thawed pellets were re-suspended in 0.2 ml phosphate buffered saline (PBS, as per Supplementary Material, Table 9).


For each sample, 0.9 ml phenol solution and 0.9 ml XS-buffer (as per Supplementary Material, Table 10) was  heated to $70\,^{\circ}\mathrm{C}$ in a 2 ml microcentrifuge tube in a water bath for 5 min. The 0.2 ml cell suspension was added to the 2 ml tubes, mixed by inversion and left at $70\,^{\circ}\mathrm{C}$ for 15 min. Then the samples were vortexed for 10 sec and frozen for 2 min, followed by centrifugation at 16.1 rcf for 5 min. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 0.9 ml phenol:chloroform:isoamyl alcohol mix and centrifuged at 16.1 rcf for 5 min. This step was repeated. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 1 ml ice cold isopropanol, 50 $\mu$l 3 M sodium acetate pH 7.5 and 1.3 $\mu$l glyco blue, mixed by inversion and frozen over night. Subsequent centrifugation at 16.1 rcf for 60 min at $0\,^{\circ}\mathrm{C}$ was followed by removal of isopropanol and addition of 1 ml 80 \% ethanol. Centrifugation at 16.1 rcf for 20 min at $0\,^{\circ}\mathrm{C}$ followed, then ethanol was removed and samples were left to air dry for 30 min before re-suspension in 0.1 ml molecular water.
\\ 
Extracted DNA concentration was analysed via Nanodrop (NanoDrop ND-1000 Spectrophometer) as well as Qubit (Invitrogen, Qubit 2.0 Flourometer), as per standard protocol for broad range DNA detection.

\subsubsection{\emph{Lipase assay}}
The lipase assay (Figure 2) was adapted from Christensen et al \cite{christensen_03}. From each replicate 0.5 ml was taken and centrifuged at 16.1 rcf for 5 min or until phases separated. The supernatant was removed and syringe sterilised (0.22 $\mu$m Millipore filter) to test for suspended EC lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water with the addition of 0.1 ml Zirkonium beads. The samples were bead beaten for three 45 sec cycles and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and and syringe sterilised (0.22 $\mu$m Millipore filter) to test for EPS associated lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water to test for membrane bound lipase.


\begin{figure}
\begin{center}
\includegraphics[width=.65\textwidth]{lipase_assay.png}\\
\includegraphics[width=.65\textwidth]{lipase_assay2.png}
\caption{Fractionation of each sample into three fractions, each subjected to lipase activity assay.}
\end{center}
\end{figure}

For each sample 0.1 ml were incubated with 0.9 ml of  p-nitrophenyl palmitate (p-npp) containing substrate solution (as per \textbf{Appendix 7.1}) for one hour. Lipase cleaves the p-nitrophenyl group off the plamitate which can be quantified at an absorbance of 410. The p-nitrophenyl concentration is directly correlative with lipase activity. The reaction is terminated by alkaline pH inactivation of lipase with 1 ml 1 M sodium carbonate. Samples were centrifuged at 16.1 rcf for 5 min. From each terminated reaction, 0.3 ml supernatant was transferred to 96 well micro titre plate and the absorbance read at a wavelength of 410 with a Microtitre reader (Molecular Devices, Spectra Max 340).
\FloatBarrier
\subsection{community analysis of enriched cultures}
DNA was extracted from enrichment culture isolates, verified with DGGE polymerase chain reaction (PCR). The DGGE PCR assay was set up with the primers 357FGC and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 15.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 2 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.
Bands of interest were selected from the DGGE and excised and re-run on DGGE PCR to test their integrity, and if satisfactory underwent sequencing PCR. 

\subsubsection{\emph{Further enrichment}}
To investigate the change of microbial composition with extended exposure to GT, secondary (t2) and tertiary (t3) enrichments were conducted by transferral into 25 ml filtered sludge supernatant (0.22 $\mu$m Millipore filter) and 2 \% GT. The t2 were conducted with an aggregate and 5 \% emulsion. t3 consisted of a further transferral of the aggregate, 5 \% of the t2 emulsion and 5 \% of the supernatant from the t2 aggregate.

To confirm the difference in morphology of activated sludge in the presence of 1 vs. 2 \% GT rather than the cause being seasonal variation in the microbial community, a comparative enrichment was conducted. DNA extracted after 7 days and processed for DGGE analysis.
 
\subsubsection{\emph{Denaturing Gradient Gel Electrophoresis}}
DGGE was conducted using the BioRad DCode system and protocol for 50 \% to 70 \% denaturing gradient on a 6.7 \% arcylamide gel. Electrophoresis was conducted at 75V and $60\,^{\circ}\mathrm{C}$ for 16.5 hrs in 1x TAE. The gel was stained with SybrGold for 20 min and imaged on a BioRad Gel Doc XR+ Imaging system and BioRad Image Lab software. Bands of interest were excised and suspended in 40 $\mu$l molecular water over night, which was used as template for nested PCR set up with 357F and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 12.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 5 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$. Products for sequencing were cleaned up with Zymo Research DNA Clean \& Concentrator-5 kit, following the PCR product protocol.

\subsubsection{\emph{Sequencing}}
The sequencing PCR assay, protocol from the Ramaciotti centre, was set up for the 357F primer at 20 $\mu$l as follows: 1 $\mu$l BigDye terminator V3.1, 20 - 50 ng PCR product, 0.32 $\mu$l primer, 3.5 $\mu$l 5x sequencing buffer and up to 20 $\mu$l with molecular water. The PCR protocol was the following: 26 cycles of denaturing phase: 10 sec at $96\,^{\circ}\mathrm{C}$, annealing phase:  at 5 sec $50\,^{\circ}\mathrm{C}$, extension phase: 4 min at $60\,^{\circ}\mathrm{C}$, followed by and storage at $4\,^{\circ}\mathrm{C}$.

The samples were cleaned up by addition of 5 $\mu$l of 125 mM EDTA and 60 $\mu$l 100 \% ethanol followed by vortexing and 15 min precipitation period. The samples were then spun at 14000 rcf for 20 min and supernatant was removed. After addition of 160 $\mu$l fresh 70 \% ethanol, samples were spun at 14000 rcf for 10 min and supernatant discarded. This step was repeated with addition of 80 $\mu$l 70 \% ethanol. The samples were dried in the dark and submitted to the Ramaciotti centre at UNSW for Sanger Sequencing.

\subsection{Lipid coloniser isolation}
Sludge isolates  were isolated from lipid droplets, washed twice in PBS, by three subsequent transfers on Lipid coloniser medium (LCM per Supplementary Material, Table 8) plates. Isolates were prepared for 16S rRNA PCR for the primers 27F and 1492R at 20 $\mu$l as follows: 10 $\mu$l EconoTaq Master mix, 7.4 $\mu$l molecular water, 0.8 $\mu$l of each primer and 1 $\mu$l template. The PCR protocol was the following: 2 min at $94\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 1 min at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $59\,^{\circ}\mathrm{C}$, extension phase: 1.40 min at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$, then prepared for Sanger sequencing by the Ramaciotti centre as previously outlined.
% incl diff primers and hene length, PCR protocol.

\subsubsection{\emph{Lipid coloniser behaviour in the presence and absence of olive oil}}
Isolates were pre-cultured in 25 ml LCM broth, either with or without olive oil (Mike - should i just exclude here that I ran out of GT?). 

\subsubsection{\emph{Screening lipid colonising isolates for AHL production}}
Pre-cultures were centrifuged at 16.1 rcf for 10 min and 20 ml supernatant was transferred to a 50 ml falcon tube. After addition of 20 ml Ethyl acetate with 0.01 \% glacial acetic acid, the mixture was agitated and left to separate. The top  layer was transferred to an 100 ml beaker. This was repeated twice and left over night for evaporation in a chemical fume hood. The AHLs were resuspended in 100  $\mu$l methanol and filter sterilised with PVDF Filter Vials (CP-ANALYTICA GmbH). 
 The reporter strain \emph{Escherichia coli} carrying the plasmid pJBA357 was pre-cultured in Lysogeny Broth (LB10 per Supplementary Material, Table 5) \cite{bertani1951studies}. The plasmid contains \emph{gfp}, which  is preceded by the \emph{luxR} promoter, activated by OHHL and the response is directly proportionate to the AHLs present.
The assay was conducted in triplicate with \emph{Aeromonas hydrophila} GC1 as the positive control, LB10 as a blank and 2.5 nm, 1 nm and 0.5 nm OHHL as a comparative reference. Per microtitre plate well 20 $\mu$l of extract was added and the methanol left to evaporate, followed by addition and mixing of 100 $\mu$l \emph{E. coli} in it's exponential growth phase. After 4 hrs of incubation OD600 was read to verify \emph{E. coli} growth as well as fluorescence by excitation at 485 nm and then emission at 535 nm.
\newpage
\section{Results}
\subsection{Bacteria colonise lipids in activated sludge}
It was essential to establish an experimental system and adapt protocols to observe the impact of lipids on activated sludge before a controlled enrichment experiment could be conducted. 

\subsubsection{\emph{change in morphology}}
Before conducting the enrichment experiment, the behaviour of activated sludge in  the presence of GT needed to be established (Aim 1). The trial was conducted with 1 \% GT  added to sludge replicates which was sampled every 7 days, as the expectation was that the enrichment would run for an extended period of time. GT formed a transparent and hydrophobic layer on the surface of the enrichment cultures, which was replaced with white droplets suspended below the culture surface within 7 days (Figure 3B). While varying in size, the droplets consistently decreased in diameter with increased incubation period until they were indistinguishable from the sludge after 21 days. Droplet formation decreased floc settlement for the duration of their presence. 


Due to the lipid being assimilated into the sludge within 3 sampling points, the sampling frequency was increased from 7 to every 3 days. To compensate, the actual enrichment experiment was conducted with 2 \% GT. The increase of GT concentration resulted in no droplet formation. Instead the cultures became increasingly viscous and lighter in colour (Figure 3A and 3C). No settlement of sludge flocs was observed, however denser, singular aggregates appeared in the viscous cultures. One of these was targeted for secondary enrichment by moving the aggregate into sludge supernatant, where the aggregate expanded and developed tendril like structures (Figure 3D).
\FloatBarrier
\begin{figure}
\includegraphics[scale=.77]{cultures.jpg}
\caption{Impact of GT addition to activated sludge cultures; A) the comparison between 2 \% GT enrichment and no treatment; B) lipid droplet formation in the presence of 1 \% GT in trial run; C) the lack of settleability of sludge after incubation with 2 \% GT; D) secondary enrichment by transferral of aggregate into sludge supernatant and 2 \% GT.}
\end{figure}
\FloatBarrier

\subsubsection{\emph{Microscopy}}
To establish the nature of the droplets and aggregates in 1 \% and 2 \% GT enrichments respectively (Aim 2), samples were isolated and stained with SybrGreen before imaging with epifluorescence microscopy (Figure 4). The fluorophore binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black. 
Figure 4 consistently shows that the biomass is closely associated with the lipid, rather than the intermediate water space and firmly attached due to the wash steps prior to staining.
While Figure 4A \& 4B show clusters of biomass around lipid, 4C, 4D, 4E \& 4F are a close up of those clusters which showcase smaller segments of lipids surrounded and colonised by lipids. These pictures are unclear because the cells occupy the space around the lipids at different levels. This spatial distribution and cell density is akin to biofilm formation. 
\begin{figure}
\includegraphics[scale=.4]{august_microscopy.png}
\caption{Imaging of lipid droplets post SybrGreen staining with fluorescence microscopy. A); B); C); D); E) and F) demonstrate the close association of biomass with lipid compared to the intermediate liquid phase.}
\end{figure}

\FloatBarrier
\subsubsection{\emph{Monitoring lipase production}}
For the floc community to utilise and hence remediate the GT in the sludge, lipase needs to be produced as lipase activity is directly proportional to the remediation potential of the sludge. It was expected that lipase production would be upregulated in he presence of GT and hence activity change with exposure time.


Lipase activity was detected upon colorimetric change from p-nitrophenyl palmitate cleavage in the substrate solution. The assay separated  EC, EPS bound and cell membrane bound lipases from each replicate. The enriched cultures displayed on average a higher lipase activity than the control replicates (Figure 5).
                                                                                                                                                                                                                                         

\begin{figure}
\includegraphics[scale=1]{lip_av.PNG}
\caption{Total lipase activity of all fractions and replicates.}
\end{figure}

The lipase activity in the supernatant fraction of the enriched cultures spiked between 3 and 9 days as well as 15 to 18 days in comparison to the untreated activate sludge where the lipase activity remained at a consistent level (Figure 6).

\begin{figure}
\includegraphics[scale=1.3]{lip_sn.PNG}
\caption{Average lipase activity in the supernatant fraction.}
\end{figure}

An immediate increase of lypolytic activity in the EPS  fraction (Figure 7) within the first 3 days is evident. Within 9 days the lipase activity in this fraction drops below the initial activity. At this time point the activty within the control cultures exceeds the enrichment cultures, which decreases rapidly and remains below the enrichment cultures lipase activity for all other time points.

\begin{figure}
\includegraphics[scale=1.3]{lip_eps.PNG}
\caption{Average lipase activity in the EPS fraction.}
\end{figure}

Membrane bound lipase (Figure 8) is the most abundant across both control and enrichment cultures, where the average activity in the control replicates increased in comparison to the enriched cultures between days 9 to 22.

\begin{figure}
\includegraphics[scale=1.3]{lip_biom.PNG}
\caption{Average lipase activity in the biomass fraction.}
\end{figure}
\FloatBarrier



\subsection{Microbial community changes in the presence of lipids}
The fatty acid side chains of GT yield oleic acid when cleaved off by lipase, which is the most commonly found fatty acid in olive and sunflower oils \cite{haba2000isolation}. To elucidate how the bacterial community is affected by presence of high GT and hence oleic acid concentration (Aim 3), PCR products of 16S DGGE PCR for replicate 4 and further enrichments were run on DGGE (Figure dd). Bands of interest, representing seperate organisms and their abundance, were excised and the DNA tested for integrity. Replicate 4 was chosen for monitoring the difference in community structure between activated sludge and sludge exposed to enrichment. Successfully amplified and sequenced bands are numbered in Figure 9 and the identifications were assigned with NCBI BLAST (Table 2). 

\begin{figure}
\includegraphics[scale=2.7]{DGGE_R4_450bp_thesis.jpg}
\caption{Community structure of control and enriched cultures of replicate 4, from day 0 to day 25 (D0 - D25).}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of replicate 4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1   &  \emph{Bacteroidetes} (JX473581.1) & --- & 8e$^{-45}$ \\
\hline
2  & \emph{Bacillus} sp. (GU271888.1) & \emph{Bacilli} & 5e$^{-70}$ \\
\hline
3 & Uncultured \emph{Dechloromonas} sp. (JQ012310.1) & \emph{$\beta$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Azospira oryzae} (KF260987.1) & \emph{$\beta$-proteobacteria} & 1E$^{-50}$ \\
\hline
5 & Uncultured \emph{Xanthmonadales} (KC588330.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
6 & Uncultured \emph{Dechloromonas} sp. (KF003189.1) & \emph{$\beta$-proteobacteria} & 4e$^{-103}$ \\
\hline
7 & \emph{Novosphingobium} sp. (KF544940.1) & \emph{$\alpha$-proteobacteria} & 1e$^{-173}$ \\
\hline
8 & \emph{Novosphingobium} (KF544932.1) & \emph{$\alpha$-proteobacteria} & 4e$^{-61}$ \\
\hline
9 & \emph{Sphingomonas} sp. (AY521009.2) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
10 & \emph{Sphingomonas suberifaciens} (AY521009.2) & \emph{$\alpha$-proteobacteria} & 3e$^{-119}$ \\
\hline
11 & \emph{Sphingomonas} sp. (JQ928361.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$ \\
\hline
12 & \emph{Roseomonas} sp.  (KF254767.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-65}$ \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

Furthermore to document and compare the active microbial community during extended exposure to GT (Aim 3), aggregate, its supernatant and an emulsion sample were put through two further rounds of enrichment. The PCR products of 16S DGGE PCR of these various secondary and tertiary (t2 and t3) enrichments were run on DGGE (Figure 10) where the numbered bands were excised and successfully sequenced (Table 3). 

\begin{figure}
\includegraphics[scale=1.2]{DGGE_misc_450bp_thesis.jpg}
\caption{Community structure of aggregates and emulsions from various day points (D) as well as secondary (t2) and tertiary (t3) enrichments, which are uniformly taken after 45th day, and community comparison between 1 and 2 \% GT enrichment.}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of various further enrichments with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 & \emph{Nevskia} sp. (GQ845011.1) & \emph{$\gamma$-proteobacteria} & 0.0  \\
\hline
2 & \emph{Sphingomonas} sp. (KC172307.1) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
3 & \emph{Sphingobium} sp. (KF437579.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-74}$ \\
\hline
4 & \emph{Sphingomonas} sp. (KF544924.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
5 & \emph{Rhodovarius lipocyclicus} (NR\_025629.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-114}$ \\
\hline
6 & \emph{Xanthobacter} sp. (AB847934.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$  \\
\hline
7 & \emph{Xanthobacter} sp. (AB245351.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-45}$  \\
\hline
8 & \emph{Sphingomonas} sp.(KF551133.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-100}$  \\
\hline
9 & \emph{Bradyrhizobium} sp. (JX505076.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-165}$  \\
\hline
10 & \emph{Sphingomonas} sp. (EF636068.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
11 & Candidatus \emph{Competibacter} sp. (JQ480426.1) & \emph{$\gamma$-proteobacteria} & 1e$^{-123}$  \\
\hline
12 & \emph{Sphingomonas} sp. (HE974351.1) & \emph{$\alpha$-proteobacteria} &  1e$^{-148}$ \\
\hline
13 & \emph{Stakelama pacifica} (HE662817.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-77}$  \\
\hline
14 & \emph{Oleomonas sagaranensis} (AJ784808.1) & \emph{$\alpha$-proteobacteria} & 6e$^{-101}$  \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

\subsection{Lipid colonising isolates behave differently in the presence of lipids}

Lipid droplets from the 1 \% GT trial enrichment were washed, spread on LCM and subsequently subcultured to isolate bacteria that directly colonise lipid (Aim 4). This process yielded 5 pure cultures, named LC1 - LC5.  Pure cultures were pre-cultured in LCM, with and without the presence of lipids. The culture LC1 showed a distinct morphological difference. When cultured in the presence of lipid the culture displays a pink phenotype (Figure VA) and appears white when cultured without lipids (Figure 11).

\begin{figure}
\includegraphics[scale=.65]{LC1_comp.png}
\caption{Lipid colonising isolate LC1 cultured in the presence (A) and absence (B) of lipid in LCM.}
\end{figure}

Sequencing of lipid colonising isolates resulted in identification of LC1 - LC4 using NCBI BLAST (Table 4), while LC5 was not identifiable as only 6.7 \% of the bases in the chromatogram were of high enough quality to compare to the NCBI BLAST database.
	
\begin{table}
\caption{Sequencing results for 16S rRNA fragments of lipid colonising isolates LC1 - LC4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
Isolate & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 &  \emph{Pandoraea} sp. (KF378759.1) & \emph{$\beta$-proteobacteria} & 2e$^{-79}$ \\
\hline
2 & Uncultured \emph{Achromobacter} sp. (KF448091.1) & \emph{$\beta$-proteobacteria} & 1e$^{-139}$ \\
\hline
3 & \emph{Enterobacter} sp. (KF411353.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Pseudomonas} sp. (KC822768.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
\end{tabular}
\end{table}

This led to the question whether the lipid colonising isolates produce AHLs (Aim 5) and whether potential AHL profiles differ when cultured in the presence and absence of lipids. To this end a LuxR bioassay was used which resulted in discernible activity for LC2, LC3 and LC5 (Figure 12) and whose response was compared to a standard curve (Figure 13). LC1 exhibits minimal AHL activity detectable by LuxR, while LC4 displays no activity. Both LC2 and LC3 evoked a markedly higher LuxR responses when cultured in the presence of lipid than without, while LC5 elicited the highest LuxR response but only marginally more so when cultured in the presence of lipids.


\begin{figure}
\includegraphics[scale=1.1]{LuxR.PNG}
\caption{Response to LuxR bioassay by lipid colonising isolates LC1 - LC5, comparatively cultured in the presence and absence of lipid.}
\end{figure}

\begin{figure}
\includegraphics[scale=1.1]{Std_curve.PNG}
\caption{Standard curve constructed from fluorescence of LuxR bioassay in response to 0.5, 1 and 2.5 nM OHHL.}
\end{figure}
\FloatBarrier
\newpage
\section{Discussion}
The function of our society depends on safe and efficient waste removal while sustainably recycling water and reducing environmental impact. The waste influent created by domestic households is extensive and fluctuates in composition. Hence the wastewater treatment system for influent remediation needs to be adaptable to deal with the volume and variability. The key component in the treatment facilities is the biological aspect. This study explored the impact of lipids on activated sludge, the aerobic part of the biological treatment. Ammonia is nitrified and organic matter is reduced, hence the oxygen demand of the effluent is lowered. Specifically the aim was to examine the effect of lipids on sludge morphology and change in microbial community structure as well as isolating lipid colonisers.

\subsection{Exposure to lipids changes the morphology of activated sludge}
It is evident that high lipid content in waste influent to activated sludge impacts the consistency of the sludge as well as floc settleability. This impact has been widely reported (ref) but the concentration of lipid was found to be essential for determining the type of morphological change exhibited. During the trial enrichment of 1 \% GT, lipid droplets formed which were suspended throughout the sludge and decreased settleablity. When the GT concentration was increased to 2 \% for the actual enrichment experiment, the sludge increased in viscosity instead of forming droplets and lightened in colour. This could be due to high concentration of fatty acids released, as the resulting sludge resembled an emulsion. The 2 \% lipid concentration for the enrichment was adapted from a lipase producing bacteria isolation study conducted by Haba et al \cite{haba2000isolation}, however the normal lipid concentration found in municipal wastewater is about 0.01 \% \cite{Forster_92}. Morphology that resembles droplets in lipid challenged activated sludge has been reported (ref), however neither the lipid concentration to elicit this response nor the underlying mechanisms were included. Secondary and tertiary transfer enrichment of the emulsion resulted in increased viscosity to an almost solid state and white appearance, which did not revert to anything resembling activated sludge at the last sampling point of 45 days of t3. No incidence of an emulsion like phenotype in the wastewater treatment plant setting has been found. Hence the lipid concentration that unbalances the biological treatment phase is between 0.01 and 1 \%. An accurate understanding of this value could pre-empt the deterioration of the biological treatment through efficient monitoring.

\subsection{Bacteria colonise lipids in activated sludge}
%Oleic acid is the most common fatty acid found in native as well as used olive and sunflower oils, which are commonly used in domestic settings \cite{haba2000isolation}. 
The direct colonisation of the lipid droplets was unexpected as this behaviour in AS sludge has not been recorded in the literature. However it has been recorded by Forster et al that the presence of lipids decreases flocculation and sludge settlement \cite{Forster_92}, the latter of which was observed in this experiment.

The microscopy pictures showcase the close association of biomass with the lipids. The aggregation around the droplets was akin to biofilms. Whether the dense mass of cells was surrounding and growing on the lipid or whether it was embedded within the lipid phase is unclear from this method. This should be further investigated by using scanning electron microscopy to gain an understanding of the nature of the cell attachment to lipids.
Throughout the trial run a correlation between time and the decrease in lipid droplet size indicated that the microbes colonised the droplets and degraded their substratum as a C source. This observation complements the highest lipase activity recorded for membrane bound lipases, equally active in the control and enrichment replicates. The untreated samples recorded a higher lipase activity between days 9 to 22, which was not reflected in the total lipase activity for that time. This could be due to the difficulty of fraction separation that arose with the increased viscosity of the samples. On the 12th day sampling point the enriched samples were centrifuged for 1 hr at $0\,^{\circ}\mathrm{C}$, without successful separation of supernatant and flocs. 
Activated sludge contains a baseline lipid concentration, which could explain the maintenance of lipase for rapid C source degradation. Lipase activity in the EPS fraction almost doubled within the first 3 days, followed by steady decline while lipase activity in the supernatant fraction increases. Secreted lipase expressed by the \emph{$\gamma$}-proteobacterium \textit{Pseudomonas aeruginosa} associates with alginate within the EPS of flocs, which anchors the enzyme with weak hydrogen bonding forces \cite{mayer1999role,wicker1987}. Hence the lipase  liberates fatty acids close to the cell. It follows that this mechanism is commonly employed, and the delayed appearance of lipase in the supernatant fraction could be due to liberation from the hydrogen bonding and hence the floc association. The disassociation of lipase could indicate the dispersal phase in the floc life cycle.

If lipids seed flocs, the buoyancy of the lipid core could be responsible for lack of settlement rather than the presence of lipids preventing flocculation as suggested by Forster et al \citep{Forster_92}. Determining which microbial communities dominate during lipid induced floc settlement failure could provide another aspect for monitoring and anticipation of system failure.


As 2 \% GT caused the sludge to form an emulsion which is divergent from \emph{in situ} morphology and impacted the functionality of the lipase assay, 1 \% lipid should be used for future studies to create a setting with higher accuracy.

\subsection{The activated sludge microbiome changes in the presence of lipids}
DGGE is a technique that highlights change in microbial community over time and allows for excision of nucleic material for amplification \cite{yang2012evolution}. This is ideal for monitoring which microbes change in abundance over time in an enriched setting.



The initial time points for the enrichment culture replicates did not yield DNA extracts that could be utilised for further analysis from days 3 and 6 for the controls, as well as day 3 for the enriched culture. This is most likely due to the pH of the phenol solution used in DNA extraction - initially it was pH 4. This pH is suitable for RNA extraction while phenol at pH 8 is preferable for DNA extraction. The pH of the phenol solution was adjusted from day 9 onwards.

\subsubsection{\emph{Microbial community in replicate 4}}
It is evident from the DGGE community analysis of replicate 4 that lipids have a profound impact on the microbial community structure. The disparity between dominant organisms in the control vs. enriched cultures gives insight into the dominant players in lipid degradation. 

The first half of the DGGE highlights the natively active microbiome in untreated activated sludge and the key organisms abundant in a functionally active wastewater remediating system. The following are all represented in high abundance throughout the 25 day sampling period. The bands in the control diminish over time as the intensity is directly proportional to abundance. The control was resupplied with filtered activated sludge supernatant so it follows that the community started starving after day 18 due do the lack of nutritional input and the abundance decreased.

Band 1, an unidentifiable member of the phylum bacteroidetes, is most prevalent during the zero time point. Whether it's abundance decreases by day 3 or day 9 in the control is not determinable due to the missing time points. However it disappears by day 3 in the enrichment. This phylum contains filamentous bacteria  which are associated with sludge bulking. %\cite{nielsen2009identity}.

%(inititally increased in intensity --> oligotrophs?)
Band 2 was identified as \emph{Bacillus} sp. which, in a study designed to isolate strains with high lipolytic activity from olive mill wastewater, exhibited the highest lipase activity \cite{ertuugrul2007isolation}. Specifically, \emph{Bacillus mucilaginosus} produces a bioflocculant which speeds up bioflocculation in starch wastewater treatment \cite{deng2003characteristics}. Whether this strain shares this expression pattern is unknown. However due to the it's recorded lipolytic activity, it is surprising that the abundance was not sustained in the enrichment.

Bands 3 and 6 are both designated as \emph{Dechloromonas} sp., but as two seperate species due to inter-species divergence of GC content of the region amplified. In a mature membrane fouling biofilm in municipal wastewater treatment in Japan, over 30 \% of the clones isolated were from the genus \emph{Dechloromonas} \cite{miura2007membrane}. This genus is also reported to degrade poly aromatic hydrocarbons \cite{oshiki2008pha}.

The rice pathogen \emph{Azospira oryzae} was appointed as the identity of band 4, otherwise known as \emph{Dechlorosoma suillum} \cite{tan2003dechlorosoma}. Members of this genus can use perchlorate as a terminal electron acceptor and reduces toxic selenite as well as selenate to elemental selenium which can be sequestered from effluent before it is released into the environment \cite{reinhold2000reassessment,hunter2007azospira,wilhelmus2013microbiological}.

%Band 5 xanthomonadales
\vspace{.5cm}
The abundance of several genera increased with lipid enrichment and replaced the previously described genera active in the control.

Bands 7 and 8 are members of the genus \emph{Novosphingobium} which have been shown to play an important role in wastewater remediation by degrading toxic dyes as well as estrogen - both which impact the ecosystem if released \cite{addison2007novosphingobium,hashimoto2009contribution}.


Bands 9, 10 and 11 \emph{Sphingomonas} sp. are present at about 5 - 10 \% in sludge as shown by FISH \cite{neef1999detection}, and play an important role in wastewater remediation. Members of this genera degrade testosterone and sterol hormones as well as the pollutant nonylphenol \cite{fujii2001sphingomonas,roh201017beta}.
% sphingomonas high GC varience paper


Band 12, \emph{Roseomonas} sp., was found at about 5 \% abundance in activated sludge in a Chinese wastewater treatment plant and they can degrade organophosphate pesticides \cite{jiang2008bacterial,jiang2006isolation}.

\subsubsection{\emph{Microbes in further enrichments}}
The further enrichment DGGE shows the communities dominant in several settings. It was noted that aggregates formed within the emulsions, they appeared solid suspended within the viscous cultures but disintegrated when probed and may have been entirely colonised lipids. Hence the microbial community within these was of interest and one such aggregate was targeted for further secondary and tertiary enrichment. The aggregate expanded and developed tendrils upon further enrichment and cells dispersed into the supernatant, which was tertiarily enriched, seperate to the aggregate. The tertiary enrichment time point (t3) is taken after 45 days to elucidate the microbiome of long term enrichment.


\emph{Nevskia} sp., band 1, is present in the aggregate at day 12 and then cease to exist at detectable levels at later time points for the aggregate. It is present in t3 aggregate supernatant, so it could have dissociated from the conglomerate to enter the supernatant phase. \emph{Nevskia} sp. are slow growing, where colony formation takes about a week, and they produce lipase \cite{kim2011nevskia}. A study by Chooklin et al endeavoured to isolate the most efficient surfactant producer from palm oil mill effluent for lipid remediation and found \emph{Nevskia} sp. \cite{chooklinutilization}.


Similarly, \emph{Sphingomonas} sp. from band 2 are present throughout day 12 to 18 but are absent in the t3 for the aggregate. They do appear in the aggregate supernatant as band 10, which band was also assigned the identity of \emph{Sphingomonas} sp. by NCBI, albeit with a different accession number. The sequences used to identify band 2 and band 10 were aligned with BLAST, with lengths of 445 bp and 410 bp respectively. The sequence identity was 100 \% with an E-value of 0.0 hence the organisms that represent the two bands are regarded as synonymous regardless of differing accession numbers.
\emph{Sphingomonas} sp. also represent bands 4 and 8. The former is exclusive to primary aggregate sampling and disappears by the tertiary enrichment. The latter arises in t3 aggregate and emulsion.
Band 12, \emph{Sphingomonas} sp., appears in t3 emulsion. Whether the band in the 1 and 2 \% GT comparison is the same organisms is not possible to determine without sequencing as these bands are fractionally above the t3 bands.
From the DGGE gel, bands 2 and 13 appear to be the same organism. However during identity assignment, the latter returned as \emph{Stakelama pacifica} from NCBI with and E-value of 3e$^{-77}$ from a 163 bp sequence, while band 2 was concluded to be \emph{Sphingomonas} sp. with a more reliable E-value of 0.0 based on a 445 bp sequence. The two sequences were aligned with NCBI BLAST to investigate sequence similarity. The sequence alignment of band 13, which covers the 269 to 431 bp region of the amplified 16S rRNA fragment, aligns from the 269$^{th}$ bp of band 2 with a sequence identity was 100 \% with an E-value of 7e$^{-87}$. It is concluded that band 13 represents the \emph{Sphingomonas} from band 2 rather than \emph{Stakelama pacifica}.
The designation \emph{Sphingomonas} sp. arose for 5 different bands which shows high variability in GC content in this genus. Similar variance for DGGE results has been recoded \cite{qiao2012effect}.
% section about high GC content variability

\emph{Sphingobium} sp., the designation for band 3, is prevalent day 15 and 18 in the aggregate as well as day 7 in the secondary emulsion enrichment. A members of this genus, isolated from river sediment, can degrade nonylphenol \cite{ushiba2003sphingobium}.


Band 5, \emph{Rhodovarius lipocyclicus}, is abundant from day 18 in the aggregate as well as in the tertiary enrichment and day 7 for t2 emulsion. However the highest abundance is recorded in the t3 aggregate supernatant, but it was not recorded in the tertiary emulsion enrichment. Limited information is available about \emph{Rhodovarius lipocyclicus}, beyond the basic information required for classification \cite{kampfer2004rhodovarius}.


\emph{Xanthobacter} sp., bands 6 and 7, are prevalent on day 18 in the aggregate. Band 7 also appears in the tertiary enrichment of the aggregate supernatant. As DGGE separates sequences by GC content, these may be two closely related species of the genus \emph{Xanthobacter} with slightly divergent GC content of the region amplified. This genus is reported to remediate aliphatic halogenated compounds which are commonly found in municipal wastewater \cite{janssen1985degradation} and have been shown to co-ordinate with \emph{Novosphingobium} sp. to degrade polyvinyl alcohol \cite{rong2009symbiotic}. \emph{Novosphingobium} sp., bands 7 and 8 in the GT enriched  4, also degrade polycyclic aromatic hydrocarbons, which can by synthesised from lipid precursors \cite{addison2007novosphingobium}.


Members of the \emph{Bradyrhizobium} genera were designated as band 9 in the t3 aggregate supernatant and found in the emulsions t2 and t3. This genus is slow growing \cite{rebah2002wastewater} and usually associated with plant nodules. In plant symbiosis, they contribute by fixing nitrogen - an attribute required for successful A/A/O process. When exposed to activated sludge, these microbes have been shown to become highly antibiotic and heavy metal resistant \cite{ahmad17samiullah}.


Candidatus \emph{Competibacter} came up as band 11, only in the secondary emulsion enrichment. Members of this genus are glycogen accumulating organisms, which produce polycyclic aromatic hydrocarbons, and represent 22 - 26 \% of enriched sludge microbiota \cite{bengtsson2008production,lemaire2008microbial}. 


Band 14 shows low abundance \emph{Oleomonas} sp., in t3 emulsion. However this genus was identified with FISH and DGGE, to constitute about 16 \% of the biomass in an upstream anaerobic bioreactor fed with brewery wastewater \cite{fernandez2008analysis}. Specifically \emph{Oleomonas sagaranensis} is involved in breaking down urea \cite{kanamori2005allophanate,kanamori2004enzymatic} which is essential for nitrification of ammonia in activated sludge.


While none of the bands from the 1 and 2 \% enrichment comparison were amplifiable for sequencing, the organisms active in these cultures were very similar. While the bands are present for both treatments, three of the bands show a difference in intensity and hence abundance.
mike - should I include letters for these bands on the DGGE picture in results so I can refer to them?

\subsubsection{\emph{Common players}}
\emph{Sphingomonas} sp. are common in both replicate 4 enrichment as well as the various further enrichments.
Approximately 92 \% of identified key organisms from DGGEs belonged to the proteobacteria. Of these 75 \% were \emph{$\alpha$}-proteobacteria  and each \emph{$\gamma$}- and \emph{$\beta$}- proteobacteria were 12.5 \%.
Hesham et al compared the microbial communities of two differently operated municipal wastewater treatment plants over six months via DGGE. The 11 OTUs common to both plants, and the most abundant, were 18 \% to alpha-proteobacteria and 18 \% to beta-proteobacteria \cite{Hesham_11}.

If the bands found in t3 from the aggregate and the aggregate supernatant are considered together, the banding pattern closely resembles that of the t3 emulsion. Despite the removal of the aggregate two transfers prior, the microbial community in the t3 aggregate and supernatant is highly similar with the difference of the absence of \emph{Rhodovarius lipocyclicus} in t3 emulsion.


While the majority of the microbes identified in the further enrichments, but not found in the enrichment or control cultures for the 25 day duration of the enrichment experiment, these microorganisms are present in low abundance and become prevalent when their niche ability to degrade certain recalcitrant compounds becomes relevant. Also the detection of organisms was limited as several bands were not sequenceable.

Wagner et al suggest fluorescence in situ hybridisation (FISH) with 16S rRNA  group-specific probes is necessary to accompany 16S rRNA sequencing sampling to ensure the accuracy of the OTU representation. However as this is a time consuming process, the majority of the community analysis is on 16S rRNA only \cite{Wagner_02} . A skewered representation of the OTUs in the community can arise from 16S primer bias which should be checked and accounted for.

\subsection{Lipid colonising isolates behave differently in the presence of lipids}

LC1, \emph{Pandoraea} sp., produces a pink phenotype when cultured in the presence of lipids which could be due to pigment production. This suggests that this expression pattern is activated in the presence of lipids. \emph{Pandoraea} sp. did not evoke a response from the LuxR bioassay and a search of NCBI Protein BLAST returns a LuxR homolog but not a LuxI homolog. Nonetheless a \emph{Pandoraea} sp. soil isolate has been shown to produce the AHL \emph{N}-octanoylhomoserine lactone, which has 8 Cs attached to the lactone ring \cite{han2013pandoraea}. As the activation was negligible in this study, this AHL may be out of the detection range of the LuxR bioassay and should be tested with the \emph{Chromobacterium violacein} CV026 bioassay.

\emph{Achromobacter} sp., LC2, produced a minor LuxR response when cultured without oil and cultured in the presence of oil, a response equivalent between 0.5 and 1 nm OHHL when compared to the standard curve (Figure x). The \emph{Achromobacter} sp. genome was searched a \emph{luxI} homolog by using a \emph{Burkholderia} sp. homolog as  this is a \emph{$\beta$}- proteobacterium known to contain an AHL synthase. The databases UniProt and SwissProt were used through EXPASY in order to search annotated genomes only and exclude draft genomes. This returned  22 \% similarity to an \emph{Agrobacterium} sp. LuxI homolog.

LC3, \emph{Enterobacter} sp., evoked a LuxR response equivalent to approximately 1 nM OHHL when cultured without oil and a response equivalent to 2.5 nM when cultured with lipid. The LuxI homolog for members of this genus can be found under the accession number  WP\_008503217.

LC4, a \emph{Pseudomonas} sp., evoked a response akin to 0.5 nm OHHL from the LuxR bioassay when cultured in the presence of lipid. Members of this genus tend to have two AHL dependent gene expression systems and an autoinducer synthesis protein can be found under the accession number   WP\_018604918.

Isolate LC5 was of interest due to it's high bioassay response, exceeding the 2.5 nM OHHL standard. LuxR response is similar when cultured with and without lipid. The organism could not be identified as the sequencing data was inconclusive for both attempts.
\newpage
\section{Conclusions and Future directions}
Bacterial communication and coordination is essential for floc formation and wastewater remediation. The colonisation in a biofilm-like manner around lipid droplets extends our understanding about how the lipid fraction of wastewater influent is utilised for remediation and needs to be investigated further as lipid droplets could seed floc formation. SEM microscopy to showcase biofilm like attachment of cells to droplets and whether cells are burrowed in lipid or attached to surface should be conducted. Determining the exact concentration of lipids which elicits droplet formation and hence decrease floc settleability is essential for potentially monitoring wastewater influent and pre-empting failure of the remediation process. An experimental system to analyse the microbiome change with extended exposure to lipids was established. Highly abundant organisms which were affected by lipid enrichment were analysed with 16S rRNA DGGE, followed by sequencing and characterisation. Approximately 92 \% of the identified microbes belonged to the phylum proteobacteria, which is well known for using AHL mediated gene expression, including for lipase expression. Lipid droplet colonising isolates were screened with a LuxR biosensor for AHL production where 3 out of 5 isolates elicited a response from the biosensor. These isolates showed a higher AHL production when cultivated in the presence of lipid than without.
\newpage 
%fix inclusion of refereces in table of contents
\bibliographystyle{plain}
\bibliography{thesis_ref.bib}

\newpage 
\section{Supplementary Material}
\FloatBarrier
\begin{table}
\caption{LB10 low yeast}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (L) \\
\hline
 Agar, optional & 12 g  \\
Bacto-tryptone    & 10 g  \\
   NaCl   & 10 g  \\
    Yeast extract    & 1.25 g \\
  \hline
\end{tabular}
\end{table}
\FloatBarrier
\begin{table}
\caption{Lipase assay substrate solution, component A at 1x}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (w/v) \\
\hline
 p-nitrophenyl palmitate   & 0.3 \% \\
 Isopropanol &  \\
 \hline
\end{tabular}
\end{table}

\begin{table}
\caption{Lipase assay substrate solution, component B at 9x}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (w/v) \\
\hline
 Gum arabicum & 0.1 \% \\
  Sodium deoxycholate & 0.2 \% \\
  Phosphate buffer, 50 mM pH 8   &  \\
  \hline
\end{tabular}
\end{table}

mix components A and B just prior to use

\begin{table}
\caption{Lipid coloniser medium}
\begin{tabular}{  p{6.9cm} | p{6.9cm} }
\hline
Component & Concentration (L) \\
\hline
Agar, optional  & 15 g  \\
  Di-potassium phosphate  & 0.3 g \\
   Glyceryl Trioleate   & 10 ml \\
      Magnesium sulphate  & 0.024 g  \\
       Proteose peptone   & 0.5 g \\
          Starch  & 0.5 g \\
           Yeast extract   & 0.5 g \\
  \hline
\end{tabular}
\end{table}


\begin{table}
\caption{Phosphate buffered saline, pH adjusted to 7.4}
\begin{tabular}{  p{6.9cm} | p{6.9cm}  }
\hline
Component & Concentration (L) \\
\hline
Di-sodium phosphate  & 1.44 g \\
  Monopotassium phosphate  & 0.24 g  \\
    Potassium chloride  &  0.2 g  \\
      Sodium chloride  & 8 g  \\
  \hline
\end{tabular}
\end{table}

\begin{table}
\caption{XS buffer}
\begin{tabular}{  p{6.9cm} | p{6.9cm}  }
\hline
Component & Concentration (L) \\
\hline
 4 M Ammonium acetate & 200 ml  \\
  0.45 M EDTA  & 40 ml \\
    20 \% SDS  & 50 ml \\
    1 M Tris-Hydrochloride pH 4    & 100 ml  \\
        Potassium ethyl Xanthogenate  & 10 g  \\
  \hline
\end{tabular}
\end{table}
\FloatBarrier


\end{document}