\documentclass[11pt]{article}
%\documentclass[draft]{article} for better debugging!
\usepackage{placeins}
\usepackage{cite}
\usepackage{graphicx}
\usepackage{rotating}
%download float barrier sty - uni mail
\linespread{1.3}

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}

\begin{document}

\subsection{Bacteria colonise lipids in activated sludge}
It was essential to establish an experimental system and adapt protocols to observe the impact of lipids on activated sludge before a controlled enrichment experiment could be conducted. 

\subsubsection{\emph{change in morphology}}
Before conducting the enrichment experiment, the behaviour of activated sludge in  the presence of GT needed to be established (Aim 1). The trial was conducted with 1 \% GT  added to sludge replicates which was sampled every 7 days, as the expectation was that the enrichment would run for an extended period of time. GT formed a transparent and hydrophobic layer on the surface of the enrichment cultures, which was replaced with white droplets suspended below the culture surface within 7 days (Figure XB). While varying in size, the droplets consistently decreased in diameter with increased incubation period until they were indistinguishable from the sludge after 21 days. Droplet formation decreased floc settlement for the duration of their presence. 


Due to the lipid being assimilated into the sludge within 3 sampling points, the sampling frequency was increased from 7 to every 3 days. To compensate, the actual enrichment experiment was conducted with 2 \% GT. The increase of GT concentration resulted in no droplet formation. Instead the cultures became increasingly viscous and lighter in colour (Figure XA and XC). No settlement of sludge flocs was observed, however denser, singular aggregates appeared in the viscous cultures. One of these was targeted for secondary enrichment by moving the aggregate into sludge supernatant, where the aggregate expanded and developed tendril like structures (Figure XD).

\begin{figure}
\includegraphics[scale=.77]{cultures.jpg}
\caption{Impact of GT addition to activated sludge cultures; A) the comparison between 2 \% GT enrichment and no treatment; B) lipid droplet formation in the presence of 1 \% GT in trial run; C) the lack of settleability of sludge after incubation with 2 \% GT; D) secondary enrichment by transferral of aggregate into sludge supernatant and 2 \% GT.}
\end{figure}
\FloatBarrier

\subsubsection{\emph{Microscopy}}
To establish the nature of the droplets and aggregates in 1 \% and 2 \% GT enrichments respectively (Aim 2), samples were isolated and stained with SybrGreen before imaging with epifluorescence microscopy (Figure xx). The fluorophore binds to DNA and emits green light upon excitation while lipid appears yellow and water appears black. 
Figure X consistently shows that the biomass is closely associated with the lipid, rather than the intermediate water space and firmly attached due to the wash steps prior to staining.
While XA and XB show clusters of biomass around lipid, XC, XD, XE and XF are a close up of those clusters which showcase smaller segments of lipids surrounded and colonised by lipids. These pictures are unclear because the cells occupy the space around the lipids at different levels. This spatial distribution and cell density is akin to biofilm formation. 
\begin{figure}
\includegraphics[scale=.4]{august_microscopy.png}
\caption{Imaging of lipid droplets post SybrGreen staining with fluorescence microscopy. A); B); C); D); E) and F) demonstrate the close association of biomass with lipid compared to the intermediate liquid phase.}
\end{figure}

\FloatBarrier
\subsubsection{\emph{Monitoring lipase production}}
For the floc community to utilise and hence remediate the GT in the sludge, lipase needs to be produced as lipase activity is directly proportional to the remediation potential of the sludge. It was expected that lipase production would be upregulated in he presence of GT and hence activity change with exposure time.


Lipase activity was detected upon colorimetric change from p-nitrophenyl palmitate cleavage in the substrate solution. The assay separated  EC, EPS bound and cell membrane bound lipases from each replicate. The enriched cultures displayed on average a higher lipase activity than the control replicates (Figure hksjhfvn).
                                                                                                                                                                                                                                         

\begin{figure}
\includegraphics[scale=1]{lip_av.PNG}
\caption{Total lipase activity of all fractions and replicates.}
\end{figure}

The lipase activity in the supernatant fraction of the enriched cultures spiked between 3 and 9 days as well as 15 to 18 days in comparison to the untreated activate sludge where the lipase activity remained at a consistent level (Figure gg).

\begin{figure}
\includegraphics[scale=1.1]{lip_sn.PNG}
\caption{Average lipase activity in the supernatant fraction.}
\end{figure}

An immediate increase of lypolytic activity in the EPS  fraction (Figure gh) within the first 3 days is evident. Within 9 days the lipase activity in this fraction drops below the initial activity. At this time point the activty within the control cultures exceeds the enrichment cultures, which decreases rapidly and remains below the enrichment cultures lipase activity for all other time points.

\begin{figure}
\includegraphics[scale=1.1]{lip_eps.PNG}
\caption{Average lipase activity in the EPS fraction.}
\end{figure}

Membrane bound lipase (Figure jj) is the most abundant across both control and enrichment cultures, where the average activity in the control replicates increased in comparison to the enriched cultures between days 9 to 22.

\begin{figure}
\includegraphics[scale=1.1]{lip_biom.PNG}
\caption{Average lipase activity in the biomass fraction.}
\end{figure}
\FloatBarrier



\subsection{Micobial community changes in the presence of lipids}
The fatty acid side chains of GT yield oleic acid when cleaved off by lipase, which is the most commonly found fatty acid in olive and sunflower oils \cite{haba2000isolation}. To elucidate how the bacterial community is affected by presence of high GT and hence oleic acid concentration (Aim 3), PCR products of 16S DGGE PCR for replicate 4 and further enrichments were run on DGGE (Figure dd). Bands of interest, representing seperate organisms and their abundance, were excised and the DNA tested for integrity. Replicate 4 was chosen for monitoring the difference in community structure between activated sludge and sludge exposed to enrichment. Successfully amplified and sequenced bands are numbered in Figure gg and the identifications were assigned with NCBI BLAST (Table cc). 

\begin{figure}
\includegraphics[scale=2.7]{DGGE_R4_450bp_thesis.jpg}
\caption{Community structure of control and enriched cultures of replicate 4, from day 0 to day 25 (D0 - D25).}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of replicate 4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1   &  \emph{Bacteroidetes} (JX473581.1) & --- & 8e$^{-45}$ \\
\hline
2  & \emph{Bacillus} sp. (GU271888.1) & \emph{Bacilli} & 5e$^{-70}$ \\
\hline
3 & Uncultured \emph{Dechloromonas} sp. (JQ012310.1) & \emph{$\beta$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Azospira oryzae} (KF260987.1) & \emph{$\beta$-proteobacteria} & 1E$^{-50}$ \\
\hline
5 & Uncultured \emph{Xanthmonadales} (KC588330.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
6 & Uncultured \emph{Dechloromonas} sp. (KF003189.1) & \emph{$\beta$-proteobacteria} & 4e$^{-103}$ \\
\hline
7 & \emph{Novosphingobium} sp. (KF544940.1) & \emph{$\alpha$-proteobacteria} & 1e$^{-173}$ \\
\hline
8 & \emph{Novosphingobium} (KF544932.1) & \emph{$\alpha$-proteobacteria} & 4e$^{-61}$ \\
\hline
9 & \emph{Sphingomonas} sp. (AY521009.2) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
10 & \emph{Sphingomonas suberifaciens} (AY521009.2) & \emph{$\alpha$-proteobacteria} & 3e$^{-119}$ \\
\hline
11 & \emph{Sphingomonas} sp. (JQ928361.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$ \\
\hline
12 & \emph{Roseomonas} sp.  (KF254767.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-65}$ \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

Furthermore to document and compare the active microbial community during extended exposure to GT (Aim 3), aggregate, its supernatant and an emulsion sample were put through two further rounds of enrichment. The PCR products of 16S DGGE PCR of these various secondary and tertiary (t2 and t3) enrichments were run on DGGE (Figure dd) where the numbered bands were excised and successfully sequenced (Table gg). 

\begin{figure}
\includegraphics[scale=1.2]{DGGE_misc_450bp_thesis.jpg}
\caption{Community structure of aggregates and emulsions from various day points (D) as well as secondary (t2) and tertiary (t3) enrichments, which are uniformly taken after 45th day, and community comparison between 1 and 2 \% GT enrichment.}
\end{figure}

\begin{table}
\caption{Sequencing results for 16S rRNA DGGE fragments of various further enrichments with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
DGGE band & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 & \emph{Nevskia} sp. (GQ845011.1) & \emph{$\gamma$-proteobacteria} & 0.0  \\
\hline
2 & \emph{Sphingomonas} sp. (KC172307.1) & \emph{$\alpha$-proteobacteria} & 0.0 \\
\hline
3 & \emph{Sphingobium} sp. (KF437579.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-74}$ \\
\hline
4 & \emph{Sphingomonas} sp. (KF544924.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
5 & \emph{Rhodovarius lipocyclicus} (NR\_025629.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-114}$ \\
\hline
6 & \emph{Xanthobacter} sp. (AB847934.1) & \emph{$\alpha$-proteobacteria} & 5e$^{-86}$  \\
\hline
7 & \emph{Xanthobacter} sp. (AB245351.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-45}$  \\
\hline
8 & \emph{Sphingomonas} sp.(KF551133.1) & \emph{$\alpha$-proteobacteria} & 2e$^{-100}$  \\
\hline
9 & \emph{Bradyrhizobium} sp. (JX505076.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-165}$  \\
\hline
10 & \emph{Sphingomonas} sp. (EF636068.1) & \emph{$\alpha$-proteobacteria} & 0.0  \\
\hline
11 & Candidatus \emph{Competibacter} sp. (JQ480426.1) & \emph{$\gamma$-proteobacteria} & 1e$^{-123}$  \\
\hline
12 & \emph{Sphingomonas} sp. (HE974351.1) & \emph{$\alpha$-proteobacteria} &  1e$^{-148}$ \\
\hline
13 & \emph{Stakelama pacifica} (HE662817.1) & \emph{$\alpha$-proteobacteria} & 3e$^{-77}$  \\
\hline
14 & \emph{Oleomonas sagaranensis} (AJ784808.1) & \emph{$\alpha$-proteobacteria} & 6e$^{-101}$  \\
\hline
\end{tabular}

\end{table}
\FloatBarrier

\subsection{Lipid colonising isolates behave differently in the presence of lipids}

Lipid droplets from the 1 \% GT trial enrichment were washed, spread on LCM and subsequently subcultured to isolate bacteria that directly colonise lipid (Aim 4). This process yielded 5 pure cultures, named LC1 - LC5.  Pure cultures were pre-cultured in LCM, with and without the presence of lipids. The culture LC1 showed a distinct morphological difference. When cultured in the presence of lipid the culture displays a pink phenotype (Figure VA) and appears white when cultured without lipids (Figure VB).

\begin{figure}
\includegraphics[scale=.65]{LC1_comp.png}
\caption{Lipid colonising isolate LC1 cultured in the presence (A) and absence (B) of lipid in LCM.}
\end{figure}

Sequencing of lipid colonising isolates resulted in identification of LC1 - LC4 using NCBI BLAST (Table cc), while LC5 was not identifiable as only 6.7 \% of the bases in the chromatogram were of high enough quality to compare to the NCBI BLAST database.
	
\begin{table}
\caption{Sequencing results for 16S rRNA fragments of lipid colonising isolates LC1 - LC4 with corresponding taxonomic class and E-value obtained for classification certainty.}
\begin{tabular}{ | l | p{7.8cm} | p{3cm} | l | }
\hline
Isolate & Bacteria with highest identity \& Acc. no. & Class & E-value \\
\hline
1 &  \emph{Pandoraea} sp. (KF378759.1) & \emph{$\beta$-proteobacteria} & 2e$^{-79}$ \\
\hline
2 & Uncultured \emph{Achromobacter} sp. (KF448091.1) & \emph{$\beta$-proteobacteria} & 1e$^{-139}$ \\
\hline
3 & \emph{Enterobacter} sp. (KF411353.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
4 & \emph{Pseudomonas} sp. (KC822768.1) & \emph{$\gamma$-proteobacteria} & 0.0 \\
\hline
\end{tabular}
\end{table}

This led to the question whether the lipid colonising isolates produce AHLs (Aim 5) and whether potential AHL profiles differ when cultured in the presence and absence of lipids. To this end a LuxR bioassay was used which resulted in discernible activity for LC2, LC3 and LC5 (Figure jhfhf). LC1 exhibits minimal AHL activity detectable by LuxR, while LC4 displays no activity. Both LC2 and LC3 evoked a markedly higher LuxR responses when cultured in the presence of lipid than without, while LC5 elicited the highest LuxR response but only marginally more so when cultured in the presence of lipids. 


\begin{figure}
\includegraphics[scale=1.1]{LCM_LuxR.png}
\caption{Response to LuxR bioassay by lipid colonising isolates LC1 - LC5, comparatively cultured in the presence and absence of lipid.}
\end{figure}
\FloatBarrier

\end{document}