\documentclass[11pt]{article}
%\documentclass[draft]{article} for better debugging!
\usepackage{graphicx}
\usepackage{cite}

\linespread{1.3}

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}

\begin{document}


\subsection{Enrichment cultures}
Enrichment cultures consisted of 50 ml AS collected from St. Mary's WWT plant in each of eight identically proportioned 250 ml Boecko flasks. The experiment was conducted in quadruplicates of 2 \% GT enrichment (Glyceryl trioleate, Sigma-Aldrich T7140-10G) and no treatment. The cultures were incubated at $37\,^{\circ}\mathrm{C}$ on a shaker set at 140 rpm. Samples were taken over a 25 day period and consisted of 0.5 ml for Deoxyribonucleic Acids (DNA) extraction and 0.5 ml for lipase assays. The 1 ml removed was replaced with sterile, syringe filtered (0.22 $\mu$m Millipore) activated sludge supernatant. The trial enrichment cultures were enriched with 1 \% GT rather than 2 \%.

\subsubsection{\emph{Microscopy}}
Lipid droplets were removed with tweezers and washed in PBS in two subsequent crystallizing dishes. After transferral to a glass slide, the droplet was stained with 5 $\mu$l SybrGreen, a DNA binding dye, and covered with a cover slide. The droplets were imaged with an Olympus Fluorescence Microscope which elicits a visual response from DNA bound dye.

\subsubsection{\emph{DNA extraction}}
DNA was extracted by the amended xanthogenate-SDS method \cite{tillett2000xanthogenate}. The samples for DNA extraction were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube, supernatant removed and the pellets frozen . In the case of DNA extraction from GT droplets, 0.5 ml from GT enrichment cultures were centrifuged at 16.1 rcf for 5 min in a 1.5 ml eppendorf tube and the pellet was decanted into sterile MilliQ water in a crystallizing dish. Semi-solid lipid droplets were removed with tweezers and washed in a second crystallizing dish, transferred to a 1.5 ml eppendorf tube and frozen. Thawed pellets were re-suspended in 0.2 ml phosphate buffered saline (PBS, as per \textbf{Appendix xxx}).


For each sample, 0.9 ml phenol solution and 0.9 ml XS-buffer (as per \textbf{Appendix 7.3}) was  heated to $70\,^{\circ}\mathrm{C}$ in a 2 ml microcentrifuge tube in a water bath for 5 min. The 0.2 ml cell suspension was added to the 2 ml tubes, mixed by inversion and left at $70\,^{\circ}\mathrm{C}$ for 15 min. Then the samples were vortexed for 10 seconds and frozen for 2 min, followed by centrifugation at 16.1 rcf for 5 min. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 0.9 ml phenol:chloroform:isoamyl alcohol mix and centrifuged at 16.1 rcf for 5 min. This step was repeated. The aqueous layer was transferred to another 2 ml microcentrifuge tube containing 1 ml ice cold isopropanol, 50 $\mu$l 3 M sodium acetate pH 7.5 and 1.3 $\mu$l glyco blue, mixed by inversion and frozen over night. Subsequent centrifugation at 16.1 rcf for 60 min at $0\,^{\circ}\mathrm{C}$ was followed by removal of isopropanol and addition of 1 ml 80 \% ethanol. Centrifugation at 16.1 rcf for 20 min at $0\,^{\circ}\mathrm{C}$ followed, then ethanol was removed and samples were left to air dry for 30 min before re-suspension in 0.1 ml molecular water.
\\ 
Extracted DNA concentration was analysed via Nanodrop (NanoDrop ND-1000 Spectrophometer) as well as Qubit (Invitrogen, Qubit 2.0 Flourometer), as per standard protocol for rad range DNA detection.

\subsubsection{\emph{Lipase assay}}
The lipase assay was adapted from Christensen et al \cite{christensen_03}. From each replicate 0.5 ml was taken and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and syringe sterilised (0.22 $\mu$m Millipore filter) to test for suspended EC lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water with the addition of 0.1 ml Zirkonium beads. The samples were bead beaten for three 45 sec cycles and centrifuged at 16.1 rcf for 5 min. The supernatant was removed and and syringe sterilised (0.22 $\mu$m Millipore filter) to test for EPS associated lipase.
The pellet was re-suspended via vortexing in 0.5 ml autoclaved MilliQ water to test for membrane bound lipase.


\begin{figure}
\begin{center}
\includegraphics[width=.65\textwidth]{lipase_assay.png}\\
\includegraphics[width=.65\textwidth]{lipase_assay2.png}
\caption{Fractionation of each sample into three fractions, each subjected to lipase activity assay.}
\end{center}
\end{figure}

For each sample 0.1 ml were incubated with 0.9 ml of  p-nitrophenyl palmitate (p-npp) containing substrate solution (as per \textbf{Appendix 7.1}) for one hour. Lipase cleaves the p-nitrophenyl group off the plamitate which can be quantified at an absorbance of 410. The p-nitrophenyl concentration is directly correlative with lipase activity. The reaction is terminated by alkaline pH inactivation of lipase with 1 ml 1 M Sodium Carbonate. Samples were centrifuged at 16.1 rcf for 5 min. From each terminated reaction, 0.3 ml supernatant was transferred to 96 well micro titre plate and the absorbance read at a wavelength of 410 with a Microtitre reader (Molecular Devices, Spectra Max 340).

\subsection{community analysis of enriched cultures}
DNA was extracted from enrichment culture isolates, verified with DGGE polymerase chain reaction (PCR). The DGGE PCR assay was set up with the primers 357FGC and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 15.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 2 $\mu$l template. The PCR protocol was the following: 2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.
Bands of interest were selected from the DGGE and excised and re-run on DGGE PCR to test their integrity, and if satisfactory underwent sequencing PCR. 

\subsubsection{\emph{Further enrichment}}
To investigate the change of microbial composition with long term enrichment, secondary (t2) and tertiary (t3) enrichments were conducted. Further enrichments with aggregate supernatant or emulsion were transferred at 5 \% to 25 ml activated sludge syringe filtered (0.22 $\mu$m Millipore filter) supernatant with 2 \% GT. The aggregate was transferred to 25 ml activated sludge syringe filtered (0.22 $\mu$m Millipore filter) supernatant with 2 \% GT. 


Also to investigate the difference in behaviour of activated sludge in the presence of 1 vs. 2 \% GT, a comparative enrichment was conducted with DNA extracted after 7 days and processed for DGGE and sequencing.
 
\subsubsection{\emph{Denaturing Gradient Gel Electrophoresis}}
DGGE was conducted using the BioRad DCode system and protocol for 50 \% to 70 \% denaturing gradient on a 6.7 \% arcylamide gel. Electrophoresis was conducted at 75V and $60\,^{\circ}\mathrm{C}$ for 16.5 hrs in 1x TAE. The gel was stained with SybrGold () for 20 min and imaged on a BioRad Gel Doc XR+ Imaging system and BioRad Image Lab software. Bands of interest were excised and suspended in 40 $\mu$l molecular water over night, which used as template for nested PCR set up with 357F and 907R for 40 $\mu$l as follows: 20 $\mu$l EconoTaq Master mix, 12.48 $\mu$l molecular water, 1 $\mu$l of each primer, 0.52 $\mu$l bovine serum albumin and 5 $\mu$l template. The PCR protocol was the following:  min at $30\,^{\circ}\mathrm{C}$, then  cycles of denaturing phase:    2 min at $95\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 30 sec at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $54\,^{\circ}\mathrm{C}$, extension phase: 1 min 30 sec at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$. Products for sequencing were cleaned up with Zymo Research DNA Clean \& Concentrator-5 kit, following the PCR product protocol.

\subsubsection{\emph{Sequencing}}
The sequencing PCR assay, protocol from the Ramaciotti centre, was set up for the 357F primer at 20 $\mu$l as follows: 1 $\mu$l BigDye terminator V3.1, 20 - 50 ng PCR product, 0.32 $\mu$l primer, 3.5 $\mu$l 5x sequencing buffer and made up to 20 $\mu$l with molecular water. The PCR protocol was the following: 26 cycles of denaturing phase: 10 sec at $96\,^{\circ}\mathrm{C}$, annealing phase:  at 5 sec $50\,^{\circ}\mathrm{C}$, extension phase: 4 min at $60\,^{\circ}\mathrm{C}$, followed by and storage at $4\,^{\circ}\mathrm{C}$.

The samples were cleaned up by addition of 5 $\mu$l of 125 mM EDTA and 60 $\mu$l 100 \% ethanol followed by vortexing and 15 min precipitation period. The samples were then spun at 14000 rcf for 20 min and supernatant was removed. After addition of 160 $\mu$l fresh 70 \% ethanol, samples were spun at 14000 rcf for 10 min and supernatant discarded. This step was repeated with addition of 80 $\mu$l 70 \% ethanol. The samples were dried in the dark and submitted to the Ramaciotti centre at UNSW for Sanger Sequencing.

\subsection{Lipid coloniser isolation}
Sludge isolates  were isolated from lipid droplets, washed twice in PBS, by three subsequent transfers on Lipid coloniser medium (LCM as per Appendix x) plates. Isolates were prepared for 16S rRNA PCR for the primers 27F and 1492R at 20 $\mu$l as follows: 10 $\mu$l EconoTaq Master mix, 7.4 $\mu$l molecular water, 0.8 $\mu$l of each primer and 1 $\mu$l template. The PCR protocol was the following: 2 min at $94\,^{\circ}\mathrm{C}$, then 30 cycles of denaturing phase: 1 min at $94\,^{\circ}\mathrm{C}$, annealing phase: 30 sec at  $59\,^{\circ}\mathrm{C}$, extension phase: 1.40 min at $72\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$ submitted for Sanger sequencing by the Ramaciotti centre as previously outlined.
% incl diff primers and hene length, PCR protocol.

\subsubsection{\emph{Lipid coloniser behaviour in the presence and absence of olive oil}}
Isolates were pre-cultured in 25 ml LCM broth, either with or without olive oil (Mike - should i just exclude here that I ran out of GT?). 

\subsubsection{\emph{Screening lipid colonising isolates for AHL production}}
Pre-cultures were centrifuged at 16.1 rcf for 10 min and 20 ml supernatant was transferred to a 50 ml falcon tube. After addition of 20 ml Ethyl acetate with 0.01 \% glacial acetic acid, the mixture was vortexed and left to separate. The top  layer was transferred to an 100 ml beaker. This was repeated twice and left over night for evaporation in a chemical fume hood. The AHLs were resuspended in 100  $\mu$l methanol and filter sterilised with PVDF Filter Vials (CP-ANALYTICA GmbH). 
 The reporter strain \emph{Escherichia coli} carrying the plasmid pJBA357 was pre-cultured in Lysogeny Broth (LB10, as per Appendix x) \cite{bertani1951studies}. The plasmid contains \emph{gfp}, which  is preceded by the \emph{luxR} promoter, activated by xxx and the response is directly proportionate to the AHLs present.
The assay was conducted in triplicate with \emph{Aeromonas hydrophila} GC1 as the positive control, LB10 as a blank and 2.5 nm, 1 nm and 0.5 nm OHHL as a comparative reference. Per microtitre plate well 20 $\mu$l of extract was added and the methanol left to evaporate, followed by addition and mixing of 100 $\mu$l \emph{E. coli} in it's exponential growth phase. After 4 hrs of incubation OD600 was read to verify \emph{E. coli} growth as well as fluorescence by excitation at 485 nm and then emission at 535 nm.

%The PCR assay was set up for $\mu$l as follows: $\mu$l EconoTaq Master mix, $\mu$l molecular water, $\mu$l of each primer and $\mu$l template. The PCR protocol was the following:  min at $30\,^{\circ}\mathrm{C}$, then  cycles of denaturing phase:   at $30\,^{\circ}\mathrm{C}$, annealing phase:  at  $30\,^{\circ}\mathrm{C}$, extension phase:  at $30\,^{\circ}\mathrm{C}$, followed by a final extension phase: 10 min at $72\,^{\circ}\mathrm{C}$ and storage at $4\,^{\circ}\mathrm{C}$.

\end{document}
\end{figure}
